# mbioinfo.2021

**Table of Contents**
- [unclassified](#unclassified)
- [nucleotide_composition](#nucleotide_composition) 塩基組成、GC含量
- [MoBE](#mobe) 人工環境の微生物 Microbiome of the Built Environment (MoBE)
- [ISS](#iss) International Space Station 国際宇宙ステーション
- [urban_microbiome](#urban_microbiome) 都市の微生物群集 MetaSUB
- [microbiome](#microbiome) 微生物群集
- [MAGUK](#maguk) membrane-associated guanylate kinases
- [tree](#tree) 系統樹
  - [jlsteenwyk](#jlsteenwyk)
- [phylogenetic_profiling](#phylogenetic_profiling)
- [date](#date) Phylogenetic Dating 分岐年代推定
- [niid](#niid) National Institute of Infectious Diseases 国立感染症研究所
- [AMR](#amr) 薬剤耐性
- [plasmid](#plasmid) プラスミド
- [CRISPR](#crispr)
- [phenotype](#phenotype)
- [recombination](#recombination)
- [money](#money) お金 お札 ATM
- [MRSA](#mrsa)
- [](#)
- [monumentome](#monumentome)
- [publication](#publication)
- [five-second rule](#five-second-rule)
- [](#)

----------
## 

----------
## unclassified

コドン使用
https://github.com/haruosuz/codon

相同
https://github.com/haruosuz/evolve/blob/master/references/README.evolve.jargon.md#homology

https://github.com/haruosuz/mgsa/blob/master/references/mgsa.tools.md#2021-06-05
https://github.com/bacteria-archaea-traits/bacteria-archaea-traits
細菌と古細菌の表現型形質データの統合

https://github.com/haruosuz/r4bioinfo/blob/master/R_tree/README.md#ancestral-reconstruction

https://github.com/haruosuz/DS4GD/blob/master/2018/CaseStudy.md#genome-signature

遺伝子の系統的分布
https://github.com/haruosuz/mgsa/blob/master/references/mgsa.phylogenetic_distribution.md

2018年12月11日に更新
https://qiita.com/haruosuz/items/d95de2fa3e5c39e5f35b
Galaxyの公共サーバ（微生物解析用） - Qiita

本：微生物
https://github.com/haruosuz/books/blob/master/references/books.microbe.md

本：バイオインフォマティクス
https://github.com/haruosuz/books/blob/master/references/books.bioinfo.md

https://www.editage.com/insights/what-types-of-articles-do-journals-publish
Choose the right format for publishing your research | Editage Insights

- https://www.nig.ac.jp/jimu/soken/courses/EfS/EfS_2016/OSC_I_schedule.html
Asking questions
download tips for Q&A English Japanese
- https://www.nig.ac.jp/jimu/soken/courses/EfS/contents/Q&A_e.pdf
Let’s Enjoy the Q&A Session!
- https://www.nig.ac.jp/jimu/soken/courses/EfS/contents/Q&A_j.pdf
Ｑ＆Ａを楽しもう！

----------
## phenotype

https://github.com/haruosuz/mgsa/blob/master/references/mgsa.tools.md#metadata

### growth rate
増殖速度
https://github.com/haruosuz/mgsa/blob/master/references/mgsa.tools.md#growth-rate

### optimal growth temperature
至適増殖温度

### 2019-09-15
https://pubmed.ncbi.nlm.nih.gov/30689741/
Bioinformatics
. 2019 Sep 15;35(18):3224-3231. doi: 10.1093/bioinformatics/btz059.
Predicting the optimal growth temperatures of prokaryotes using only genome derived features
David B Sauer 1, Da-Neng Wang 1
https://academic.oup.com/bioinformatics/article/35/18/3224/5301315
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6748728/

### 2007-01-12
https://pubmed.ncbi.nlm.nih.gov/17222055/
PLoS Comput Biol
. 2007 Jan 12;3(1):e5. doi: 10.1371/journal.pcbi.0030005. Epub 2006 Nov 30.
Protein and DNA sequence determinants of thermophilic adaptation
Konstantin B Zeldovich 1, Igor N Berezovsky, Eugene I Shakhnovich
https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.0030005
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1769408/

----------
## nucleotide_composition
塩基組成、GC含量

https://github.com/haruosuz/mgsa/blob/master/references/mgsa.nucleotide_composition.md

----------
## MoBE

人工環境の微生物
Microbiology of the Built Environment (MoBE)
https://github.com/haruosuz/microbe/blob/master/references/README.MoBE.md

----------
## ISS
International Space Station
国際宇宙ステーション

https://github.com/haruosuz/microbe/blob/master/references/microbe_ISS.md

----------
## urban_microbiome

都市の微生物群集
https://github.com/haruosuz/microbe/blob/master/references/microbe.urban.md

----------
## microbiome
微生物群集

https://github.com/haruosuz/mgsa/blob/master/references/mgsa.metagenome.md

https://github.com/haruosuz/mgsa/blob/master/references/mgsa.rRNA.md

1:15 AM · May 23, 2021
https://twitter.com/phylogenomics/status/1396137556345974788
Jonathan Eisen on Twitter: "This is really a phenomenally useful paper that I think everyone interested in #microbiome studies should read: "Identifying biases and their potential solutions in human microbiome studies" by @JTNearing et al. https://t.co/dhWrS1sj1E https://t.co/Icn6HWhkrN" / Twitter

https://pubmed.ncbi.nlm.nih.gov/34006335/
Review Microbiome
. 2021 May 18;9(1):113. doi: 10.1186/s40168-021-01059-0.
Identifying biases and their potential solutions in human microbiome studies
Jacob T Nearing 1, André M Comeau 2, Morgan G I Langille 3 4
https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01059-0
batch effect
バッチ効果

2020.03.19
https://bio.nikkeibp.co.jp/atcl/news/p1/20/03/18/06702/
トヨタとPBS、遺伝研、バイオフィリックデザインの大気菌叢変化の要因推定：
導入した植物の葉面や土壌などメタゲノム解析

----------
## MAGUK
membrane-associated guanylate kinases

https://en.wikipedia.org/wiki/Membrane-associated_guanylate_kinase

https://ebi.ac.uk/interpro/entry/panther/PTHR23122/taxonomy/uniprot/#sunburst
MEMBRANE-ASSOCIATED GUANYLATE KINASE MAGUK (PTHR23122) - PANTHER entry - InterPro

https://en.wikipedia.org/wiki/Guanylate_kinase
In prokaryotes (such as Escherichia coli), lower eukaryotes (such as yeast) and in vertebrates, GK is a highly conserved monomeric protein of about 200 amino acids. GK has been shown to be structurally similar to protein A57R (or SalG2R) from various strains of Vaccinia virus.[2][3][4] Systems biology analyses carried out by the team of Andreas Dräger also identified a pivotal role of this enzyme in the replication of SARS-CoV-2 within the human airways.[5][6][7]

https://pubmed.ncbi.nlm.nih.gov/20359327/
BMC Evol Biol
. 2010 Apr 1;10:93. doi: 10.1186/1471-2148-10-93.
Evolution of the MAGUK protein gene family in premetazoan lineages
Alex de Mendoza 1, Hiroshi Suga, Iñaki Ruiz-Trillo
https://bmcecolevol.biomedcentral.com/articles/10.1186/1471-2148-10-93
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2859873/

The MAGUK family have been divided into different classes or groups, according to phylogenetic position and protein domain architecture (see for example [2] and [10], and Figure ​Figure11 for our own MAGUK classification). These MAGUKs classes are known as calcium/calmodulin-dependent proteins kinase (CASK), palmitoylated membrane protein (MPP), zona occludens (ZO), caspase recruitment domain family (CARMA), Disc Large Homolog (DLG), Calcium channel β subunit (CACNB), and membrane-associated guanylate kinase with an inverted repeat (MAGI). All classes contain the following: one or several PDZ domains (except CACNB), a catalytically inactive guanylate kinase (GUK) domain with homology to yeast guanylate kinase and a Src Homology 3 (SH3) domain (except for MAGI) (Figure ​(Figure1).1).

Figure 1
Domain architectures of the different MAGUK classes.

Figure 2
Unrooted phylogenetic tree of GUK domain sequences. 

Figure 3
Phylogeny of SH3+GUK domain sequences.

Figure 4
Protein domain architectures for the Capsaspora and Monosiga MAGUK and MAGUK-like homologs, except for MAGI.

Figure 5
Distribution of MAGUKs in opisthokonts. 

https://pubmed.ncbi.nlm.nih.gov/35659305/
Microbiome
. 2022 Jun 4;10(1):84. doi: 10.1186/s40168-022-01274-3.
Ecogenomics sheds light on diverse lifestyle strategies in freshwater CPR
https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01274-3
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9166423/
Moreover, inosine monophosphate (IMP) can be synthesized from phosphoribosyl diphosphate by Gracilibacteria, Microgenomatia, and Paceibacteria. Further processing of IMP to ATP occurs in ABY1, Gracilibacteria, Microgenomatia, and Paceibacteria, but the synthesis of GTP is not completely encoded in Microgenomatia which lack guanylate kinase. 


https://pubmed.ncbi.nlm.nih.gov/19732467/
BMC Evol Biol
. 2009 Sep 7;9:222. doi: 10.1186/1471-2148-9-222.
FoxO gene family evolution in vertebrates
Minghui Wang 1, Xiangzhe Zhang, Hongbo Zhao, Qishan Wang, Yuchun Pan
https://bmcecolevol.biomedcentral.com/articles/10.1186/1471-2148-9-222
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2746812/
https://link.springer.com/article/10.1186/1471-2148-9-222
Synonymous codon usage analyses

Estimation of substitution rates and testing natural selection

the ratio of the nonsynonymous (dN) to synonymous substitution rates (dS), ω = dN/dS

----------
## recombination
組み換え

https://pubmed.ncbi.nlm.nih.gov/30670614/
mBio
. 2019 Jan 22;10(1):e02494-18. doi: 10.1128/mBio.02494-18.
Impact of Homologous Recombination on the Evolution of Prokaryotic Core Genomes
Pedro González-Torres 1 2 3, Francisco Rodríguez-Mateos 4, Josefa Antón 1 5, Toni Gabaldón 6 3 7
https://journals.asm.org/doi/10.1128/mbio.02494-18
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6343036/

https://pubmed.ncbi.nlm.nih.gov/17110367/
Bioinformatics
. 2006 Dec 15;22(24):3096-8. doi: 10.1093/bioinformatics/btl474. Epub 2006 Nov 16.
GARD: a genetic algorithm for recombination detection

Motivation: Phylogenetic and evolutionary inference can be severely misled if recombination is not accounted for, hence screening for it should be an essential component of nearly every comparative study. The evolution of recombinant sequences can not be properly explained by a single phylogenetic tree, but several phylogenies may be used to correctly model the evolution of non-recombinant fragments.

https://pubmed.ncbi.nlm.nih.gov/11847565/
J Mol Evol
. 2002 Mar;54(3):396-402. doi: 10.1007/s00239-001-0034-9.
The effect of recombination on the accuracy of phylogeny estimation
David Posada 1, Keith A Crandall
https://link.springer.com/article/10.1007/s00239-001-0034-9

https://pubmed.ncbi.nlm.nih.gov/11114432/
Trends Ecol Evol
. 2000 Dec 1;15(12):489-490. doi: 10.1016/s0169-5347(00)02027-9.
How does recombination affect phylogeny estimation?
D Posada
https://www.cell.com/trends/ecology-evolution/fulltext/S0169-5347(00)02027-9
To read this article in full you will need to make a payment

https://pubmed.ncbi.nlm.nih.gov/11014833/
Genetics
. 2000 Oct;156(2):879-91. doi: 10.1093/genetics/156.2.879.
Consequences of recombination on traditional phylogenetic analysis
M H Schierup 1, J Hein
https://academic.oup.com/genetics/article/156/2/879/6051420
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1461297/

----------
## tree
系統樹

https://www.jstage.jst.go.jp/article/jsbibr/2/1/2_jsbibr.2021.7/_html/-char/ja
総説
分子系統解析の最前線
松井 求 
2021 年 2 巻 1 号 p. 30-57
3.5　系統樹の信頼性評価
得られた系統樹の統計的信頼性を評価するために標準的に使用されているのがBootstrap法［70］である。
信頼性評価法の歴史についてはSimonの総説を参照されたい［74］。

系統樹に根(root)をつける
https://github.com/haruosuz/evolve/blob/master/references/README.evolve.jargon.md#root

https://github.com/haruosuz/evolve/blob/master/references/README.evolve.tools.md#iqtree
Dec 1, 2022
http://www.iqtree.org/doc/Concordance-Factor
gCF and sCF complement classical measures of branch support (e.g. bootstrap) in phylogenetics by providing a full description of underlying disagreement among loci and sites.
HINT: See very nice tips on how to use and interpret concordance factors written by Rob Lanfear.
https://www.robertlanfear.com/blog/files/concordance_factors.html
Calculating and interpreting gene- and site-concordance factors in phylogenomics
How concordance factors relate to each other and to bootstraps

### jlsteenwyk

https://github.com/haruosuz/evolve/blob/master/references/README.evolve.tools.md#jlsteenwyk

### 2019-07-09
https://pubmed.ncbi.nlm.nih.gov/31289177/
mBio
. 2019 Jul 9;10(4):e00925-19. doi: 10.1128/mBio.00925-19.
A Robust Phylogenomic Time Tree for Biotechnologically and Medically Important Fungi in the Genera Aspergillus and Penicillium
Jacob L Steenwyk 1, Xing-Xing Shen 1, Abigail L Lind 2 3, Gustavo H Goldman 4, Antonis Rokas 5 2
https://journals.asm.org/doi/10.1128/mBio.00925-19
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC6747717/

Topology tests. The phylogeny of the genera Aspergillus and Penicillium has been a topic of debate. Our topology supports the reciprocal monophyly of Aspergillus and Penicillium and rejects the monophyly of narrow Aspergillus. Both of these results are consistent with some previous studies (61) (see Fig. 6) but in contrast to other previous studies, which recovered a topology where Penicillium is sister to section Nidulantes within Aspergillus and narrow Aspergillus (sections Nigri, Ochraceorosei, Flavi, Circumdati, Candidi, and Terrei) is monophyletic (43, 62). To further evaluate both of these hypotheses, we conducted separate topology constraint analyses using the Shimodaira-Hasegawa test (69) and the approximately unbiased tests (70). Both tests rejected the constrained topologies (Table 1; P value < 0.001 for all tests), providing further support that Aspergillus and Penicillium are reciprocally monophyletic and that narrow Aspergillus is not monophyletic (Fig. 5). 

FIG 5
A visual comparison of the differences between the phylogeny reported in this study and the phylogeny reported in the work of Kocsubé et al. (61). 

### 2024-01-08
https://x.com/EvolvedBiofilm/status/1812461344186085459
Akos T. Kovacs on X: "Phylogenetic reconciliation: making the most of genomes to understand microbial ecology and evolution @ISMEJournal from @sllsi with @PhilHugenholtz @anjspa1 @micecoevo @Paul__Sheridan @eme_laura @RochelleSoo1 @wwood @drenal_eu https://t.co/pWKtIBq0mD https://t.co/lE7qVnSkxz" / X
総説
https://pubmed.ncbi.nlm.nih.gov/39001714/
Review ISME J
. 2024 Jan 8;18(1):wrae129. doi: 10.1093/ismejo/wrae129.
Phylogenetic reconciliation: making the most of genomes to understand microbial ecology and evolution
Tom A Williams 1, Adrian A Davin 2, Lénárd L Szánthó 3 4, Alexandros Stamatakis 5 6 7, Noah A Wahl 5, Ben J Woodcroft 8, Rochelle M Soo 9, Laura Eme 10, Paul O Sheridan 11, Cecile Gubry-Rangin 12, Anja Spang 13 14, Philip Hugenholtz 9, Gergely J Szöllősi 3 4 15
https://academic.oup.com/ismej/article/doi/10.1093/ismejo/wrae129/7713227?login=false

### 2022-06-16

https://github.com/haruosuz/evolve/blob/master/references/README.evolve.tools.md#iqtree_rootstrap
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9260635/

Microchiroptera–Megachiroptera or Yinpterochiroptera–Yangochiroptera?
Using the whole amino acid data set, our results show 65.5% rootstrap support for the Yinpterochiroptera–Yangochiroptera hypothesis and 23.2% for the Microchiroptera–Megachiroptera hypothesis. The remaining11.3% of the rootstrap support goes to supporting the branch leading to Rhinolophoidea as root branch of the bats (Fig. 4).

figure 4.
The ML rooted tree as inferred from the whole Chiroptera amino acid data set. 

The Ambiguous Root of Cetartiodactyla
The ML tree inferred with the whole amino acid data set places the clade containing Tylopoda (represented by its only extant family; Camelidae) and Suina as the sister group to all other cetartiodactylans with 71.8% rootstrap support (Fig. 6). Yet, The AU test did not reject Tylopoda alone as the sister group to all other cetartiodactylans. 

figure 6.
The ML rooted tree of as inferred from the whole Cetartiodactyla amino acid data set. 

### 2019-12-01

https://github.com/haruosuz/evolve/blob/master/references/README.evolve.tools.md#iqtree_assessing-phylogenetic-assumptions
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6893154/

The tree reconstructed from all of the partitions (Tall) and the tree reconstructed from the partitions that pass the MaxSymTest (Tpass, 29 partitions) both show Perissodactyla (odd-toed ungulates) as a sister group to Cetartiodactyla (even-toed ungulates) (fig. 3a and supplementary extended figs. 4 and 5, Supplementary Material online). Even so, the bootstrap support for this branch is not high: 73% for Tall and 34% for Tpass. On the other hand, the tree reconstructed from the data that fail the MaxSymTest (Tfail, 22 partitions) shows Perissodactyla as the sister group to the clade that contains Carnivora + Pholidota with 49% bootstrap support (fig. 3b and supplementary extended fig. 6, Supplementary Material online).

fig. 3.
—Maximum-likelihood trees of mammalian relationships based on analysis of Lartillot 2012 data set. 

The tree inferred from the partitions that show significant violation of the SRH conditions (Tfail) differs in its topology from the tree inferred from the partitions that do not show significant violation of the SRH conditions (Tpass) with respect to the interordinal relationships in Laurasiatheria (fig. 3). The tree inferred from partitions that violate the SRH conditions (Tfail) is consistent with the results from the original paper in that it places Perissodactyla as a sister group to Carnivora + Pholidota (Lartillot and Delsuc 2012). However, other studies using ML analysis show Perissodactyla to be a sister group to Cetartiodactyla (Graur et al. 1997; Murphy et al. 2001; Tsagkogeorga et al. 2013; Liu et al. 2017), which is also the relationship we find in this study with the tree inferred from partitions that do not show significant violation of the SRH assumptions.


### 2023-12
https://pubmed.ncbi.nlm.nih.gov/37369847/
Review Nat Rev Genet
. 2023 Dec;24(12):834-850. doi: 10.1038/s41576-023-00620-x. Epub 2023 Jun 27.
Incongruence in the phylogenomics era
Jacob L Steenwyk 1 2 3, Yuanning Li 4, Xiaofan Zhou 5, Xing-Xing Shen 6, Antonis Rokas 7 8 9
https://search.lib.keio.ac.jp/permalink/81SOKEI_KEIO/uccs31/cdi_proquest_miscellaneous_2830666552
https://www-nature-com.kras.lib.keio.ac.jp/articles/s41576-023-00620-x
Measures of branch support

### 2023-07-27

https://pubmed.ncbi.nlm.nih.gov/37853257/
Published Erratum Nat Biotechnol
. 2023 Oct 18. doi: 10.1038/s41587-023-02027-9. Online ahead of print.
Author Correction: Generation of accurate, expandable phylogenomic trees with uDance
Metin Balaban 1, Yueyu Jiang 2, Qiyun Zhu 3 4, Daniel McDonald 5, Rob Knight 5 6 7 8, Siavash Mirarab 9 10 11

https://pubmed.ncbi.nlm.nih.gov/37500914/
Nat Biotechnol
. 2023 Jul 27:10.1038/s41587-023-01868-8. doi: 10.1038/s41587-023-01868-8. Online ahead of print.
Generation of accurate, expandable phylogenomic trees with uDance
Metin Balaban 1, Yueyu Jiang 2, Qiyun Zhu 3 4, Daniel McDonald 5, Rob Knight 5 6 7 8, Siavash Mirarab 9 10 11
https://search.lib.keio.ac.jp/permalink/81SOKEI_KEIO/uccs31/cdi_proquest_miscellaneous_2854431047
https://www-nature-com.kras.lib.keio.ac.jp/articles/s41587-023-01868-8
Gene tree conflicts, branch support and diversity across the tree
uDance produces species and partition trees with branch support (local posterior probability), thanks to ASTRAL35. 

### 2022-06-16
https://pubmed.ncbi.nlm.nih.gov/32915964/
Syst Biol
. 2022 Jun 16;71(4):921-928. doi: 10.1093/sysbio/syaa068.
An Evolving View of Phylogenetic Support
Chris Simon 1
https://academic.oup.com/sysbio/article/71/4/921/5904279?login=false
[Bayes factors; bootstrap; branch support; concordance factors; internode certainty; posterior probabilities; spectral analysis; transfer bootstrap expectation.].

### 2022-02-03
### Krieger_2022-02-03

https://pubmed.ncbi.nlm.nih.gov/35084505/
Mol Biol Evol
. 2022 Feb 3;39(2):msac020. doi: 10.1093/molbev/msac020.
A Peroxide-Responding sRNA Evolved from a Peroxidase mRNA
Madeline C Krieger 1 2, H Auguste Dutcher 1 3, Andrew J Ashford 1 4, Rahul Raghavan 
https://academic.oup.com/mbe/article/39/2/msac020/6516022
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8825220/

### 2021-05
### Coleman_2021

https://pubmed.ncbi.nlm.nih.gov/33958449/
Science
. 2021 May 7;372(6542):eabe0511. doi: 10.1126/science.abe0511.
A rooted phylogeny resolves early bacterial evolution
Gareth A Coleman # 1, Adrián A Davín # 2, Tara A Mahendrarajah 3, Lénárd L Szánthó 4 5, Anja Spang 3 6, Philip Hugenholtz # 7, Gergely J Szöllősi # 8 5 9, Tom A Williams # 10
https://science.sciencemag.org/content/372/6542/eabe0511
Our analyses place the root between two major bacterial clades, the Gracilicutes and Terrabacteria. We found no support for a root between the Candidate Phyla Radiation (CPR), a lineage comprising putative symbionts and parasites with small genomes, and all other Bacteria. Instead, the CPR was inferred to be a member of the Terrabacteria and formed a sister lineage to the Chloroflexota and Dormibacterota. 

### 2021-09
https://pubmed.ncbi.nlm.nih.gov/34734192/
Nat Comput Sci
. 2021 Sep;1(9):573-577. doi: 10.1038/s43588-021-00129-5. Epub 2021 Sep 22.
Fast and accurate bootstrap confidence limits on genome-scale phylogenies using little bootstraps
Sudip Sharma 1 2, Sudhir Kumar 1 2 3
https://www.nature.com/articles/s43588-021-00129-5
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8560003/

### 2021-01
https://pubmed.ncbi.nlm.nih.gov/33510015/
Science
. 2021 Jan 29;371(6528):466-467. doi: 10.1126/science.abf3995.
Insights from SARS-CoV-2 sequences
Michael A Martin 1 2, David VanInsberghe 1, Katia Koelle 3 4
https://science.sciencemag.org/content/371/6528/466.long

Uses for viral sequence data
Viral phylogenies, rooted at the most recent common ancestor (TMRCA), are inferred on the basis of genetic differences. These phylogenies can be used to estimate viral emergence, characterize the geographic spread of the virus, reconstruct epidemiological dynamics of viral spread within a region, and identify instances of adaptation.

### 2020-07
https://pubmed.ncbi.nlm.nih.gov/32424311/
Review Nat Rev Genet
. 2020 Jul;21(7):428-444. doi: 10.1038/s41576-020-0233-0. Epub 2020 May 18.
Phylogenetic tree building in the genomic age
Paschalia Kapli 1, Ziheng Yang 1, Maximilian J Telford 2
https://www.nature.com/articles/s41576-020-0233-0
https://search.lib.keio.ac.jp/permalink/81SOKEI_KEIO/uccs31/cdi_pubmed_primary_32424311
https://www-nature-com.kras.lib.keio.ac.jp/articles/s41576-020-0233-0
Confidence in clades using the bootstrap

### 2020-07
https://pubmed.ncbi.nlm.nih.gov/32427978/
Nat Microbiol
. 2020 Jul;5(7):876-877. doi: 10.1038/s41564-020-0738-5.
Phylogenetic interpretation during outbreaks requires caution
Ch Julián Villabona-Arenas 1 2, William P Hanage 3, Damien C Tully 4 5
https://www.nature.com/articles/s41564-020-0738-5
severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)

### 2018-06
https://pubmed.ncbi.nlm.nih.gov/29795540/
Review Nat Microbiol
. 2018 Jun;3(6):652-661. doi: 10.1038/s41564-018-0156-0. Epub 2018 May 24.
Methods for phylogenetic analysis of microbiome data
Alex D Washburne # 1, James T Morton # 2 3, Jon Sanders 3, Daniel McDonald 3, Qiyun Zhu 3, Angela M Oliverio 4 5, Rob Knight 2 3
https://www.nature.com/articles/s41564-018-0156-0
https://search.lib.keio.ac.jp/permalink/81SOKEI_KEIO/1rcqpmf/cdi_proquest_miscellaneous_2045274002
https://www-nature-com.kras.lib.keio.ac.jp/articles/s41564-018-0156-0
In this Review Article, we organize the field of phylogenetically structured data analysis by discussing the major classes of methods. We first emphasize a fundamental issue in the field: the imperfection of estimated phylogenies. We then define four classes: (1) comparative methods; (2) ancestral state reconstruction and descendant trait imputation; (3) phylogenetic variables; and (4) phylogeny-aware distances (Table 1); and provide Supplementary Online Tutorials with examples (https://knightlab-analyses.github.io/phylogenetic-tutorials/). 

### Guglielmini_2013

https://pubmed.ncbi.nlm.nih.gov/22977114/
Mol Biol Evol
. 2013 Feb;30(2):315-31. doi: 10.1093/molbev/mss221. Epub 2012 Sep 13.
Evolution of conjugation and type IV secretion systems
Julien Guglielmini 1, Fernando de la Cruz, Eduardo P C Rocha
https://academic.oup.com/mbe/article/30/2/315/1013981
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC3548315/

Fig. 1.
Scheme of the most-studied T4SS, the vir system of A. tumefaciens Ti plasmid. The VirBX proteins are depicted as BX (e.g., B5 refers to the VirB5 protein). The coupling protein VirD4 (D4) and the mobilization complex, which includes the relaxase (MOB)-DNA complex are also represented. OM: outer membrane; IM: inner membrane.

### Garcillan-Barcia_2009

https://pubmed.ncbi.nlm.nih.gov/19396961/
Review FEMS Microbiol Rev
. 2009 May;33(3):657-87. doi: 10.1111/j.1574-6976.2009.00168.x.
The diversity of conjugative relaxases and its application in plasmid classification
María Pilar Garcillán-Barcia 1, María Victoria Francia, Fernando de la Cruz
https://academic.oup.com/femsre/article/33/3/657/591359

Figure 1
A scheme of the relationships between the main relaxase protein families. A first relaxase cluster (shown on a dark-grey background) contains relaxase groups that contain just one active Tyr in the catalytic centre (see text). A second relaxase cluster (light-grey) contains relaxases with two Tyr in the catalytic centre. For the remaining groups (white background) not enough is known about the biochemistry of the respective relaxases. According to the analysis of their amino acid sequences and of experimental work (see text) they seem to be nonhomologous, and thus may use different DNA-processing mechanisms. Some relaxase protein families also overlap other protein families, such as plasmid RC-replication proteins (Rep), IS91-like transposases (IS91) or HD hydrolases. Areas of circles are proportional to relaxase number. The MOBP area includes (MOBP+MOBHEN+MOBQ1).

Figure 4
Conserved sequence motifs in representative relaxase protein clades within the MOBF family. The clades to which the sequences belong are shown at the left margin of the figure. The CLUSTALW alignment considered only the 300 N-terminal residues of each protein, containing the relaxase domain. Vertical arrowheads point to residues known to be important for function in TrwC_R388 (Guasch et al., 2003). 


----------
## phylogenetic_profiling

https://en.wikipedia.org/wiki/Phylogenetic_profiling

### 2022-04-14

https://pubmed.ncbi.nlm.nih.gov/35421089/
PLoS One
. 2022 Apr 14;17(4):e0251833. doi: 10.1371/journal.pone.0251833. eCollection 2022.
Phylogenetic profiling in eukaryotes: The effect of species, orthologous group, and interactome selection on protein interaction prediction
Eva S Deutekom 1, Teunis J P van Dam 1, Berend Snel 1
https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0251833
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9009711/

### 2021-10-29

https://pubmed.ncbi.nlm.nih.gov/34715785/
BMC Genomics
. 2021 Oct 29;22(1):774. doi: 10.1186/s12864-021-08070-7.
GFICLEE: ultrafast tree-based phylogenetic profile method inferring gene function at the genomic-wide level
Yang Fang 1, Menglong Li 2, Xufeng Li 1, Yi Yang 3
https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-021-08070-7
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8557005/

### 2020-07-22

https://pubmed.ncbi.nlm.nih.gov/32697802/
PLoS Comput Biol
. 2020 Jul 22;16(7):e1007553. doi: 10.1371/journal.pcbi.1007553. eCollection 2020 Jul.
Scalable phylogenetic profiling using MinHash uncovers likely eukaryotic sexual reproduction genes
David Moi 1 2 3, Laurent Kilchoer 1 2 3, Pablo S Aguilar 4 5, Christophe Dessimoz 1 2 3 6 7
https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1007553
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7423146/

----------
## date
分岐年代推定

https://github.com/haruosuz/evolve/blob/master/references/README.evolve.jargon.md#divergence-time

https://github.com/haruosuz/evolve/blob/master/references/README.evolve.tools.md#timetree

IQ-TREE
http://www.iqtree.org/doc/Dating
Phylogenetic Dating

BactDating
https://github.com/haruosuz/r4bioinfo/blob/master/R_tree/README.md#bactdating

### TempEst

http://tree.bio.ed.ac.uk/software/tempest/
Can import Newick as well as NEXUS tree files

https://beast.community/tempest

https://beast.community/tempest_tutorial
Using TempEst for data exploration | BEAST Documentation
https://beast.community/tempest_tutorial#the-temporal-signal-and-rooting

https://pubmed.ncbi.nlm.nih.gov/27774300/
Virus Evol
. 2016 Apr 9;2(1):vew007. doi: 10.1093/ve/vew007. eCollection 2016 Jan.
Exploring the temporal structure of heterochronous sequences using TempEst (formerly Path-O-Gen)
Andrew Rambaut 1, Tommy T Lam 2, Luiz Max Carvalho 3, Oliver G Pybus 4
https://academic.oup.com/ve/article-lookup/doi/10.1093/ve/vew007
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC4989882/

root-to-tip

Although this procedure finds the root for which the data appear most ‘clock-like’, it does allow the possibility of a negative slope being offered. This is not a problem, because the estimation of a negative evolutionary rate indicates that the data set contains little or no temporal signal.

### 2021-03-30

https://pubmed.ncbi.nlm.nih.gov/35720267/
iScience
. 2022 May 30;25(7):104477. doi: 10.1016/j.isci.2022.104477. eCollection 2022 Jul 15.
Helicobacter pylori genomes reveal Paleolithic human migration to the east end of Asia
Rumiko Suzuki 1, Naruya Saitou 2, Osamu Matsuari 3, Seiji Shiota 4, Takashi Matsumoto 1, Junko Akada 1, Nagisa Kinjo 5, Fukunori Kinjo 6, Kuniko Teruya 7, Makiko Shimoji 7, Akino Shiroma 7, Mototsugu Kato 8, Kazuhito Satou 7, Takashi Hirano 7, Masahiro Asaka 9, Kirill Kryukov 2, Yoshan Moodley 10, Yoshio Yamaoka 1 11
https://www.cell.com/iscience/fulltext/S2589-0042(22)00748-9
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9204748/
Estimation of the divergence dates
Based on the populations defined by fineSTRUCTURE, we selected representative strains to estimate divergence dates. We detected recombination signals by using the 4-gamete test and eliminated recombinant regions. Then, we constructed a Bayesian tree using BEAST2 (Bouckaert et al., 2014) and estimated the divergence dates. The divergence date of hspOkinawa was estimated around 20 thousand years ago (kya), and that of the cluster including hpRyukyu and Afghanistan-Nepal strains was estimated around 45 kya (Figure 2).

### 2021-03-30
### Yahara_2021

4:37 AM · Apr 5, 2021
https://twitter.com/KojiYahara/status/1378793886336778240
Koji Yahara on Twitter: "Our new paper on emergence and evolution of antimicrobial resistance genes and mutations in N. gonorrhoeae is out in Genome Medicine https://t.co/X2BhtHRYx3 (1/8)" / Twitter

https://pubmed.ncbi.nlm.nih.gov/33785063/
Genome Med
. 2021 Mar 30;13(1):51. doi: 10.1186/s13073-021-00860-8.
Emergence and evolution of antimicrobial resistance genes and mutations in Neisseria gonorrhoeae
Koji Yahara 1, Kevin C Ma 2, Tatum D Mortimer 2, Ken Shimuta 3 4, Shu-Ichi Nakayama 4, Aki Hirabayashi 3, Masato Suzuki 3, Michio Jinnai 5, Hitomi Ohya 5, Toshiro Kuroki 5 6, Yuko Watanabe 5, Mitsuru Yasuda 7, Takashi Deguchi 8, Vegard Eldholm 9, Odile B Harrison 10, Martin C J Maiden 10, Yonatan H Grad 2, Makoto Ohnishi 11
https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-021-00860-8
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8008663/
Here, we obtained complete, closed, genome sequences of three Japanese ST-1901 N. gonorrhoeae isolates harboring penA-34 from 2005 and reconstructed a dated phylogeny based on core-genome alignment of 1075 genome sequences from isolates sampled from 35 countries. 

![](https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8008663/bin/13073_2021_860_Fig1_HTML.jpg)

Fig. 1
Whole genome sequence dated phylogeny, antimicrobial resistance, and genetic polymorphisms in the ST-1901-associated core-genome group. Left: A clonal dated phylogeny with corrected branch lengths to account for homologous recombination. 

### Yahara_2018

https://pubmed.ncbi.nlm.nih.gov/30063202/
Microb Genom
. 2018 Aug;4(8):e000205. doi: 10.1099/mgen.0.000205. Epub 2018 Jul 31.
Genomic surveillance of Neisseria gonorrhoeae to investigate the distribution and evolution of antimicrobial-resistance determinants and lineages
Koji Yahara 1, Shu-Ichi Nakayama 2, Ken Shimuta 2, Ken-Ichi Lee 2, Masatomo Morita 2, Takuya Kawahata 3, Toshiro Kuroki 4 5, Yuko Watanabe 4, Hitomi Ohya 4, Mitsuru Yasuda 6, Takashi Deguchi 6, Xavier Didelot 7, Makoto Ohnishi 2 1
https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000205#tab2
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC6159555/

In order to demonstrate that the data of the lineages have temporal signal, we plotted the sampling year versus root-to-tips correlation using TempEst [45] (Fig. S2). We also performed the date randomization tests using the ‘roottotip’ function of BactDating [46] and confirmed that the temporal signal is significant in both ST7363- and ST1901-associated lineages (P=0.018 and P <10−5). Using the maximum-likelihood tree as a starting tree, we applied the standard model of ClonalFrameML to each linage and inferred individual recombination events imported from outside the lineage. We excluded all recombined sequences from the whole-genome alignment, and then extracted 1079 and 3255 core biallelic SNPs for the ST7363- and ST1901-associated lineages. We then constructed a time-resolved phylogeny using the SNPs and beast version 1.10.0 [47], for which we specified the GTR +G4 model of DNA substitution with estimation of the shape parameter of the gamma distribution. 

- TempEst
- BactDating
- ClonalFrameML
- beast 

![](https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC6159555/bin/mgen-5-205-g003.jpg)

Fig. 3.
Time-resolved phylogeny and penA polymorphisms. 

Plasmid-encoded resistance

https://github.com/haruosuz/plasmids/blob/master/references/plasmid_resistance.md
## Francois Balloux

### 2022-03-03

https://pubmed.ncbi.nlm.nih.gov/35241674/
Nat Commun
. 2022 Mar 3;13(1):1131. doi: 10.1038/s41467-022-28819-2.
Role of mobile genetic elements in the global dissemination of the carbapenem resistance gene blaNDM
Mislav Acman 1, Ruobing Wang 2, Lucy van Dorp 3, Liam P Shaw 4, Qi Wang 2, Nina Luhmann 5, Yuyao Yin 2, Shijun Sun 2, Hongbin Chen 2, Hui Wang 2, Francois Balloux 3
https://www.nature.com/articles/s41467-022-28819-2
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8894482/

### 2018-03-21

https://pubmed.ncbi.nlm.nih.gov/29563494/
Nat Commun
. 2018 Mar 21;9(1):1179. doi: 10.1038/s41467-018-03205-z.
The global distribution and spread of the mobilized colistin resistance gene mcr-1
Ruobing Wang 1, Lucy van Dorp 2, Liam P Shaw 2, Phelim Bradley 3, Qi Wang 1, Xiaojuan Wang 1, Longyang Jin 1, Qing Zhang 4, Yuqing Liu 4, Adrien Rieux 5, Thamarai Dorai-Schneiders 6, Lucy Anne Weinert 7, Zamin Iqbal 3 8, Xavier Didelot 9, Hui Wang 10, Francois Balloux 11
https://www.nature.com/articles/s41467-018-03205-z
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC5862964/

### Lam_2018
### 2018-07-13

https://pubmed.ncbi.nlm.nih.gov/30006589/
Nat Commun
. 2018 Jul 13;9(1):2703. doi: 10.1038/s41467-018-05114-7.
Population genomics of hypervirulent Klebsiella pneumoniae clonal-group 23 reveals early emergence and rapid global dissemination
Margaret M C Lam 1, Kelly L Wyres 1, Sebastian Duchêne 1, Ryan R Wick 1, Louise M Judd 1, Yunn-Hwen Gan 2, Chu-Han Hoh 2, Sophia Archuleta 3, James S Molton 3, Shirin Kalimuddin 4, Tse Hsien Koh 5, Virginie Passet 6, Sylvain Brisse 6, Kathryn E Holt 7 8
https://www.nature.com/articles/s41467-018-05114-7
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC6045662/
We detected a strong temporal signal in the CG23 genome alignment (see Supplementary Fig. 2a-c, Supplementary Methods), sufficient to estimate evolutionary rates and dates for the most recent common ancestors (MRCAs) of key CG23 lineages with BEAST (best-fitting Bayesian model was the UCLD constant model, see Supplementary Methods). The mean evolutionary rate for CG23 was estimated to be 3.40 × 10−7 substitutions site−1 year−1 (95% HPD; 2.43 × 10−7–4.38 × 10−7). The MRCAs for the entire CG23 population, CG23-I and equine sublineage nested within CG23-I were estimated to be 1878 (95% HPD; 1827–1915), 1928 (95% HPD; 1908–1953) and 1972 (95% HPD, 1961–1975), respectively.

----------

https://ja.wikipedia.org/wiki/創始者効果

人と物の移動により微生物が移動する

https://pubmed.ncbi.nlm.nih.gov/28912233/
Science
. 2017 Sep 15;357(6356):1099-1100. doi: 10.1126/science.aao3007.
Microbial mass movements
Yong-Guan Zhu 1, Michael Gillings 2, Pascal Simonet 3, Dov Stekel 4, Steve Banwart 5, Josep Penuelas 6 7
https://www.science.org/doi/10.1126/science.aao3007
Wastewater, tourism, and trade are moving microbes around the globe at an unprecedented scale

https://primate.or.jp/serialization/109%EF%BC%8Eブラジルでの小頭症は1個のアミノ酸が置換し/
ブラジルにウイルスが持ち込まれた経路については、2014年のサッカー・ワールドカップ（6月から7月）、その直後のリオデジャネイロでのカヌー競技（8月）、2013年6月のコンフェデレーション・カップが候補に挙げられていた。
ジカウイルスのゲノムでは、1年間に1000塩基中の1個の割合で変異が起きていると推定されている。この分子進化速度に基づいて計算すると、ウイルスがアメリカ大陸に持ち込まれた時期は、ワールドカップやカヌー競技よりも前で、コンフェデレーション・カップの時期に一致していた。しかし、この競技はポリネシアで最初のジカウイルス感染が確認される前に終わっていたため、このイベントとは関係ないと考えられた（1）。

https://pubmed.ncbi.nlm.nih.gov/27013429/
Science
. 2016 Apr 15;352(6283):345-349. doi: 10.1126/science.aaf5036. Epub 2016 Mar 24.
Zika virus in the Americas: Early epidemiological and genetic findings
https://science.sciencemag.org/content/352/6283/345.long
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4918795/
Fig. 3
Timescale of the introduction of ZIKV to the Americas.
There are two published hypotheses for how ZIKV came to be introduced into Brazil, during (i) the 2014 World Cup soccer tournament (Jun 12th - Jul 13th) (29) or (ii) the Va’a canoe event held in Rio de Janeiro between 12-17 Aug 2014 (34). Alternatively, introduction could have occurred during (iii) the 2013 Confederations Cup soccer tournament (15th–30th Jun 2013). Events (ii) and (iii) notably included competitors from French Polynesia. Our results suggest that the introduction of ZIKV to the Americas predated events (i) and (ii). Although the molecular clock dates are more consistent with the Confederations cup, that event ended before ZIKV cases were first reported in French Polynesia (4). Consequently, we believe that large-scale patterns in human mobility will provide more useful and testable hypotheses about viral introduction and emergence (33, 35, 36) than ad hoc hypotheses focused on specific events.

----------
## niid

- https://www.niid.go.jp/niid/en/
National Institute of Infectious Diseases
- https://www.niid.go.jp/niid/ja/
国立感染症研究所
- https://researchmap.jp/read0133866?lang=en
Masato SUZUKI
- https://twitter.com/KojiYahara
Group Leader @ National Institute of Infectious Diseases.  AMR surveillance, biomedical informatics, microbial genomics, evolution, epidemiology.

----------
## AMR
Antimicrobial Resistance
薬剤耐性

- https://twitter.com/AntibioticResis
- https://wwwnc.cdc.gov/eid/spotlight/antimicrobial-resistance
- 
- https://ja.wikipedia.org/wiki/抗生物質
- https://ja.wikipedia.org/wiki/コリスチン
- https://en.wikipedia.org/wiki/Colistin
- https://ja.wikipedia.org/wiki/カルバペネム系抗生物質
- https://ja.wikipedia.org/wiki/薬剤耐性
- https://ja.wikipedia.org/wiki/カルバペネム耐性腸内細菌
- https://en.wikipedia.org/wiki/Carbapenem-resistant_enterobacteriaceae (CRE)
- https://ja.wikipedia.org/wiki/Β-ラクタマーゼ
- https://en.wikipedia.org/wiki/Beta-lactamase
- https://www.kegg.jp/kegg/annotation/br01553.html
KEGG Antimicrobial Resistance: beta-Lactamase Genes

### who

https://pubmed.ncbi.nlm.nih.gov/40245910/
Lancet Infect Dis
. 2025 Apr 11:S1473-3099(25)00118-5. doi: 10.1016/S1473-3099(25)00118-5. Online ahead of print.
The WHO Bacterial Priority Pathogens List 2024: a prioritisation study to guide research, development, and public health strategies against antimicrobial resistance
; WHO Bacterial Priority Pathogens List Advisory Group
https://www.thelancet.com/journals/laninf/article/PIIS1473-3099(25)00118-5/fulltext

https://www.mhlw.go.jp/content/10906000/001309717.pdf
WHO Bacterial Priority Pathogens List 2024概要

2024年05月20日
https://japan-who.or.jp/news-releases/2405-40/
薬剤耐性菌のリストを更新 | 公益社団法人 日本ＷＨＯ協会
詳細はWHOのウエブサイトをご覧ください。
17 May 2024 
https://www.who.int/news/item/17-05-2024-who-updates-list-of-drug-resistant-bacteria-most-threatening-to-human-health
updated Bacterial Priority Pathogens List (BPPL) 2024
17 May 2024
https://www.who.int/publications/i/item/9789240093461

### 2022-02-16
### Waddington_2022

12:36 AM · Feb 17, 2022
https://twitter.com/Baker_Lab_Cam/status/1493972456465444871
Stephen Baker on Twitter: "New review out today. I think illumina sequencing didn’t exist when we started it, so been updated a few times. Glad it saw the light of day. Big up to waddo, christine B @meganecarey @EllenHigginson @vbalaji7 @GenomeMedicine https://t.co/EN1JqRPNk6" / Twitter


https://pubmed.ncbi.nlm.nih.gov/35172877/
Review Genome Med
. 2022 Feb 16;14(1):15. doi: 10.1186/s13073-022-01020-2.
Exploiting genomics to mitigate the public health impact of antimicrobial resistance
Claire Waddington 1 2, Megan E Carey 1 2, Christine J Boinett 3, Ellen Higginson 1 2, Balaji Veeraraghavan 4, Stephen Baker 5 6
https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-022-01020-2
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8849018/

Table 1
Use cases for whole-genome sequencing (WGS) in mitigating the public health impact of antimicrobial resistance (AMR)

Use case 7: Investigating colistin resistance detected in commensal E. coli in food stock animals in China [23]


AMR surveillance
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8849018/#Sec2title

International surveillance

National surveillance

The routine inclusion of WGS into national surveillance can also be used to evaluate the risk from emerging AMR threats, such as transferable colistin resistance [70]. Phenotypic colistin resistance testing is not routinely performed and therefore is not readily detectable without WGS [70]. Routine WGS of GNB in the UK generated a substantial data resource over time, which was mined to determine the extent of the spread of plasmid-encoded colistin resistance after it emerged as an AMR threat [70]. Although colistin resistance can be detected in organisms from a range of environments, this retrospective study found that it remained uncommon in the UK, leading to the monitoring of colistin prescribing in a bid to minimise any potential selective pressure within the UK.

Local surveillance

Community surveillance and outbreak investigation

Understanding the drivers of AMR
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8849018/#Sec7title

AMR and ‘One Health’

A WGS-based investigation of commensal E. coli isolates in livestock in China after a rapid increase of colistin resistance, for example, showed that this increase was due to the emergence of a plasmid associated colistin resistance gene in E. coli, designated MCR-1, and led to international efforts to control the dissemination of MCR-1 via food supply networks [23].


Limitations, challenges, and future directions
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8849018/#Sec11title

Technical limitations

Highly transmissible MGEs (such as plasmids) facilitate non-mutational AMR gene acquisition by HGT and are highly implicated in AMR dissemination [135]. 

Additionally, AMR genes are commonly associated with long, repetitive insertion sequences, and short-read sequencing techniques cannot resolve if these are carried on a plasmid or chromosomally located [38, 54, 140, 141]. Most outbreak investigations have largely assumed the relative conservation of plasmid structures over the limited time period of an outbreak and the extent to which HGT events contribute to outbreaks of AMR is not fully understood [54]. However, 

### 2021-05-10
### Shawa_2021

9:37 AM · May 18, 2021
https://twitter.com/hornistyf/status/1394452147090059267
Y. Furuta on Twitter: "学生の論文が出ました。 ザンビアで分離された薬剤耐性腸内細菌科細菌のゲノム解析を行い、複数の株で薬剤耐性遺伝子を含む領域(最長41 kb)がプラスミドから染色体に移っていることを確認しました。 
https://aricjournal.biomedcentral.com/articles/10.1186/s13756-021-00941-8
Novel chromosomal insertions of ISEcp1-blaCTX-M-15 and diverse antimicrobial resistance genes in Zambian clinical isolates of Enterobacter cloacae and Escherichia coli

https://pubmed.ncbi.nlm.nih.gov/33971966/
Antimicrob Resist Infect Control
. 2021 May 10;10(1):79. doi: 10.1186/s13756-021-00941-8.
Novel chromosomal insertions of ISEcp1-bla CTX-M-15 and diverse antimicrobial resistance genes in Zambian clinical isolates of Enterobacter cloacae and Escherichia coli
Misheck Shawa 1, Yoshikazu Furuta 1, Gillan Mulenga 2, Maron Mubanga 3, Evans Mulenga 3, Tuvshinzaya Zorigt 1, Christone Kaile 2, Manyando Simbotwe 4, Atmika Paudel 1, Bernard Hang'ombe 3, Hideaki Higashi 5
https://aricjournal.biomedcentral.com/articles/10.1186/s13756-021-00941-8
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8111917/

### 2021-04

https://pubmed.ncbi.nlm.nih.gov/33655229/
Lancet Microbe
. 2021 Apr;2(4):e135-e136. doi: 10.1016/S2666-5247(21)00039-2. Epub 2021 Feb 24.
COVID-19 drug practices risk antimicrobial resistance evolution
Ebrahim Afshinnekoo 1 2, Chandrima Bhattacharya 1 2, Ana Burguete-García 3, Eduardo Castro-Nallar 4, Youping Deng 5, Christelle Desnues 6, Emmanuel Dias-Neto 7, Eran Elhaik 8, Gregorio Iraola 9, Soojin Jang 10, Paweł P Łabaj 11, Christopher E Mason 1 2 12, Niranjan Nagarajan 13, Michael Poulsen 14, Bharath Prithiviraj 15, Rania Siam 16 17, Tieliu Shi 18, Haruo Suzuki 19, Johannes Werner 20, Maria Mercedes Zambrano 21, Malay Bhattacharyya 22, MetaSUB Consortium
https://www.thelancet.com/journals/lanmic/article/PIIS2666-5247(21)00039-2/fulltext
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7906697/


### 2021-04-06

https://pubmed.ncbi.nlm.nih.gov/32249837/
Sci Rep
. 2020 Apr 6;10(1):5970. doi: 10.1038/s41598-020-63167-5.
Massive analysis of 64,628 bacterial genomes to decipher water reservoir and origin of mobile colistin resistance genes: is there another role for these enzymes?
Mariem Ben Khedher 1, Sophie Alexandra Baron 1 2, Toilhata Riziki 1, Raymond Ruimy 3, Didier Raoult 1 2, Seydina M Diene 4 5, Jean-Marc Rolain 6 7
https://www.nature.com/articles/s41598-020-63167-5
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7136264/
The colistin resistance mechanism also involves genes encoding phosphoethanolamine transferase (PET) and/or glycosyltransferase proteins that are essential for membrane phospholipid biosynthesis and appear extremely ubiquitous due to their presence in all areas of life including bacteria, archaea and eukaryotes (plants, arthropods)12–14. 

Results
Homologous sequences of mobile colistin resistance genes in available database
As a starting point, the first MCR-1.1 protein described (NG_050417) was used as query in a BlastP analysis against the NCBI database to fish out a total of 13’658 protein hits with aa identity ranged from 30% to 100% and alignment ≥30% (Fig. 1A). Results include all MCR-variants (from MCR-2 to MCR-9) with aa identity ranged from 30.82% with MCR-4.1 to 82.66% with MCR-6.1 (Fig. 1B). 

Methods
Looking for homologous MCR-1 sequences from the NCBI database
Reference sequence of MCR-1 (AKF16168.1) was used as query for BlastP analysis against the NCBI database using as threshold e-value 10e-5. All homologous sequence with identity ≥30% and alignment ≥30% were kept for further analysis. Protein alignments were performed using Mafft tool (https://mafft.cbrc.jp/alignment/software/) and phylogenetic trees were inferred using neighbor-joining method in FastTree program (http://www.microbesonline.org/fasttree/) and visualized with FigTree v1.4.2 (http://tree.bio.ed.ac.uk/software/figtree/), 


https://pubmed.ncbi.nlm.nih.gov/29773743/
Review Science
. 2018 May 18;360(6390):733-738. doi: 10.1126/science.aar3777.
Genomic insights into the emergence and spread of antimicrobial-resistant bacterial pathogens
Stephen Baker 1 2 3, Nicholas Thomson 4 5, François-Xavier Weill 6, Kathryn E Holt 5 7
https://science.sciencemag.org/content/360/6390/733.long
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6510332/

- https://twitter.com/DrKatHolt
Kat Holt (@DrKatHolt) / Twitter
Puzzle solver, bacterial genome wrangler, antibiotic resistance fighter. Prof at Monash Uni & London School of Hygiene and Tropical Medicine. BSc, MEpi, PhD.

https://pubmed.ncbi.nlm.nih.gov/28233789/
Sci Rep
. 2017 Feb 24;7:43232. doi: 10.1038/srep43232.
In silico serine β-lactamases analysis reveals a huge potential resistome in environmental and pathogenic species
Christian Brandt 1 2, Sascha D Braun 2 3, Claudi
https://www.nature.com/articles/srep43232
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5324141/

### Pal_2015

https://pubmed.ncbi.nlm.nih.gov/26576951/
BMC Genomics
. 2015 Nov 17;16:964. doi: 10.1186/s12864-015-2153-5.
Co-occurrence of resistance genes to antibiotics, biocides and metals reveals novel insights into their co-selection potential
Chandan Pal 1, Johan Bengtsson-Palme 1, Erik Kristiansson 2, D G Joakim Larsson 3
https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-015-2153-5
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4650350/

----------
## plasmid
プラスミド

- https://github.com/haruosuz/plasmids/blob/master/references/plasmids.tools.md
- https://github.com/haruosuz/plasmids/blob/master/references/plasmid_resistance.md
- https://github.com/haruosuz/plasmids/blob/master/references/plasmid_metagenome.md
- https://github.com/haruosuz/plasmids/blob/master/references/plasmid_IncP1.md
- https://www.ofsf.or.jp/
公益財団法人　大隅基礎科学創成財団｜Ohsumi Frontier Science Foundation
- https://www.ofsf.or.jp/en/
Ohsumi Frontier Science Foundation

### 2021

- http://www.plasmidbiologysociety.org/plasmids-around-the-globe-2021/
Plasmids Around the Globe 2021 | International Society for Plasmid Biology

### Kirstahler_2021

https://pubmed.ncbi.nlm.nih.gov/34061588/
mSystems
. 2021 May 26;e0028321. doi: 10.1128/mSystems.00283-21. Online ahead of print.
A Peek into the Plasmidome of Global Sewage
Philipp Kirstahler 1, Frederik Teudt 1, Saria Otani 1, Frank M Aarestrup 1, Sünje Johanna Pamp 1

https://journals.asm.org/doi/10.1128/mSystems.00283-21
To distinguish between potential plasmid and nonplasmid contigs, we used a scheme described previously (21). The scheme contains Pfam identifiers highly specific for plasmids and viruses. Proteins with a plasmid replication initiator protein Rep_3 (PF01051) domain (n = 24,824) were investigated further, together with the full set of reference Rep_3 domain proteins (n = 1,637) downloaded from Pfam (version 33.1). The two data sets were combined and Rep_3 domain proteins with a length of <40 amino acid residues were discarded, resulting in a data set of 16,930 Rep_3 (PF01051) domain proteins. The protein sequences were aligned using MAFFT (version 7.221) as part of the Galaxy platform (46, 47). A phylogenetic tree was then built using FastTree (version 2.1.10) (48) and visualized using FigTree (version 1.4.4) (https://github.com/rambaut/figtree/releases).

5:47 PM · May 27, 2021
https://twitter.com/SJPamp/status/1397836852036620288
Sünje J. Pamp 🦠🧬🔬🌍 on Twitter: "NEW paper! Our recent study on plasmids &amp; antimicrobial resistance using @nanopore sequencing of sewage samples from around the globe🌎 is published now in @mSystems! 👉https://t.co/z2tv2PcrPf 🦠🧬" / Twitter

### Yang_2021

https://pubmed.ncbi.nlm.nih.gov/33721023/
Nucleic Acids Res
. 2021 Apr 19;49(7):3981-3996. doi: 10.1093/nar/gkab149.
A ProQ/FinO family protein involved in plasmid copy number control favours fitness of bacteria carrying mcr-1-bearing IncI2 plasmids
Jun Yang 1 2, Hai-Hong Wang 2 3, Yaoyao Lu 1 2, Ling-Xian Yi 1 2, Yinyue Deng 4, Luchao Lv 1 2, Vincent Burrus 5, Jian-Hua Liu 1 2
https://academic.oup.com/nar/article/49/7/3981/6171297
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8053102/
A putative ProQ/FinO family protein encoded by the IncI2 plasmid, designated as PcnR (plasmid copy number repressor), balances the mcr-1 expression and bacteria fitness by repressing the plasmid copy number.

Previous studies showed that plasmid-encoded specific regulators, such as H-NS or PsiB, alleviate the plasmid cost by controlling the expression of certain genes carried by the plasmid or host bacteria (12,30). 

### Wegrzyn_2021

https://pubmed.ncbi.nlm.nih.gov/33660784/
Nucleic Acids Res
. 2021 Apr 6;49(6):3394-3408. doi: 10.1093/nar/gkab113.
Defining a novel domain that provides an essential contribution to site-specific interaction of Rep protein with DNA
Katarzyna Wegrzyn 1, Elzbieta Zabrocka 1, Katarzyna Bury 1, Bartlomiej Tomiczek 1, Milosz Wieczor 2, Jacek Czub 2, Urszula Uciechowska 1, María Moreno-Del Alamo 3, Urszula Walkow 1, Igor Grochowina 1, Rafal Dutkiewicz 1, Janusz M Bujnicki 4 5, Rafael Giraldo 3, Igor Konieczny 1
https://academic.oup.com/nar/article/49/6/3394/6154473
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8034659/
The phylogenetic analysis revealed that the composition of this unique domain is typical within the described TrfA-like protein family. 

INTRODUCTION
The binding of proteins to nucleic acids is often accomplished via specific motifs. Many of these motifs (e.g. helix-turn-helix (HTH), zinc finger, leucine zipper) are present within protein domains that are directly engaged in interaction with nucleic acids (1). In proteins that initiate DNA replication, usually the HTH motif and its variant, the winged HTH (WH) motif, are present within the domain responsible for DNA binding.

Phylogenetic analysis and determination of sequence divergence

RESULTS

Interestingly, in some Rep proteins, we identified a short hydrophobic region located before the WH1 domain. This region shares a degree of similarity with a sequence within the N-terminal TrfA region (Figure ​(Figure1).1). We thus asked how the TrfA-like proteins are related to other replication initiators. To answer this question, we performed phylogenetic analysis of the representative DNA replication initiators using the maximum likelihood method (ML) (Supplementary Figure S1). Our ML analysis of the WH domains revealed a close monophyletic relationship among all TrfA-like proteins containing extended N-termini. Moreover, the phylogenetic analysis indicated that the TrfA-like proteins are most closely related to RepC, an IncQ-type replication protein C (51,52), and belong to a common protein family together with other plasmid replication initiation factors (Supplementary Figures S1, S2). 

### WASPS
https://archaea.i2bc.paris-saclay.fr/wasps/

https://pubmed.ncbi.nlm.nih.gov/31589313/
Bioinformatics
. 2020 Mar 1;36(5):1629-1631. doi: 10.1093/bioinformatics/btz745.
WASPS: web-assisted symbolic plasmid synteny server
Catherine Badel 1, Violette Da Cunha 1, Ryan Catchpole 1, Patrick Forterre 1 2, Jacques Oberto 1
https://academic.oup.com/bioinformatics/article/36/5/1629/5582648
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7703779/

### pMCR_1511

https://pubmed.ncbi.nlm.nih.gov/27895009/
Antimicrob Agents Chemother
. 2017 Jan 24;61(2):e02229-16. doi: 10.1128/AAC.02229-16. Print 2017 Feb.
IncP Plasmid Carrying Colistin Resistance Gene mcr-1 in Klebsiella pneumoniae from Hospital Sewage
Feifei Zhao 1 2, Yu Feng 1 2, Xiaoju Lü 1 2, Alan McNally 3, Zhiyong Zong 4 2 5
https://aac.asm.org/content/61/2/e02229-16.long
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5278755/

FIG 2
Genetic context of mcr-1 on pMCR_1511.

![](https://aac.asm.org/content/aac/61/2/e02229-16/F2.medium.gif)

The backbone of pMCR_1511 was highly similar (99% identity) to that of plasmid pHNFP671 (GenBank accession number KP324830), which was an IncP plasmid in E. coli isolate FP671 from Guangzhou, China but did not carry mcr-1. 

The sequence of pMCR_1511 has been deposited into DDBJ/EMBL/GenBank under accession number KX377410.

- https://www.ncbi.nlm.nih.gov/nuccore/KP324830
- https://www.ncbi.nlm.nih.gov/nuccore/KX377410

### pAKD4

https://pubmed.ncbi.nlm.nih.gov/20018208/
Comparative Study Plasmid
. 2010 Mar;63(2):98-107. doi: 10.1016/j.plasmid.2009.11.005. Epub 2009 Dec 16.
Comparative genomics of pAKD4, the prototype IncP-1delta plasmid with a complete backbone
Diya Sen 1, Hirokazu Yano, Haruo Suzuki, Jaroslaw E Król, Linda Rogers, Celeste J Brown, Eva M Top
https://www.sciencedirect.com/science/article/abs/pii/S0147619X09001267?via%3Dihub
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2819548/

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2819548/figure/F1/

Figure 1
Genetic map of pAKD4. 

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2819548/figure/F2/

Figure 2
Alignment of the IncP-1δ plasmids showing their evolutionary relationship. 

The 3′-end of the ssb gene, which is usually found upstream of trfA in most IncP-1 plasmids, is also missing in these plasmids. This gene codes for a single strand DNA-binding protein, which helps in DNA replication, recombination and repair. TrfA1 and Ssb proteins play a crucial role in plasmid replication in some bacterial hosts such as P. aeruginosa and may not be needed in some others such as P. putida and E. coli [32]. Therefore we postulate that IncP-1δ plasmids co-evolved with hosts that do not require TrfA1 and Ssb proteins and thereby contracted their host range. Further research is needed to test this hypothesis.

The complete sequence of pAKD4 has been deposited in the GenBank database under the accession number GQ983559.

- https://www.ncbi.nlm.nih.gov/nuccore/GQ983559

### Yano_2018_csbj

https://www.sciencedirect.com/science/article/pii/S2001037018301685
Reconsidering plasmid maintenance factors for computational plasmid design - ScienceDirect

Table 1
Lists of plasmids in different incompatibility groups.

IncY	P1 (Escherichia virus)	AF234172 (phage P1 mod749::IS5 c1–100)	RepA


Fig. 2. Replication module. (A) Replication module of IncP-1 plasmids. Ssb encodes a single-strand DNA binding protein.

2.3.2. G + C Content

Fig. 5. Plot of G + C contents of 209 plasmids and their host chromosomes. 
 The G + C contents of plasmids tend to be lower than (and are correlated with) those of the host chromosomes.

![](https://ars.els-cdn.com/content/image/1-s2.0-S2001037018301685-gr5.jpg)

----------

### Acinetobacter

### 2022-01-17

https://pubmed.ncbi.nlm.nih.gov/34839707/
Philos Trans R Soc Lond B Biol Sci
. 2022 Jan 17;377(1842):20200472. doi: 10.1098/rstb.2020.0472. Epub 2021 Nov 29.
What makes a megaplasmid?
James P J Hall 1, João Botelho 2 3, Adrian Cazares 4 5, David A Baltrus 6
https://royalsocietypublishing.org/doi/full/10.1098/rstb.2020.0472
https://pmc.ncbi.nlm.nih.gov/articles/PMC8628078/

1. Megaplasmids: very large plasmids

Figure 1.
Percentage of plasmid publications containing the term ‘megaplasmid’.

The recent availability of long-read single-molecule sequencing technologies, such as the PacBio SMRT protocol and Nanopore devices, enables megaplasmids to be unambiguously resolved [25–27], and with this increased detection rate, recent years have seen a steep increase in the proportion of plasmid publications containing the term ‘megaplasmid’ (figure 1).

2. How big is a megaplasmid?

Figure 2.
Plasmid size distributions for families with greater than 100 plasmids in the COMPASS database [13].

In the next sections, we discuss these features of megaplasmids—their size, genetics, and gene content—that explain their evolutionary and ecological role (figure 3).

Figure 3.
What makes a megaplasmid. 

3. Why are megaplasmids bigger?

4. Are megaplasmids just chromids-in-training?

5. Plasmid plasticity promotes megaplasmid emergence

6. Boxes: megaplasmids to watch

(c) Acinetobacter megaplasmids, global distribution, local adaptation: pR4WN-1BD1 (285 kb) and relatives

A role for local adaptation in shaping plasmid accessory gene content was discernible in these data, as plasmids from similar geographical locations had similar accessory genes, even if they were not closely related [27]. Increased sampling of non-clinical sites may be necessary to reveal the full megaplasmid picture in Acinetobacter, as an analysis of Acinetobacter plasmids from hospitals and soil/water suggested that larger plasmids were obtained from environmental samples [141].


### 2021-09-21

https://pubmed.ncbi.nlm.nih.gov/34621254/
Front Microbiol
. 2021 Sep 21:12:728644. doi: 10.3389/fmicb.2021.728644. eCollection 2021.
Ubiquitous Conjugative Mega-Plasmids of Acinetobacter Species and Their Role in Horizontal Transfer of Multi-Drug Resistance
Sofia Mindlin 1, Olga Maslova 1, Alexey Beletsky 2, Varvara Nurmukanova 1, Zhiyong Zong 3, Andrey Mardanov 2, Mayya Petrova 1
https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2021.728644/full
https://pmc.ncbi.nlm.nih.gov/articles/PMC8490738/

### 2020-12-02_Front_Microbiol

https://colil.dbcls.jp/browse/papers/33343549/1/100/year
  is cited by the following papers.
合計: 14, 

What makes a megaplasmid?

Ubiquitous Conjugative Mega-Plasmids of Acinetobacter Species and Their Role in Horizontal Transfer of Multi-Drug Resistance.

https://pubmed.ncbi.nlm.nih.gov/33343549/
Front Microbiol
. 2020 Dec 2;11:605952. doi: 10.3389/fmicb.2020.605952. eCollection 2020.
A Novel Family of Acinetobacter Mega-Plasmids Are Disseminating Multi-Drug Resistance Across the Globe While Acquiring Location-Specific Accessory Genes
Timothy M Ghaly 1, Ian T Paulsen 2 3, Ammara Sajjad 1, Sasha G Tetu 2 3, Michael R Gillings 1 3
https://www.frontiersin.org/articles/10.3389/fmicb.2020.605952/full
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7738440/

Materials and Methods

Plasmid Pan‐ and Core-Genome Analyses
Gene and protein sequences for all 21 plasmids were predicted using Prokka v1.13 (parameters: -- kingdom Bacteria). Plasmid pan‐ and core-genome analyses were carried out using Perl scripts available in the GET_HOMOLOGUES v20092018 software package (

Phylogenetic Analysis
The evolutionary relationship between plasmids was inferred from the plasmid core-genome. Appropriate phylogenetic loci were identified using the GET_PHYLOMARKERS software package (Vinuesa et al., 2018) with default parameters. The GET_PHYLOMARKERS uses the single-copy plasmid core-genome clusters identified by GET_HOMOLOGUES and removes recombinant sequences as well as loci that produce outlier phylogenies. 

Figure 3
Plasmid core-genome phylogram and accessory genome dendrogram. (left) A maximum-likelihood tree showing the inferred plasmid phylogeny based on the plasmid core-genome and (right) a maximum-likelihood tree based on the presence and absence of all 1,262 accessory genes in the plasmid pangenome. 

![https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7738440/figure/fig3/](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7738440/bin/fmicb-11-605952-g003.jpg)

Figure 4
A co-phylogeny linking the host strain (left) and plasmid (right) phylogenies.

![https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7738440/figure/fig4/](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7738440/bin/fmicb-11-605952-g004.jpg)


----------
## CRISPR
clustered regularly interspaced short palindromic repeat; クリスパー

https://en.wikipedia.org/wiki/CRISPR

### 2022-01-13

https://pubmed.ncbi.nlm.nih.gov/35025885/
PLoS Biol
. 2022 Jan 13;20(1):e3001514. doi: 10.1371/journal.pbio.3001514. eCollection 2022 Jan.
Microbial defenses against mobile genetic elements and viruses: Who defends whom from what?
Eduardo P C Rocha 1, David Bikard 2
https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3001514
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8791490/

Box 1. Glossary.

Clustered regularly interspaced short palindromic repeats/CRISPR associated protein (CRISPR/Cas) systems: provide adaptive immunity based on spacer sequences within CRISPR arrays. These sequences guide nucleases to destroy cognate invading genetic elements.

The resulting DNA can be recovered by other defense systems, like clustered regularly interspaced short palindromic repeats/CRISPR associated protein (CRISPR/Cas) or the Argonaut, to build more specific defenses against subsequent infections by the same phages [30,31].

Fig 1
Examples of defense systems discussed in the text.
Homologous recombination via the RecBCD pathway can defend from MGEs because RecBCD is a powerful exonuclease that degrades linear double-stranded DNA (but is inactivated by some phage encoded proteins, like Gam). Epigenetic modifications are the basis of many defense systems that can be counteracted by phage proteins, like Ocr in phage T7, or by epigenetic modification of phage DNA. Abortive infection is a costly defense strategy that induces cell growth arrest or death when a threat to the cell is detected. Adaptive immunity by CRISPR/Cas systems targets MGEs with specific DNA sequences, while allowing other exogenous DNA to remain in the cell and eventually recombine with the chromosome. CRISPR/Cas, clustered regularly interspaced short palindromic repeats/CRISPR associated protein; MGE, mobile genetic element.


Fig 2
Defense and antidefense systems are often studied in the context of the interaction between one host and one MGE, usually a virulent phage (left). Yet, the presence of numerous MGEs in populations and their ability to encode their own defense systems renders the picture more complex (right). 


Why are there so many defense systems in each genome?
Most bacterial genomes encode several R–Ms, but often also CRISPR/Cas, retrons, and many other defense systems [35]. For example, the first 2 sequenced genomes of Helicobacter pylori encode a total of more than 20 putative R–M systems [52], and genomes with multiple CRISPR arrays and Cas systems are frequent [53].


### 2021-06-24
10:58 AM · May 27, 2021
https://twitter.com/mason_lab/status/1397733925057970177
Chris Mason on Twitter: "I'm elated to release four @metasub papers today, with new species, data interfaces, &amp; online resources. The first is in @CellCellPress, "A global metagenomic map of urban microbiomes and antimicrobial resistance" led by @dcdanko @Daniela_Bezdan https://t.co/47SB0Hda05" a 🧵 /1" / Twitter
https://twitter.com/mason_lab/status/1397733926278402053
Chris Mason on X: "Our metagenome atlas spans ~5,000 samples from mass-transit systems in 60 cities over 3 years, showing thousands of species, antimicrobial resistance (AMR) markers &amp; novel biology everywhere: 10,928 new viruses, 1,302 new bacteria, 2 new archaea, and 838,532 new CRISPR arrays /2" / X
https://twitter.com/mason_lab/status/1397733959317049344
Chris Mason on X: "then, with @omarabudayyeh and @jgooten, we mined the metagenome-assembled genomes (MAGs) to find &gt;800K new CRISPR arrays, which showed the ongoing dynamics of bacteria battling their ever-evolving viruses and some unknown CRISPR types /12 https://t.co/6DkLeHOaY9" / X

### 2021-06-24
https://pubmed.ncbi.nlm.nih.gov/34043940/
Cell
. 2021 Jun 24;184(13):3376-3393.e17. doi: 10.1016/j.cell.2021.05.002. Epub 2021 May 26.
A global metagenomic map of urban microbiomes and antimicrobial resistance
David Danko 1, Daniela Bezdan 2,
International MetaSUB Consortium
https://www.sciencedirect.com/science/article/pii/S0092867421005857?via%3Dihub
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8238498/
https://www.cell.com/cell/fulltext/S0092-8674(21)00585-7
Figure 5
Newly observed genetic sequences
(C) Mapping rate of CRISPR spacers from MetaSUB data to viral genomes in RefSeq and viral genomes found in MetaSUB data.
(E and F) Fractional breakdowns of identifiable CRISPR systems found in the MetaSUB data.
From these MAGs, we identified 838,532 CRISPR arrays, of which 3,245 could be annotated for specific CRISPR systems. The annotated CRISPR arrays were mostly type 1-E and 1-F, but a number of type II and III systems were identified as well (Figures 5E and 5F), and some arrays had unclear (ambiguous) type assignment. Critically, when we aligned spacers to both our viral MAGs and all viral sequences in RefSeq, the spacers in our identified CRISPR arrays closely matched our predicted MAG-derived viruses. 

### 2020-11-06
https://pubmed.ncbi.nlm.nih.gov/33159058/
Nat Commun
. 2020 Nov 6;11(1):5652. doi: 10.1038/s41467-020-19415-3.
Discovery of multiple anti-CRISPRs highlights anti-defense gene clustering in mobile genetic elements
Rafael Pinilla-Redondo 1 2 3, Saadlee Shehreen 4, Nicole D Marino 2, Robert D Fagerlund 5 6, Chris M Brown 4 6, Søren J Sørensen 1, Peter C Fineran 7 8 9, Joseph Bondy-Denomy 10 11 12
https://www.nature.com/articles/s41467-020-19415-3
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7648647/

helix-turn-helix (HTH)

Methods
Bioinformatic searches and phylogenetic analyses of Aca and Acr proteins

Software and statistical analysis
Numerical data were analyzed and plotted with GraphPad Prism 6.0 Software. HHPred searches were carried out for the prediction of protein domains (e.g., HTH). 

### 2020-01-24
9:17 PM · Nov 21, 2019
https://twitter.com/epcrocha/status/1197489104507883521
erocha on Twitter: "If one CRISPR-Cas is great, why have two ? Or more? Or combinations of them? Atypical organizations and epistatic interactions of CRISPRs and cas clusters in genomes and their mobile genetic elements: https://t.co/fBjMLKXX1d https://t.co/Bv7dWNAgOt"
https://pubmed.ncbi.nlm.nih.gov/31745554/
Nucleic Acids Res
. 2020 Jan 24;48(2):748-760. doi: 10.1093/nar/gkz1091.
Atypical organizations and epistatic interactions of CRISPRs and cas clusters in genomes and their mobile genetic elements
Aude Bernheim 1 2 3 4, David Bikard 2, Marie Touchon 1, Eduardo P C Rocha 1
https://academic.oup.com/nar/article/48/2/748/5634034
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7145637/
Phages and 25 000 prophages were almost devoid of CRISPR–Cas systems, whereas 3% of plasmids had CRISPR–Cas systems or isolated CRISPRs. The latter were often compatible with the chromosomal cas clusters, suggesting that plasmids can co-opt the latter.
ファージと25000プロファージにはCRISPR-Casシステムがほとんど存在しなかったが、プラスミドの3％にはCRISPR-Casシステムが存在するか、孤立したCRISPRが存在していた。後者はしばしば染色体上のcasクラスターと互換性があり、プラスミドが後者を共同利用できることを示唆していた。

### 2019-11
https://pubmed.ncbi.nlm.nih.gov/31626589/
Microb Genom
. 2019 Nov;5(11):e000309. doi: 10.1099/mgen.0.000309.
Pangenome of Acinetobacter baumannii uncovers two groups of genomes, one of them with genes involved in CRISPR/Cas defence systems associated with the absence of plasmids and exclusive genes for biofilm formation
Eugenio L Mangas 1, Alejandro Rubio 1, Rocío Álvarez-Marín 2, Gema Labrador-Herrera 2, Jerónimo Pachón 3 2, María Eugenia Pachón-Ibáñez 3 2, Federico Divina 4, Antonio J Pérez-Pulido 1
https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000309#tab2
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6927304/
One of these groups rarely has plasmids, and bears clustered regularly interspaced short palindromic repeat (CRISPR) sequences, in addition to CRISPR-associated genes (cas genes) or restriction-modification system genes. This fact strongly supports the lack of plasmids. Furthermore, the scarce plasmids in this group also bear CRISPR sequences, and specifically contain genes involved in prokaryotic toxin-antitoxin systems that could either act as the still little known CRISPR type IV system or be the precursors of other novel CRISPR/Cas systems. In addition, a limited set of strains present a new cas9-like gene, which may complement the other cas genes in inhibiting the entrance of new plasmids into the bacteria. 
そのうちの1つのグループは、プラスミドをほとんど持たず、CRISPR関連遺伝子（cas遺伝子）や制限修飾系遺伝子に加えて、CRISPR（clustered regular interspaced short palindromic repeat）配列を持っています。この事実は、プラスミドの欠如を強く裏付けるものである。さらに、このグループの希少なプラスミドにもCRISPR配列があり、原核生物の毒素・抗毒素システムに関与する遺伝子が特異的に含まれていることから、まだあまり知られていないCRISPRタイプIVシステムとして作用するか、他の新規CRISPRCasシステムの前駆体となる可能性がある。さらに、限られた菌株には新しいcas9様遺伝子が存在し、他のcas遺伝子を補完して新たなプラスミドの菌体内への侵入を抑制している可能性がある。

### 2019-08
https://pubmed.ncbi.nlm.nih.gov/31165781/
Nat Rev Microbiol
. 2019 Aug;17(8):513-525. doi: 10.1038/s41579-019-0204-7.
CRISPR-Cas in mobile genetic elements: counter-defence and beyond
Guilhem Faure 1 2, Sergey A Shmakov 1 3, Winston X Yan 4, David R Cheng 4, David A Scott 4, Joseph E Peters 5, Kira S Makarova 1, Eugene V Koonin 6

### 2016-08
### p7ME01
https://github.com/haruosuz/plasmids/blob/master/references/plasmid_IncP1.md#p7me01
https://pubmed.ncbi.nlm.nih.gov/27345842/
Environ Microbiol Rep
. 2016 Aug;8(4):508-19. doi: 10.1111/1758-2229.12424. Epub 2016 Jun 27.
Analysis of defence systems and a conjugative IncP-1 plasmid in the marine polyaromatic hydrocarbons-degrading bacterium Cycloclasticus sp. 78-ME
Michail M Yakimov 1, Francesca Crisafi 1, Enzo Messina 1, Francesco Smedile 1, Anna Lopatina 2, Renata Denaro 1, Dietmar H Pieper 3, Peter N Golyshin 4, Laura Giuliano 1
https://sfamjournals.onlinelibrary.wiley.com/doi/full/10.1111/1758-2229.12424
Get access to the full version of this article.
https://search.lib.keio.ac.jp/permalink/81SOKEI_KEIO/uccs31/cdi_crossref_primary_10_1111_1758_2229_12424
Two ‘defence islands’ were identified in its genome: one contained CRISPR-Cas and toxin-antitoxin system, while the second was composed of an array of genes for toxin-antitoxin and restriction-modification proteins. Among 116 spacers of CRISPR-Cas system only seven spacers matched phages and plasmids. Additionally, we showed that Cycloclasticus sp. 78-ME harbours a conjugative plasmid p7ME01 of a new IncP-1θ ancestral archetype, which likely suppresses the acquisition of exogenous DNA by this organism. Based on this finding, we suggested that the adaptive immunity of Cycloclasticus sp. 78-ME is linked with the acquisition of this plasmid.

### 2020-06-10
https://pubmed.ncbi.nlm.nih.gov/32523110/
Nat Commun
. 2020 Jun 10;11(1):2934. doi: 10.1038/s41467-020-16731-6.
Development of CRISPR-Cas13a-based antimicrobials capable of sequence-specific killing of target bacteria
https://www.nature.com/articles/s41467-020-16731-6
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7287087/
https://www.amed.go.jp/news/release_20200610-02.html
狙った細菌を選択的に殺菌できる遺伝子標的型抗菌薬を創出―薬剤耐性菌問題の解決へ― | 国立研究開発法人日本医療研究開発機構

https://ja.wikipedia.org/wiki/CRISPR
CRISPR (英: clustered regularly interspaced short palindromic repeat; クリスパー)は数十塩基対の短い反復配列を含み、原核生物における一種の獲得免疫系として働く座位である。配列決定された原核生物のうち真正細菌の4割と古細菌の9割に見出されており[1][2]、プラスミドやファージといった外来の遺伝性因子に対する抵抗性に寄与している[3][4]。

さらにCRISPR-Casシステムには抗菌作用も存在し、DNA編集を行うCas9を使用した場合にはプラスミド上に遺伝子があると適切な殺菌ができず、細菌の変異をもたらすリスクがある一方、RNA編集を行うCas13aを使用する場合はそのような懸念がなく薬剤耐性菌を標的とした増殖の抑制に臨床上使いやすいと期待する研究もある[22][23][24]。

リピートとスペーサー
CRISPRリピートの要素は24から48塩基対で[43]通常は回文配列を含み、stem-loopヘアピン構造を形成する[44]。このリピート要素の間に似たような長さのスペーサーが介在している[43]。スペーサーの配列はプラスミドやファージの持つ配列と同一であったり[8][9][10]、あるいは自身のゲノム中の配列と同一(self-targeting spacers)であったりする[45]。ファージが感染すると、速やかに新たなスペーサーが追加される[46]。

https://www.cosmobio.co.jp/product/detail/crispr-cas.asp?entry_id=14354
特集：CRISPR-Cas9 とは DNA二本鎖を切断してゲノム配列の任意の場所を削除、置換、挿入することができる新しい遺伝子改変技術 | DNA二本鎖を切断してゲノム配列の任意の場所を削除、置換、挿入することができる新しい遺伝子改変技術 | コスモ・バイオ株式会社
CRISPR-Cas9システムは、細菌や古細菌においてウイルスやプラスミドといった遺伝的要素の侵入物を標的し、排除するよう進化した適応免疫の一つです。


https://www.tmd.ac.jp/grad/bac/CRISPR.html
京都大学　微生物感染症学分野
CRISPRとはClustered Regularly Interspaced Short Palindromic Repeatsの略で、近年原核生物でファージやプラスミドに対する獲得免疫機構として機能していることが判明したDNA領域のことを指します。


https://crisp-bio.blog.jp/archives/18437923.html
CRISPRメモ_2019/06/21 (3件) : crisp_bio

1. CRISPR/Cas9システムにより、コリスチン耐性大腸菌臨床分離株からmcr-1遺伝子を帯びたプラスミドを除去
[出典] "Eliminating mcr-1-harbouring plasmids in clinical isolates using the CRISPR/Cas9 system" Wang P [..] Wang X. J Antimicrob Chemother 2019-06-15. 

https://crisp-bio.blog.jp/archives/12546975.html
CRISPRメモ_2018/09/30 (10件) : crisp_bio

2. CRISPR/Cas9システムを帯びた接合性プラスミドでコリスチン耐性遺伝子を除去し多剤耐性菌に打ち克つ
[出典] "Exploiting a conjugative CRISPR/Cas9 system to eliminate plasmid harboring mcr-1 gene from Escherichia coli" Dong H, Xiang H, Mu D, Wang D, Wang T. Int J Antimicrob Agents. 2018 Sep 26.

3. CRISPR生物学を巡る１０の謎
[出典] Comment "Open questions: CRISPR biology" Koonin EV. BMC Biology. 2018 Sep 24.
投げかけられた謎は、CRISPR/Casシステム自体およびクラスとタイプの偏在とその由来、CRISPR-Cas活性と遺伝子水平伝播 (HGT)のバランス、CRISPR-Casと細胞死/休眠の関係、免疫応答以外の機能、病原性との関係、トランスポゾンやプラスミドにコードされているCRISPR-Casシステムの機能、CRISPR-Casの起源など10項目
https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-018-0565-9
What are the functions of CRISPR-Cas systems encoded by transposons and plasmids?
Apart from archaeal and bacterial genomes, CRISPR-Cas systems are encoded by a large family of Tn7-like transposons [9] and by various plasmids [2]. All transposon-encoded and some of the plasmid-encoded CRISPR-Cas are “minimal” variants that lack the enzymes involved in target cleavage and hence are predicted to be involved in non-defense roles. The nature of these functions remains to be elucidated, and might involve RNA-guided transposition [9].


2015
https://seikagaku.jbsoc.or.jp/10.14952/SEIKAGAKU.2015.870686/data/index.html
CRISPR-Cas9の構造と機能
2. CRISPR-Cas9
原核生物のゲノムにはCRISPRアレイとCasオペロンから構成されるCRISPR領域が存在する16)．CRISPRアレイは外来核酸に由来する塩基配列（スペーサー配列）とリピート配列からなる．細菌に感染した外来核酸はCas1–Cas2複合体により断片化され新たなスペーサー配列としてCRISPRアレイに取り込まれる17)．


----------


----------
## monumentome

### 2023-09-22

https://pubmed.ncbi.nlm.nih.gov/37808297/
Front Microbiol
. 2023 Sep 22:14:1253371. doi: 10.3389/fmicb.2023.1253371. eCollection 2023.
The microbial biodiversity at the archeological site of Tel Megiddo (Israel)
Yali Zhang 1, S Emil Ruff 2, Nikolay Oskolkov 3, Braden T Tierney 4, Krista Ryon 4, David Danko 4, Christopher E Mason 5 6 7 8, Eran Elhaik 1
https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2023.1253371/full
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10559971/
Data availability statement
The data presented in the study are deposited in the GeoSeeq.com repository, accession name: Monumentome Project available at: https://portal.geoseeq.com/sample-groups/0c919d45-410e-4edc-8d0b-1529f6516726.


### 2021-12-08

Dec. 8, 2021
AUTHOR: MADELINE BARRON, PH.D.
https://asm.org/Articles/2021/December/Microbes-as-Enemies-and-Allies-in-the-World-of-Art
Microbes as Enemies and Allies in the World of Art Conservation

----------

## money
お金 お札 ATM
Jane M Carlton

### money.jp

2016/02/02
http://fukuyuki.net/post-639/
硬貨が汚いという迷信について調べた結果。。。 | ふくゆきブログ

2015/06/26
https://www.narinari.com/Nd/20150632327.html
お金の触りすぎ原因で性病に、業務多忙で手を洗わなかった女性銀行員。 | Narinari.com
香港城市大学・細菌学研究センターによる調査で、アジアで流通する紙幣の中で“最も汚い”という、不名誉ならく印を押されたこともある中国の人民元紙幣。先日、中国の銀行で働くある女性は、原因不明の性病に悩まされることになったのだが、その原因はなんと“汚いお札”の触りすぎにあったそうだ。
代表的なウイルス性の性感染症である尖圭コンジローマと診断された。

2013-08-12
2013年8月12日(月)
https://www.recordchina.co.jp/b75402-s0-c30-d0035.html
アジアで最も汚く雑菌まみれのお札、それは人民元―台湾
香港城市大学細菌学研究センターが香港、中国、インドなどアジア7地域の紙幣に付着した細菌を調べたところ、中国の人民元が17万8000個と最多。2位の香港、カンボジア紙幣（各1万個）を大きく上回った。人民元のなかでも特に流通の頻繁な1元札の細菌が最も多く、1枚におよそ180万個の細菌がついていることも。

2013.06.19
https://www.atpress.ne.jp/news/36512
現金の清潔度調査を実施～日常的に使用する紙幣と硬貨の汚染細菌付着実態が判明～｜株式会社衛生微生物研究センターのプレスリリース

2014-04-21
12:08 PM · Apr 21, 2014
https://twitter.com/WSJJapan/status/458079813899517952
ウォール・ストリート・ジャーナル日本版 on X: "米ニューヨーク大学の科学者たちは、紙幣には驚くほどの数のバクテリアが住んでいることを発見した⇒「口にお金を入れる」のはやめよう―紙幣に膨大なバクテリア http://t.co/KaSEvhLBRr http://t.co/nqJEXMN3l3" / X

### 2021-01-27

https://pubmed.ncbi.nlm.nih.gov/33585735/
ACS Omega
. 2021 Jan 27;6(5):3499-3507. doi: 10.1021/acsomega.0c04546. eCollection 2021 Feb 9.
Metagenomic Sequencing Revealed the Potential Pathogenic Threats of Banknotes
Jun Lin 1 2 3 4, Wenqian Jiang 1 3, Yang Shi 1 3, Weiwen Cai 1 3
https://pubs.acs.org/doi/10.1021/acsomega.0c04546
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7876676/

### 2017-04-13

6:55 AM · May 23, 2017
https://twitter.com/Newsweek_JAPAN/status/866774474166874113
ニューズウィーク日本版 on X: "細菌の感染ルートを探るには、お札を追え！ 手のひらや地下鉄構内の空気より、紙幣の表面にはたくさんの最近が棲んでいることがわかった。耐性菌対策にもなる発見だ #細菌 #耐性菌 #感染症 https://t.co/GpcxIw3YPJ … https://t.co/dZpLZTSzjs" / X

2017年5月22日（月）
https://www.newsweekjapan.jp/stories/world/2017/05/post-7651.php
細菌の感染ルートを探るには、お札を追え！｜ニューズウィーク日本版 オフィシャルサイト
＜香港の研究で、手の平や地下鉄構内の空気より、紙幣の表面に細菌はたくさん棲み、しかも長生きしていることがわかった＞
香港大学の研究チームは、紙幣に付着した細菌の生存期間を調べるため、香港市内にある12の医療機関と3つの地下鉄駅から1枚ずつ、合わせて15枚の香港ドル紙幣を回収した。
スイスの科学誌「フロンティアズ・イン・マイクロバイオロジー」に掲載

紙幣が細菌追跡の目印に
次のページ都市の細菌集団探しは成長分野
https://www.newsweekjapan.jp/stories/world/2017/05/post-7651_2.php
耐性菌との戦いに役立つ

11:06 AM · Apr 14, 2017
https://twitter.com/yokadzaki/status/852704490436177921
Yusuke Okazaki on X: "お札のメタゲノム。サンプリングは地下鉄と病院のレジ。しかもアンプリコンでなくHiseqショットガン。微生物以外のDNAの分析も面白そうだけどこのあとやるのかな　https://t.co/wb3MCleJLc" / X

https://pubmed.ncbi.nlm.nih.gov/28450856/
Front Microbiol
. 2017 Apr 13:8:632. doi: 10.3389/fmicb.2017.00632. eCollection 2017.
Toward a Metagenomic Understanding on the Bacterial Composition and Resistome in Hong Kong Banknotes
Yoshitaro Heshiki 1, Thrimendra Dissanayake 1, Tingting Zheng 1, Kang Kang 1, Ni Yueqiong 1, Zeling Xu 2, Chinmoy Sarkar 3, Patrick C Y Woo 4 5 6 7 8, Billy K C Chow 2, David Baker 2, Aixin Yan 2, Christopher J Webster 3, Gianni Panagiotou 1 9, Jun Li 1
https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2017.00632/full
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5389987/

### 2017-04-06

https://pubmed.ncbi.nlm.nih.gov/28384336/
PLoS One
. 2017 Apr 6;12(4):e0175527. doi: 10.1371/journal.pone.0175527. eCollection 2017.
Filthy lucre: A metagenomic pilot study of microbes found on circulating currency in New York City
Julia M Maritz 1, Steven A Sullivan 1, Robert J Prill 2, Emre Aksoy 1, Paul Scheid 1, Jane M Carlton 1
https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0175527
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5383295/
In this pilot study, we swabbed circulating $1 bills obtained from a New York City bank in February (Winter) and June (Summer) 2013 and used shotgun metagenomic sequencing to profile the communities found on their surface.

SEPTEMBER 6, 2017
https://time.com/4918626/money-germs-microbes-dirty/
Here's How Dirty Your Money Really Is | TIME

In a 2017 study published in the journal PLOS ONE, researchers swabbed $1 bills from a bank in New York City to see what was living on paper currency.

Cash is also often streaked with drugs. In a study of 10 one-dollar bills from cities across the country, nearly 80% of them had traces of cocaine.
https://academic.oup.com/jat/article/20/4/213/838490
Cocaine Contamination of United States Paper Currency | Journal of Analytical Toxicology | Oxford Academic

Jan 04, 2017
https://money.com/money-cash-currency-bacteria-disease-sickness/
Does Paper Money Spread Disease? U.S. Cash Holds Bacteria | Money
The Money in Your Wallet Might Be Covered With Poop, Mold, and Cocaine

https://pubmed.ncbi.nlm.nih.gov/11566423/
Forensic Sci Int
. 2001 Oct 1;121(3):189-93. doi: 10.1016/s0379-0738(01)00401-7.
Drug contamination of US paper currency
A J Jenkins 1

2014-04-24
April 24, 2014
https://www.cbsnews.com/news/dirty-money-your-cash-is-home-to-thousands-of-bacteria/
Dirty Money: Your cash is home to thousands of bacteria - CBS News
The unpublished study examined 80 circulating $1 bills in New York City between two periods - winter (February 2013) and summer (July 2013).

2014/05/22(木) 22:33
https://girlschannel.net/topics/140416/
お金を触ったら手洗い必須かも！「お札には3,000ものバクテリアが」…((；ﾟДﾟ)) | ガールズちゃんねる - Girls Channel -
2014/05/18
https://news.nicovideo.jp/watch/nw1072131
お金を触ったら手洗い必須かも「お札には3,000ものバクテリアが」―研究結果 | ニコニコニュース
APR 30, 2014
You won’t believe how much bacteria is on a dollar bill
http://www.womenshealthmag.com/health/bacteria-on-money
researchers collected 20 $1 bills in both February and July 2013 (because certain microbes may be more prevalent depending on the season).

### 2016-11-16

https://pubmed.ncbi.nlm.nih.gov/27904880/
mSphere
. 2016 Nov 16;1(6):e00226-16. doi: 10.1128/mSphere.00226-16. eCollection 2016 Nov-Dec.
Microbial Community Patterns Associated with Automated Teller Machine Keypads in New York City
Holly M Bik 1, Julia M Maritz 1, Albert Luong 2, Hakdong Shin 2, Maria Gloria Dominguez-Bello 2, Jane M Carlton 1
https://journals.asm.org/doi/10.1128/msphere.00226-16
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5112336/

11/21/2016
https://www.dailysunny.com/2016/11/21/nynews161121-2/
菌まみれのＡＴＭ ２年の調査で数十種類を検出 | DAILYSUN NEW YORK

### 2015-06-08

https://pubmed.ncbi.nlm.nih.gov/26035208/
PLoS One
. 2015 Jun 2;10(6):e0128711. doi: 10.1371/journal.pone.0128711. eCollection 2015.
Screening currency notes for microbial pathogens and antibiotic resistance genes using a shotgun metagenomic approach
Saakshi Jalali 1, Samantha Kohli 2, Chitra Latka 3, Sugandha Bhatia 4, Shamsudheen Karuthedath Vellarikal 5, Sridhar Sivasubbu 5, Vinod Scaria 1, Srinivasan Ramachandran 1
https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0128711
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4452720/

https://www.genomeweb.com/scan/week-plos-31
This Week in PLOS: Jun 8, 2015
In PLOS One, researchers from India use shotgun metagenomic sequencing to look at the microbes found on paper currency in that country. By sequencing DNA from bills deemed most and least circulated from Reserve Bank of India data, the team detected sequences coinciding with dozens of antibiotic resistance genes, as well as sequences believed to represent pathogenic and cellulose degrading organisms. "Our analysis suggests a significant diversity in the microbial population on paper currency notes," study authors write, "and presence of antibiotic resistance genes."

### 2014-02-26

https://pubmed.ncbi.nlm.nih.gov/24571076/
Review Future Microbiol
. 2014;9(2):249-61. doi: 10.2217/fmb.13.161.
Paper money and coins as potential vectors of transmissible disease
Emmanouil Angelakis 1, Esam I Azhar, Fehmida Bibi, Muhammad Yasir, Ahmed K Al-Ghamdi, Ahmad M Ashshi, Adel G Elshemi, Didier Raoult
https://www.futuremedicine.com/doi/10.2217/fmb.13.161
26 Feb 2014

### 2013-08-28

https://pubmed.ncbi.nlm.nih.gov/23985137/
Antimicrob Resist Infect Control
. 2013 Aug 28;2(1):22. doi: 10.1186/2047-2994-2-22.
Money and transmission of bacteria
Habip Gedik 1, Timothy A Voss, Andreas Voss
https://aricjournal.biomedcentral.com/articles/10.1186/2047-2994-2-22
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3765964/
Euro, US Dollar, Canadian Dollar, Croatian Luna, Romanian Leu (RON), Moroccan Dirham, and Indian Rupee banknotes were included into the first part of the study. Cultures of the Romanian Leu yielded all 3 multi-drug resistant pathogens; MRSA, VRE and ESBL-producing E. coli. 

----------
## MRSA

https://ja.wikipedia.org/wiki/メチシリン耐性黄色ブドウ球菌
methicillin-resistant Staphylococcus aureus

2024年3月26日
https://www.niid.go.jp/niid/ja/typhi-m/iasr-reference/2637-related-articles/related-articles-529/12583-529r07.html
国内外の家畜関連MRSA（LA-MRSA）の状況について
医療関連型メチシリン耐性黄色ブドウ球菌（healthcare-associated methicillin-resistant Staphylococcus aureus: HA-MRSA）, 市中獲得型MRSA（community-acquired MRSA: CA-MRSA）とは, 遺伝子的性状などが異なるMRSAが分離され, 家畜関連MRSA（livestock-associated MRSA: LA-MRSA）

https://x.com/H_Nakaminami
中南 秀将 (@H_Nakaminami) / X

https://www.jstage.jst.go.jp/article/jsei/37/5/37_164/_article/-char/ja/
総説
本邦におけるメチシリン耐性黄色ブドウ球菌（MRSA）の流行型の変化
中南 秀将
https://www.jstage.jst.go.jp/article/jsei/37/5/37_164/_pdf/-char/ja
2022年7月6日
http://journal.kyorin.co.jp/journal/kankyokansen/detail_j.php?-DB=kankyokansen&-recid=330&-action=browse
日本環境感染学会：
─総説─
本邦におけるメチシリン耐性黄色ブドウ球菌（MRSA）の流行型の変化
中南　秀将
http://www.kankyokansen.org/journal/full/03705/037050164.pdf
一般的に，CA-MRSA の方が HA-MRSAよりも増殖速度が速い２０）
20）Okuma K, 
https://pubmed.ncbi.nlm.nih.gov/12409412/
J Clin Microbiol
. 2002 Nov;40(11):4289-94. doi: 10.1128/JCM.40.11.4289-4294.2002.
Dissemination of new methicillin-resistant Staphylococcus aureus clones in the community
Keiko Okuma 1, Kozue Iwakawa, John D Turnidge, Warren B Grubb, Jan M Bell, Frances G O'Brien, Geoffrey W Coombs, John W Pearman, Fred C Tenover, Maria Kapi, Chuntima Tiensasitorn, Teruyo Ito, Keiichi Hiramatsu
https://pmc.ncbi.nlm.nih.gov/articles/PMC139674/
TABLE 1.
Genotyping and antibiogram of tested MRSA strains
|
Doubling time (min)h
|
hDoubling time during exponential growth phase (optical density of 0.05 to ∼1.0 at 660 nm) measured by using TN-2612 (Advantec Toyo Kaisha, Ltd., Tokyo, Japan) as previously described (8).
|
C-MRSA/NORSA strains grew significantly faster than H-MRSA strains: the mean doubling times (8) of the former group of strains were 28.79 ± 7.09 and 28.24 ± 2.48 min, respectively, whereas that for the latter was 38.81 ± 7.01 min (see Table 1). The difference was statistically significant (P value of <0.0001 by t test). This high growth rate may be a prerequisite in the absence of antibiotics for C-MRSA to achieve successful colonization of humans by outcompeting the numerous bacterial species in the environment.



https://pubmed.ncbi.nlm.nih.gov/35762208/
Elife
. 2022 Jun 28:11:e74819. doi: 10.7554/eLife.74819.
Stable antibiotic resistance and rapid human adaptation in livestock-associated MRSA
Marta Matuszewska # 1, Gemma G R Murray # 1, Xiaoliang Ba 1, Rhiannon Wood 1, Mark A Holmes 1, Lucy A Weinert 1
https://elifesciences.org/articles/74819
https://pmc.ncbi.nlm.nih.gov/articles/PMC9239682/
Figure 3. A type V SCCmec has been maintained since the 1980s, with occasional replacements.
(c) Well-supported clades and SCCmec type are annotated on the CC398 phylogeny in external rings.

----------

## robust
ロバスト

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6350394/#Sec10
Comparative genomics of Bacteria commonly identified in the built environment

Robustness and limitations

It is important to note that the results remained similar when different data sets were compared (Additional file 1: Table S9). We tested the robustness to the composition of the genome data set by testing different subsets of bacteria (e.g., phyla of Proteobacteria, Firmicutes, and Actinobacteria), and also by randomly selecting one representative for species that have multiple strains sequenced. Of the four genomic features (genome size, GC content, GCSI, and S value), only the S value showed consistent results and tended to be higher in the “Common BE genera” compared to the “Others.”

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2608848/
Comparison of Correspondence Analysis Methods for Synonymous Codon Usage in Bacteria
It is important to note that these results remained similar when all complete bacterial genomic sequences available from the NCBI repository on August 2008 were included (data not shown). Similar results were obtained when only long sequences with >300 codons were used (data not shown). We also verified the consistency of the results when using detection thresholds below |r| = 0.70 (data not shown). Thus we conclude that WCA is more effective than the other three methods in generating axes that correspond to variation in synonymous codon usage, regardless of the data sets and statistical criteria used.https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2602791/
Using Mahalanobis distance to compare genomic signatures between bacterial plasmids and chromosomes
We tested the robustness of our result that Mahalanobis distance performs better than the conventional δ-distance by varying chromosomal segment size, word size (e.g. tri- and tetra-nucleotides) (12,29,37) and composition of the host data set. Campbell et al. (11) and Wong et al. (20) used 50-kb chromosomal segments, while van Passel et al. (25) used plasmid-size chromosomal segments; that is, the segment size was not fixed but depended on the plasmid size. In the present study, we used a fixed segment size. We found that our results remained similar when partitioning chromosomal sequences into different sizes of segments; e.g. 2, 5, 10 and 20 kb in length (data not shown). Among these, 5 kb was selected because the median rank of the genomic signature similarities between plasmids and their known hosts was maximized. The results were also consistent with those obtained when using relative abundances of different word sizes (data not shown). We also demonstrated that our results were robust to the composition of the host data set by testing different subsets of bacteria, e.g. Proteobacteria, Firmicutes, Gram-positive bacteria, Gram-negative bacteria and also when only one representative was randomly selected in the case of species for which multiple strains have been sequenced (data not shown). Thus, Mahalanobis distance performed better than δ-distance in identifying plasmid hosts based on genomic signature similarity, regardless of the datasets used.

----------
# publication

2023-06
https://pubmed.ncbi.nlm.nih.gov/37341708/
Microb Genom
. 2023 Jun;9(6):mgen001043. doi: 10.1099/mgen.0.001043.
Transconjugant range of PromA plasmids in microbial communities is predicted by sequence similarity with the bacterial host chromosome
Maho Tokuda 1, Masahiro Yuki 2, Moriya Ohkuma 2, Kazuhide Kimbara 1, Haruo Suzuki 3, Masaki Shintani 1 2 4
https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.001043#tab2
https://pmc.ncbi.nlm.nih.gov/articles/PMC10327500/
The R code used for analysis and figure generation is accessible at Github (https://github.com/tokudamaho/Tokuda_et_al._PromA_plasmid).

2020-06-09
https://pubmed.ncbi.nlm.nih.gov/32582111/
Front Microbiol
. 2020 Jun 9;11:1187. doi: 10.3389/fmicb.2020.01187. eCollection 2020.
Determination of Plasmid pSN1216-29 Host Range and the Similarity in Oligonucleotide Composition Between Plasmid and Host Chromosomes
Maho Tokuda 1, Haruo Suzuki 2 3, Kosuke Yanagiya 1, Masahiro Yuki 4, Kengo Inoue 5, Moriya Ohkuma 4, Kazuhide Kimbara 1, Masaki Shintani 1 4 6
https://www.frontiersin.org/articles/10.3389/fmicb.2020.01187/full
https://pmc.ncbi.nlm.nih.gov/articles/PMC7296055/

2018-12-15
https://pubmed.ncbi.nlm.nih.gov/30619542/
Review Comput Struct Biotechnol J
. 2018 Dec 15:17:70-81. doi: 10.1016/j.csbj.2018.12.001. eCollection 2019.
Reconsidering plasmid maintenance factors for computational plasmid design
Hirokazu Yano 1, Masaki Shintani 2 3, Masaru Tomita 4 5, Haruo Suzuki 4 5, Taku Oshima 6
https://pmc.ncbi.nlm.nih.gov/articles/PMC6312765/
2.3. Constraints in the Genome
Bioinformatics analysis revealed constraints in plasmids with respect to size, sequence composition (G + C content, oligonucleotide composition, and codon usage), and gene direction. These features may be a result of plasmid-host co-evolution, which can stabilize plasmids in host cell populations. It is important to note that the sequence composition can vary among genes/segments within a plasmid/genome [23,92,93].
2.3.1. Size Constraint
2.3.2. G + C Content
2.3.3. Oligonucleotide Composition
2.3.4. Codon Usage
2.3.5. Gene Direction

2010-09-27
https://pubmed.ncbi.nlm.nih.gov/20851899/
J Bacteriol
. 2010 Nov;192(22):6045-55. doi: 10.1128/JB.00277-10. Epub 2010 Sep 17.
Predicting plasmid promiscuity based on genomic signature
Haruo Suzuki 1, Hirokazu Yano, Celeste J Brown, Eva M Top
https://jb.asm.org/content/192/22/6045.long
https://pmc.ncbi.nlm.nih.gov/articles/PMC2976448/

2008-10-25
https://pubmed.ncbi.nlm.nih.gov/18953039/
Comparative Study Nucleic Acids Res
. 2008 Dec;36(22):e147. doi: 10.1093/nar/gkn753. Epub 2008 Oct 25.
Using Mahalanobis distance to compare genomic signatures between bacterial plasmids and chromosomes
Haruo Suzuki 1, Masahiro Sota, Celeste J Brown, Eva M Top
https://academic.oup.com/nar/article/36/22/e147/1196271
https://pmc.ncbi.nlm.nih.gov/articles/PMC2602791/

----------

----------
## five-second rule
5秒ルール

https://ja.wikipedia.org/wiki/5秒ルール

2021.09.26
https://gendai.media/articles/-/84268
食べ物を落としたときの「3秒ルール」は科学的なのか？（村上 道夫,永井 孝志,小野 恭子,岸本 充生） | ブルーバックス | 講談社
『基準値のからくり』5
https://gendai.media/articles/-/84268?page=2


2017-03-24 09:00:22
https://ameblo.jp/hopenglishjp/entry-12256613022.html
【外国人も知ってた】5秒ルール

2016年11月2日
https://www.newsweekjapan.jp/stories/world/2016/11/post-6166.php
すぐに拾えばＯＫ？「５秒ルール」の真実
The Truth of the Five-Second Rule
細菌が最も付着しやすいのはステンレスの床、最も付着しにくいのはカーペットだった。また食べ物に含まれる水分が多いほど細菌の付着率も高くなり、スイカが最も多くの細菌を集めた。

2016/09/25 19:00
https://forbesjapan.com/articles/detail/13684
床に落ちた食品の5秒ルールは「危険」、米研究結果　ある意味正しい面も
https://forbesjapan.com/articles/detail/13684/2/1/1
シャフナーによると、物の表面から食品への細菌の移行に最も大きく影響するのは水分だと考えられる。

https://twitter.com/mason_lab/status/778277213724631041
Christopher Mason on Twitter: "For anyone who's compared DNA yield from wet vs. dry swabs, here you go; At last, the demise of the 5-second rule: https://t.co/2jfanVLlDD"
12:58 PM - 20 Sep 2016

https://www.nytimes.com/2016/09/20/science/five-second-rule.html
‘Five-Second Rule’ for Food on Floor Is Untrue, Study Finds

The composition of the food and the surface on which it falls matter as much if not more than the length of time it remains on the floor, the study found. Watermelon, with its moisture, drew the highest rate of contamination and the gummy candy the least.

食品が床に落ちてからの時間の長さだけでなく、食品の性質（水分量など）や床の種類の方が、汚染の度合いに大きな影響を与える。
スイカのように水分が多い食品は、最も高い汚染率を示し、グミのような乾燥した食品は汚染率が最も低かった。

この研究は、従来信じられてきた「5秒以内なら安全」という通説を科学的に否定している。

2016.09.13 TUE 08:00
https://wired.jp/2016/09/13/five-second-rule/
落とした食べ物の「5秒ルール」はウソ：研究結果
床に食べ物が落ちても5秒以内なら大丈夫？「5秒ルール」と呼ばれる通説には科学的に根拠がないことを、米ラトガース大学の研究チームが証明した。
概して最も伝播率が高かったのは、タイルとステンレスだった。木材の伝播率は食品によって異なる一方、カーペットについては、その上に落ちた食べものへの細菌の伝播が最も起こりにくかった。
表面が滑らかで水分を多く含むスイカでは、床から汚染を吸い上げる効果が最も大きかった。

http://gigazine.net/news/20160912-five-second-rule/
食べ物を落としてもすぐに食べればOKな「5秒ルール」に科学的根拠はあるのか？ - GIGAZINE

https://pubmed.ncbi.nlm.nih.gov/27590818/
Appl Environ Microbiol
. 2016 Oct 14;82(21):6490-6496. doi: 10.1128/AEM.01838-16. Print 2016 Nov 1.
Longer Contact Times Increase Cross-Contamination of Enterobacter aerogenes from Surfaces to Food
Robyn C Miranda 1, Donald W Schaffner 2
https://journals.asm.org/doi/10.1128/aem.01838-16
https://pmc.ncbi.nlm.nih.gov/articles/PMC5066366/

ABSTRACT
Bacterial cross-contamination from surfaces to food can contribute to foodborne disease. The cross-contamination rate of Enterobacter aerogenes on household surfaces was evaluated by using scenarios that differed by surface type, food type, contact time (<1, 5, 30, and 300 s), and inoculum matrix (tryptic soy broth or peptone buffer). The surfaces used were stainless steel, tile, wood, and carpet. The food types were watermelon, bread, bread with butter, and gummy candy. Surfaces (25 cm2) were spot inoculated with 1 ml of inoculum and allowed to dry for 5 h, yielding an approximate concentration of 107 CFU/surface. Foods (with a 16-cm2 contact area) were dropped onto the surfaces from a height of 12.5 cm and left to rest as appropriate. Posttransfer, surfaces and foods were placed in sterile filter bags and homogenized or massaged, diluted, and plated on tryptic soy agar. The transfer rate was quantified as the log percent transfer from the surface to the food. Contact time, food, and surface type all had highly significant effects (P < 0.000001) on the log percent transfer of bacteria. The inoculum matrix (tryptic soy broth or peptone buffer) also had a significant effect on transfer (P = 0.013), and most interaction terms were significant. More bacteria transferred to watermelon (∼0.2 to 97%) than to any other food, while the least bacteria transferred to gummy candy (∼0.1 to 62%). Transfer of bacteria to bread (∼0.02 to 94%) was similar to transfer of bacteria to bread with butter (∼0.02 to 82%), and these transfer rates under a given set of conditions were more variable than with watermelon and gummy candy.

IMPORTANCE The popular notion of the “five-second rule” is that food dropped on the floor and left there for <5 s is “safe” because bacteria need time to transfer. The rule has been explored by a single study in the published literature and on at least two television shows. Results from two academic laboratories have been shared through press releases but remain unpublished. We explored this topic by using four different surfaces (stainless steel, ceramic tile, wood, and carpet), four different foods (watermelon, bread, bread with butter, and gummy candy), four different contact times (<1, 5, 30, and 300 s), and two bacterial preparation methods. Although we found that longer contact times result in more transfer, we also found that other factors, including the nature of the food and the surface, are of equal or greater importance. Some transfer takes place “instantaneously,” at times of <1 s, disproving the five-second rule.

接触時間が長いほど、表面から食品への Enterobacter aerogenes の交差汚染が増加する

要旨（ABSTRACT）
表面から食品への細菌の交差汚染は、食中毒の原因となる可能性がある。本研究では、家庭内のさまざまな表面における Enterobacter aerogenes の交差汚染率を、表面の種類、食品の種類、接触時間（<1秒、5秒、30秒、300秒）、および接種液（トリプチックソイブロスまたはペプトン緩衝液）の違いによって評価した。
使用した表面はステンレス、タイル、木材、カーペットの4種、食品はスイカ、パン、バター付きのパン、グミキャンディの4種である。各表面（25cm²）には1mlの接種液をスポット接種し、5時間乾燥させた（表面あたり約10⁷ CFUの濃度）。食品（接触面積16cm²）は高さ12.5cmから表面に落とし、設定した時間だけ接触させた。接触後、表面と食品はそれぞれ滅菌済みのフィルターバッグに入れて均質化またはもみほぐし、希釈後にトリプチックソイ寒天培地に塗布して菌数を測定した。
菌の移行率は、「表面から食品への移行率の対数（log%）」として定量化した。その結果、接触時間、食品の種類、表面の種類は、いずれも菌の移行に非常に大きな影響を与えることが明らかになった（P < 0.000001）。接種液の種類（トリプチックソイブロスかペプトン緩衝液か）も有意な影響があり（P = 0.013）、ほとんどの交互作用も有意であった。
食品の中では、スイカへの菌の移行率が最も高く（約0.2〜97%）、グミキャンディへの移行率が最も低かった（約0.1〜62%）。パン（約0.02〜94%）とバター付きパン（約0.02〜82%）への移行率は類似していたが、スイカやグミに比べてばらつきが大きかった。

重要性（IMPORTANCE）
一般的に知られる「5秒ルール」とは、床に落とした食べ物でも5秒以内なら「安全」とされる考え方であり、細菌が移動するのに時間がかかるという仮定に基づいている。しかし、このルールは、論文としては1件の研究と、2つのテレビ番組でしか取り上げられていない。2つの学術研究機関からのプレスリリースも存在するが、正式な論文発表はされていない。
本研究では、4種の表面（ステンレス、タイル、木材、カーペット）、4種の食品（スイカ、パン、バター付きパン、グミキャンディ）、4段階の接触時間（<1秒、5秒、30秒、300秒）、2種の細菌懸濁液を用いて、包括的にこのテーマを検討した。その結果、接触時間が長いほど細菌の移行が多くなることが確認されたが、食品や表面の性質といった他の要因も同様かそれ以上に重要であることがわかった。また、1秒未満でも細菌は“瞬時に”移行することが確認され、5秒ルールは否定された。



2014.03.14
https://www.gizmodo.jp/2014/03/5_61.html
｢落とした食べ物の5秒ルール｣の科学的根拠が実証される | ギズモード・ジャパン
イギリスのバーミンガムにあるアストン大学の微生物学のアンソニー・ヒルトン教授率いる研究チーム

----------






