Last Update: 2020-08-01

----------

# bacteria
**細菌**

## Table of Contents
- [unclassified](#unclassified)
- [heavy metals](#heavy-metals)
- [arsenic](#arsenic)
- [antimony](#antimony)
- [halogen](#halogen)
- [iodide](#iodide)
  - [iodate reduction](#iodate-reduction)
- [Bifidobacterium](#bifidobacterium)
- [Clostridia](#clostridia) クロストリジウム綱
- [Clostridium](#clostridium) クロストリジウム属
- [Clostridium difficile](#clostridium-difficile)
- [CPR](#cpr)
- [Geobacter](#geobacter)
  - [Inoue](#inoue) Inoue Kengo
  - [Geobacter_Shewanella](#geobacter_shewanella)
- [Lactobacillus](#lactobacillus)
vaginal microbiota
- [Mycobacterium](#mycobacterium)
- [Pseudomonas](#pseudomonas)
  - [Pseudomonas aeruginosa](#pseudomonas-aeruginosa)
  - [Pseudomonas stutzeri](#pseudomonas-stutzeri)
- [Rhodococcus](#rhodococcus)
  - Br-6
  - DLD
  - Bromate Reduction
- [Shewanella](#shewanella)
  - [reference_representative](#shewanella_reference_representative)
  - [phylogeny](#shewanella_phylogeny)
  - [comparative_genomics](#shewanella_comparative_genomics)
  - [space](#shewanella_space)
  - Shewanella_outer_membrane_cytochromes
  - Shewanella oneidensis MR-1
- [Sinorhizobium](#sinorhizobium)
  - KGO-5
- [Streptococcus](#streptococcus)
- [mitochondria](#mitochondria) 
- [Rickettsiales](#rickettsiales) リケッチア目
- [Holospora](#holospora) ホロスポラ属
- [Legionella](#legionella) レジオネラ
- [Wolbachia](#wolbachia) ボルバキア
- [people](#people)
  - [amachi](#amachi) Amachi Seigo
  - [Gilmore](#gilmore) Michael S. Gilmore
  - [Ogura](#ogura) Yoshitoshi Ogura
- [excerpt](#excerpt) 抜粋［引用］

----------
## unclassified

https://ja.unionpedia.org/i/真正細菌

![https://en.wikipedia.org/wiki/Tree_of_life_(biology)](https://upload.wikimedia.org/wikipedia/commons/thumb/9/98/A_Novel_Representation_Of_The_Tree_Of_Life.png/220px-A_Novel_Representation_Of_The_Tree_Of_Life.png)


https://ja.wikipedia.org/wiki/シトクロムc
（cytochrome c, cyt c）
ヘムタンパク質の一種である。タンパク質のシトクロムcファミリーに属する。他のシトクロムと異なり可溶性（100 g/L）で、電子伝達系において不可欠な因子である。

----------
## CPR
Candidate Phyla Radiation (CPR)

https://ja.wikipedia.org/wiki/CPR群
CPR（Candidate Phyla radiation）は、細菌の巨大系統群である。培養例が皆無で記載種を一切含まないが、全細菌の少なくとも15%、もしくはそれ以上を占めると推定されている[1]。
系統樹からは大古代の分岐と推定され、ほかの全ての細菌と姉妹群を形成する。

https://www.ncbi.nlm.nih.gov/pubmed/31446647
Mol Ecol. 2019 Sep;28(18):4259-4271. doi: 10.1111/mec.15208. Epub 2019 Sep 12.
Genomic signatures and co-occurrence patterns of the ultra-small Saccharimonadia (phylum CPR/Patescibacteria) suggest a symbiotic lifestyle.
Lemos LN1, Medeiros JD2, Dini-Andreote F3,4, Fernandes GR5, Varani AM6, Oliveira G7, Pylro VS8.
https://onlinelibrary.wiley.com/doi/abs/10.1111/mec.15208
By testing the hypothesis of genome streamlining or symbiotic lifestyle, our results revealed clear signatures of gene losses in these genomes, 


https://www.ncbi.nlm.nih.gov/pubmed/31519891
Nat Commun. 2019 Sep 13;10(1):4173. doi: 10.1038/s41467-019-12171-z.
The distinction of CPR bacteria from other bacteria based on protein family content.
Méheust R1,2, Burstein D1,3,4, Castelle CJ1,2,5, Banfield JF6,7,8,9
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6744442/
Enrichment analysis
Enrichment/depletion of protein families was calculated based on the frequency of the computed protein families in the first and second ‘NCBI’ datasets. The enrichment of each family in CPR vs. non-CPR bacteria was computed using a Fisher’s exact test on a contingency table of presence/absence in CPR and non-CPR bacteria genomes. Families were considered enriched or depleted if their p-values, after correction for false detection rate (Benjamini-Hochberg), were significant (<10−5) in both datasets. The remaining families were assigned as equally distributed.

----------
## Geobacter

- https://ja.wikipedia.org/wiki/ゲオバクター属
- http://www.bacterio.net/geobacter.html
- https://www.ncbi.nlm.nih.gov/genome/browse/#!/prokaryotes/Geobacter

https://www.ncbi.nlm.nih.gov/pubmed/31126529
Adv Microb Physiol. 2019;74:1-96. doi: 10.1016/bs.ampbs.2019.02.007. Epub 2019 May 15.
The electrifying physiology of Geobacter bacteria, 30 years on.
Reguera G1, Kashefi K1.

Fig. 2 Neighbor joining phylogenetic tree of the families Geobacteraceae and Desulfur- omonadaceae. 

Table 2 Type strains (in alphabetical order) for validly described Geobacter species available in pure culture.
```Source
Growth substrates
Temperature Range [optimal]
```

https://onlinelibrary.wiley.com/doi/abs/10.1111/mmi.14443
Molecular evidence for the adaptive evolution of Geobacter sulfurreducens to perform dissimilatory iron reduction in natural environments - Liu - - Molecular Microbiology
- E‐pili are constructed by truncated PilA which is considered to have evolved from full‐length pilin by gene fission under positive evolutionary selection.


https://www.ncbi.nlm.nih.gov/pubmed/27940542
Appl Environ Microbiol. 2017 Feb 1;83(4). pii: e02689-16. doi: 10.1128/AEM.02689-16. Print 2017 Feb 15.
Arsenic Detoxification by Geobacter Species.
Dang Y1,2, Walker DJ1, Vautour KE1, Dixon S3, Holmes DE4,3.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5288829/
- Analysis of 14 different Geobacter genomes showed that all of these species have genes coding for an arsenic detoxification system (ars operon), and several have genes required for arsenic respiration (arr operon) and methylation (arsM). Genes encoding four arsenic repressor-like proteins were detected in the genome of G. sulfurreducens; however, only one (ArsR1) regulated transcription of the ars operon. Elimination of arsR1 from the G. sulfurreducens chromosome resulted in enhanced transcription of genes coding for the arsenic efflux pump (Acr3) and arsenate reductase (ArsC).
- Some bacteria are able to methylate arsenic with S-adenosyl-l-methionine-dependent methyltransferase (ArsM), forming volatile methylarsines, which removes arsenic from the immediate environment by converting it into a gaseous compound (6). Other prokaryotes utilize arr and/or ars arsenic transformation systems. While both systems involve reduction of arsenate,
- Arsenic detoxification machinery in Geobacter.
All of the Geobacter genomes available to date have the arsenic detoxification machinery (ars genes), whereas only a few Geobacter species have genes required for arsenite methylation (arsM) and arsenate respiration (arr genes) (Table 1).
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5288829/table/T1/
TABLE 1
Arsenic detoxification and respiratory genes found in the genomes of the Geobacter species and strains used in this study
- Regulation of the ars operon.
The G. sulfurreducens genome has four different genes coding for putative ArsR-Smt family regulatory proteins scattered throughout the genome: GSU2952 (arsR1), GSU2149 (arsR2), GSU0399 (arsR3), and GSU2625 (arsR4). All four of these genes' proteins have the helix-turn-helix (HTH) motif that is characteristic of DNA binding transcriptional regulators (see Fig. S4A in the supplemental material), and all of them are found near transport proteins. 
- REFERENCES
6. Bentley R, Chasteen TG. 2002. Microbial methylation of metalloids: arsenic, antimony, and bismuth. Microbiol Mol Biol Rev 66:250–271. 

https://www.ncbi.nlm.nih.gov/pubmed/22114840
Adv Microb Physiol. 2011;59:1-100. doi: 10.1016/B978-0-12-387661-4.00004-5.
Geobacter: the microbe electric's physiology, ecology, and practical applications.
Lovley DR1, Ueki T, Zhang T, Malvankar NS, Shrestha PM, Flanagan KA, Aklujkar M, Butler JE, Giloteaux L, Rotaru AE, Holmes DE, Franks AE, Orellana R, Risso C, Nevin KP.

05 September 2011
https://www.nature.com/articles/news.2011.519
Electrified bacterial filaments zap uranium | Nature
- Hair-like filaments (yellow) allow Geobacter (orange) to precipitate uranium while keeping the toxic metal away from the cell.
- Some bacteria, including a species called Geobacter sulfurreducens, are known to get their energy from reducing — or adding electrons to — metals in the environment. When uranium dissolved in groundwater is reduced in this way, the metal becomes much less soluble, reducing the spread of contamination.
- , such as the Fukushima nuclear plant in Japan, which was devastated by a tsunami earlier this year.
- Uranium is not the main radioisotope released at Fukushima, but Reguera sees potential for widening the reach of Geobacter pili. 

ファージ
https://www.ncbi.nlm.nih.gov/pubmed/25083935
ISME J. 2015 Feb;9(2):333-46. doi: 10.1038/ismej.2014.128. Epub 2014 Aug 1.
Evidence of Geobacter-associated phage in a uranium-contaminated aquifer.
Holmes DE1, Giloteaux L2, Chaurasia AK2, Williams KH3, Luef B4, Wilkins MJ5, Wrighton KC6, Thompson CA2, Comolli LR3, Lovley DR2.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4303627/
- The programs PHAST (Zhou et al., 2011) and/or Prophinder were also used to find phage-related sequences. Others were determined by the presence of known phage domains (http://pfam.sanger.ac.uk/, http://blast.jcvi.org/web-hmm/) or the word ‘phage' in protein coding features.
- Further evidence that some of the subsurface Geobacter phage are lysogenic comes from the fact that multiple phage attachment sites (att) were detected in the Geobacter chromosomes, and the programs PHAST (Zhou et al., 2011 PHAST) and/or Prophinder (http://aclame.ulb.ac.be/Tools/Prophinder/) identified prophage in all five of the genomes.

http://aclame.ulb.ac.be/perl/Aclame/Prophages/view_prophage.cgi?
ACLAME: Prophinder
```
Geobacter metallireducens GS-15	NC_007517	chromosome
Geobacter sulfurreducens PCA	NC_002939	chromosome
Geobacter uraniireducens Rf4	NC_009483	chromosome
Gloeobacter violaceus PCC 7421	NC_005125	chromosome
```

好熱菌
https://www.ncbi.nlm.nih.gov/pubmed/12732575
Appl Environ Microbiol. 2003 May;69(5):2985-93.
Thermophily in the Geobacteraceae: Geothermobacter ehrlichii gen. nov., sp. nov., a novel thermophilic member of the Geobacteraceae from the "Bag City" hydrothermal vent.
Kashefi K1, Holmes DE, Baross JA, Lovley DR.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC154550/
Unlike previously described members of the Geobacteraceae, which are mesophiles, strain SS015 was a thermophile and grew at temperatures of between 35 and 65°C, with an optimum temperature of 55°C. 
```
Geothermobacter ehrlichii	213224	PRJNA500323	500323	Proteobacteria	delta/epsilon subdivisions	3.23595	61.9	-	VNIB01	33	3045	2933	2019/08/29	2019/09/02	Contig	DOE Joint Genome Institute	SAMN12255868	GCA_008124615.1	-	ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/008/124/615/GCA_008124615.1_ASM812461v1	-	SS015
```

### Geobacter_phylogeny
系統解析

https://www.ncbi.nlm.nih.gov/pubmed/25139417
Int J Syst Evol Microbiol. 2014 Nov;64(Pt 11):3786-91. doi: 10.1099/ijs.0.066662-0. Epub 2014 Aug 19.
Geobacter soli sp. nov., a dissimilatory Fe(III)-reducing bacterium isolated from forest soil.
Zhou S1, Yang G2, Lu Q3, Wu M3.
http://ijs.microbiologyresearch.org/content/journal/ijsem/10.1099/ijs.0.066662-0#tab3
Fig. 1.
Phylogenetic consensus tree based on 16S rRNA gene sequences, 
Desulfuromusa kysingii DSM 7343T (GenBank accession no. X79414) was used as an outgroup.
```
Pelobacter propionicus DSM 2379T (CP000482)
Geobacter chapellei 172T (U41561)
Geobacter psychrophilus P35T (AY653549)
Geobacter thiogenes JCM 14045T (AF223382)
Geobacter lovleyi SZT (CP001089)
Geobacter luticola OSK6T (AB682759)
Geobacter bemidjiensis BemT (CP001124)
Geobacter bremensis Dfr1T (U96917)
Geobacter uraniireducens Rf4T (CP000698)
Geobacter daltonii FRC-32T (CP001390)
Geobacter toluenoxydans TMJ1T (EU711072)
```
![](https://www.microbiologyresearch.org/docserver/fulltext/ijsem/64/11/066662-f1.gif)

https://www.ncbi.nlm.nih.gov/pubmed/19284579
BMC Genomics. 2009 Mar 11;10:103. doi: 10.1186/1471-2164-10-103.
Evolution from a respiratory ancestor to fill syntrophic and fermentative niches: comparative fenomics of six Geobacteraceae species.
Butler JE1, Young ND, Lovley DR.
https://link.springer.com/article/10.1186/1471-2164-10-103
Figure 1
Genome-based Geobacteraceae phylogeny. Bayesian inference of the phylogenetic tree of the six Geobacteraceae species discussed in the text, using another Deltaproteobacterial species, Anaeromyxobacter dehalogenans, as the outgroup. The tree was based on a concatenation of the proteins in the 481 families that had exactly one ortholog conserved in each of the seven genomes (see Additional file 6). Values at branch points are posterior probabilities.
![](https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2164-10-103/MediaObjects/12864_2008_Article_1987_Fig1_HTML.jpg)

https://www.ncbi.nlm.nih.gov/pubmed/15388715
Int J Syst Evol Microbiol. 2004 Sep;54(Pt 5):1591-9.
Comparison of 16S rRNA, nifD, recA, gyrB, rpoB and fusA genes within the family Geobacteraceae fam. nov.
Holmes DE1, Nevin KP, Lovley DR.
https://www.microbiologyresearch.org/content/journal/ijsem/10.1099/ijs.0.02958-0#tab2
Fig. 1.
Phylogenetic trees predicted by maximum-likelihood analysis comparing sequences of the 16S rRNA gene (a), fusA (b), gyrB (c), nifD (d), recA (e) and rpoB (f) from species of the Geobacteraceae. Chlorobium tepidum and Nostoc sp. PCC 7120 were used as outgroups and 1000 replicates were used for maximum-likelihood (first bootstrap value) and maximum-parsimony (second value) bootstrap analyses. Taxa are represented by type strains unless the strain name is indicated.
![](https://www.microbiologyresearch.org/docserver/fulltext/ijsem/54/5/IJE45259-1.gif)

https://www.ncbi.nlm.nih.gov/pubmed/18461135
PLoS One. 2008 May 7;3(5):e2103. doi: 10.1371/journal.pone.0002103.
The mosaic genome of Anaeromyxobacter dehalogenans strain 2CP-C suggests an aerobic common ancestor to the delta-proteobacteria.
Thomas SH1, Wagner RD, Arakaki AK, Skolnick J, Kirby JR, Shimkets LJ, Sanford RA, Löffler FE.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2330069/
Figure 1
16S rRNA gene-based phylogenetic tree of the delta-Proteobacteria indicates that A. dehalogenans strain 2CP-C is deeply nested in the order Myxococcales.
![](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2330069/bin/pone.0002103.g001.jpg)

https://www.ncbi.nlm.nih.gov/pubmed/10966399
Appl Environ Microbiol. 2000 Sep;66(9):3842-9.
Phylogeny of microorganisms populating a thick, subaerial, predominantly lithotrophic biofilm at an extreme acid mine drainage site.
Bond PL1, Smriga SP, Banfield JF.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC92229/
FIG. 6
Evolutionary distance dendogram of slime clones within the delta subdivision of the Proteobacteria based on 1,121 nucleotides of 16S rDNA sequences. Desulfovibrio desulfuricans (not shown) was used as the outgroup. Indications of branch point support and evolutionary distance are as described for Fig. ​Fig.4.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC92229/figure/F6/
![](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC92229/bin/am0901966006.jpg)

https://www.ncbi.nlm.nih.gov/pubmed/8900001
Appl Environ Microbiol. 1996 Nov;62(11):4108-13.
A freshwater anaerobe coupling acetate oxidation to tetrachloroethylene dehalogenation.
Krumholz LR1, Sharp R, Fishbain SS.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC168232/
https://www.researchgate.net/figure/Phylogenetic-tree-of-TT4B-and-other-representatives-of-the-delta-subclass-of-the_fig4_14307027
Phylogenetic tree of TT4B and other representatives of the delta subclass of the Proteobacteria based on 16S rRNA sequence comparisons. 
![](https://www.researchgate.net/profile/Susan_Fishbain/publication/14307027/figure/fig4/AS:667587264393217@1536176615479/Phylogenetic-tree-of-TT4B-and-other-representatives-of-the-delta-subclass-of-the.png)

https://www.sciencedirect.com/topics/medicine-and-dentistry/pelobacter
- Figure 4. Maximum-likelihood tree showing the phylogenetic relationship between the members of the family Geobacteraceae within the class Deltaproteobacteria using 16S rRNA gene (&gt; 1300 bp).
![](https://ars.els-cdn.com/content/image/1-s2.0-B9780123876614000045-f01-04-9780123876614.jpg)
- Fig. 2. Neighbor joining phylogenetic tree of the families Geobacteraceae and Desulfuromonadaceae.
![](https://ars.els-cdn.com/content/image/1-s2.0-S0065291119300086-gr2.jpg)


- https://ja.wikipedia.org/wiki/粘液細菌
目	:	ミクソコックス目 Myxococcales
- https://www.ncbi.nlm.nih.gov/genome/browse/#!/prokaryotes/Desulfuromusa
- http://www.bacterio.net/desulfuromusa.html
- https://www.ncbi.nlm.nih.gov/genome/browse/#!/overview/Desulfuromonas
- https://microbewiki.kenyon.edu/index.php/Desulfuromonas
Bacteria; Proteobacteria; delta/epsilon subdivisions; Deltaproteobacteria; Desulfuromonadales; Desulfuromonadaceae
- https://en.wikipedia.org/wiki/Desulfuromusa_ferrireducens
Order:	Desulfuromonadales
Family:	Desulfuromonadaceae
Genus:	Desulfuromusa
Species:	D. ferrireducens



### Geobacter_codon_usage
コドン使用

https://www.ncbi.nlm.nih.gov/pubmed/22032172
Genome Biol. 2011 Oct 27;12(10):R109. doi: 10.1186/gb-2011-12-10-r109.
Variation in global codon usage bias among prokaryotic organisms is associated with their lifestyles.
Botzman M1, Margalit H.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3333779/
Other prokaryotes with almost as high CAIave values are quite diverse: Geobacter metallireducens (CAIave value of 0.79) is an anaerobic bacterium that uses iron oxides as the electron acceptor in the oxidation of organic compounds to carbon dioxide [23], 


### Geobacter_comparative_genomics
比較ゲノム

https://www.hindawi.com/journals/isrn/2013/850179/
A Whole Genome Pairwise Comparative and Functional Analysis of Geobacter sulfurreducens PCA
Figure 1: Phylogenetic tree constructed using 16S rRNA of Geobacter species.
https://www.hindawi.com/journals/isrn/2013/850179/fig1/

https://www.ncbi.nlm.nih.gov/pubmed/22967216
BMC Genomics. 2012 Sep 12;13:471. doi: 10.1186/1471-2164-13-471.
Comparative genomic analysis of Geobacter sulfurreducens KN400, a strain with enhanced capacity for extracellular electron transfer and electricity production.
Butler JE1, Young ND, Aklujkar M, Lovley DR.

https://www.ncbi.nlm.nih.gov/pubmed/22616984
BMC Genomics. 2012 May 22;13:200. doi: 10.1186/1471-2164-13-200.
Genomic determinants of organohalide-respiration in Geobacter lovleyi, an unusual member of the Geobacteraceae.
Wagner DD1, Hug LA, Hatt JK, Spitzmiller MR, Padilla-Crespo E, Ritalahti KM, Edwards EA, Konstantinidis KT, Löffler FE.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3403914/
- Computational analyses
Codon adaptation indices (CAI) for strain SZ ORFs encoded on the plasmid and chromosomal genomic islands were computed against a codon usage table based upon all strain SZ chromosomal ORFs using the E-CAI server [29]. 
- Plasmid maintenance
The pSZ77 RepA appears to share phylogenetic affiliation with the RepA of various plasmids from β- or γ-Proteobacteria (Figure 5), including an IncQ-like mobilizable plasmid [70] and an IncP-1-like environmental plasmid [71]. 
Figure 5
Unrooted proML tree of 35 plasmid replication initiator protein (RepA) sequences aligning with pSZ77 RepA in PSI-BLAST. 
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3403914/figure/F5/

https://www.ncbi.nlm.nih.gov/pubmed/20078895
BMC Genomics. 2010 Jan 17;11:40. doi: 10.1186/1471-2164-11-40.
Evolution of electron transfer out of the cell: comparative genomics of six Geobacter genomes.
Butler JE1, Young ND, Lovley DR.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825233/
Figure 1
Genome-based Geobacter phylogeny.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825233/figure/F1/

https://www.ncbi.nlm.nih.gov/pubmed/18844997
BMC Genomics. 2008 Oct 9;9:471. doi: 10.1186/1471-2164-9-471.
Comparative genomics of Geobacter chemotaxis genes reveals diverse signaling function.
Tran HT1, Krushkal J, Antommattei FM, Lovley DR, Weis RM.

2019-4-30
https://engineer.fabcross.jp/archeive/190426_electricity-conducting-bacteria.html
電気を通すバクテリアの構造を解明――タンパク質を使った生体用配線開発の可能性も

https://mainichi.jp/articles/20160604/ddl/k22/040/140000c
微生物発電：１００倍超に　二又・静岡大教授が効率化物質を発見　／静岡 - 毎日新聞

2015年06月09日
http://eetimes.jp/ee/articles/1506/09/news041.html
東大駒場リサーチキャンパス公開2015：赤い微生物「ジオバクター」の燃料電池、廃水処理システムを変える？ - EE Times Japan

Jan. 23, 2015
https://www.jst.go.jp/pr/announce/20120605/index.html
微生物が互いに電子をやり取りする未知の「電気共生」を発見

5/24/12
https://io9.gizmodo.com/5908318/10-surprising-things-that-bacteria-like-to-eat
Frank Miller Is Illustrating a YA Novel That Will Tell the Lady of the Lake's Origin
10. Nuclear Waste
Geobacter sulfurreducens
9. Arsenic

酢酸を電子供与体に、硝酸を電子受容体として、土壌細菌の代表株であるゲオバクター（Geobacter sulfurreducens ）とチオバチルス（Thiobacillus denitrificans ）を共培養

Mar 28, 2012
https://www.youtube.com/watch?v=LxaAWbN2koo
Michigan State University 2011 President's Report: Zapping nuclear waste. - YouTube

2011年9月7日
http://sustainablejapan.net/?p=452
発電しながら放射能汚染土壌を浄化する微生物「ジオバクター」の謎、ミシガン州立大が解明 | SJN News　再生可能エネルギー最新情報

https://wired.jp/2011/09/16/ウランの地下水汚染を防ぐ細菌、そのメカニズム/
ジオバクター科の細菌は、ウラン鉱山等の地下水汚染防止に使われてきた。その仕組みは今までわかっていなかったが、このほどその一部を明らかにする研究が発表された。

----------

### Inoue
Inoue Kengo

https://www.ncbi.nlm.nih.gov/pubmed/29748402
Genome Announc. 2018 May 10;6(19). pii: e00352-18. doi: 10.1128/genomeA.00352-18.
Complete Genome Sequence of Geobacter sulfurreducens Strain YM18, Isolated from River Sediment in Japan.
Inoue K1, Ogura Y2, Kawano Y3, Hayashi T2.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5946036/
Compared to the two previously sequenced G. sulfurreducens strains, YM18 contains 220 unique CDSs. G. sulfurreducens strains generally have over 100 c-type cytochromes. A slightly smaller number of genes were predicted to encode c-type cytochromes in YM18 (101 CDSs) than in PCA and KN400 (106 and 107, respectively). The complete genome sequence of YM18 and more detailed genome comparison with strains PCA and KN400 will identify commonly preserved genetic features for effective electron transfer to electrodes.

2016
IFO 公益財団法人発酵研究所
http://www.ifo.or.jp/rc_pdf/30.pdf
微生物による有機物の化学エネルギーから
電気エネルギーへの変換能力解明に向けた
基礎的研究
井　上　謙　吾
宮崎大学農学部
方法：
著者らが分離した新規分離発電株 YM18 株につい
て，全ゲノムを決定し，既報の発電株 G. sulfurreducens
PCA 株，KN400 株との比較ゲノム解析を行った．
結果・考察：
YM18 株の全ゲノム配列を決定したところ，約
3.7 Mbp で，3,354 の ORF を持っており，PCA 株に対
して，122,022 個の SNPs を有していた．発電能力が
PCA 株よりも高い YM18 株と KN400 株のみに共通す
る遺伝子が35個見いだされ，これらの遺伝子が高い発
電能力に何らかの役割を果たしている可能性が考えら
れた．

2013.03.29 
https://www.sbj.or.jp/sbj/sbj_vol90_no03.html
バイオミディア
pdf電子を放出する微生物 …井上　謙吾…（132）
https://www.sbj.or.jp/wp-content/uploads/file/sbj/9003/9003_biomedia_1.pdf
これまでに詳細な研究が行われてきた鉄還元菌
Shewanella oneidensis，Geobacter sulfurreducensでは
効率的な細胞外への電子移動には主にc型ヘム（ヘムc）
を補因子とするシトクロムcが利用される2)．

2011-12
https://ci.nii.ac.jp/naid/40019971881
微生物の細胞外電子伝達
Microbial Extracellular Electron Transfer
井上 謙吾
https://www.jseb.jp/wordpress/wp-content/uploads/11-12-033.pdf
鉄還元細菌の細胞外電子伝達についての研究は Shewanella oneidensis や Geobacter sulfurreducens を中心に
先駆的な研究がなされてきた。
図 3．Geobacter sulfurreducens（左）と Shewanella oneidensis（右）における予想細胞外電子伝達モデル。

https://www.jstage.jst.go.jp/article/kagakutoseibutsu/46/7/46_7_509/_article/-char/ja/
海外だより
Lovley研究室でのGeobacter 属細菌と微生物燃料電池の研究
井上 謙吾
2008 年 46 巻 7 号 p. 509-513
https://www.jstage.jst.go.jp/article/kagakutoseibutsu/46/7/46_7_509/_pdf/-char/ja

----------
### Geobacter_Shewanella
- https://mra.asm.org/search/Geobacter
- https://mra.asm.org/search/Shewanella

9:12 AM - 3 Jul 2019
https://twitter.com/Kyukimasa/status/1146406334897651712
- ”電線バクテリア”
https://www.nytimes.com/2019/07/01/science/bacteria-microbes-electricity.html
鉄還元菌と硝酸還元菌が導電性の生きたケーブルで繋がって電子を受け渡して共生しているという話。
1980年代になって発見されてから多くの種が発見されて研究進んでる(生物燃料電池とかの応用も)
- ケーブルバクテリア、海底や川底から数cmも長い導電性ワイヤーが伸びてて肉眼でも見えるとか。世界中から集めてて分類している
https://sciencedirect.com/science/article/pii/S0723202016300388
属名として”Electrothrix”とか提案されてたのか
- ここ最近、電気微生物の研究進んでるとは聞いたけど、検索してたら鉄還元細菌ジオバクター
https://labroots.com/trending/microbiology/5830/microbe-geobacter
ぬいぐるみになってるのをいきなり発見してしまう
https://giantmicrobes.com/us/products/geobacter.html
妙に再現度高い

https://www.nytimes.com/2019/07/01/science/bacteria-microbes-electricity.html
Wired Bacteria Form Nature’s Power Grid: ‘We Have an Electric Planet’ - The New York Times
Geobacter
Shewanella

2018.06.25
https://www.sbj.or.jp/sbj/sbj_vol95_no12.html
バイオミディア
pdf環境中の電子移動と微生物…鹿島 裕之…（ 739 ）
https://www.sbj.or.jp/wp-content/uploads/file/sbj/9512/9512_biomedia_1.pdf
モデル生物の
Shewanella属，Geobacter属細菌では，複数のシトクロ
ムcタンパク質が呼吸鎖のある内膜と細胞外膜表面とを
結ぶ電子回路として働く1,2)．

https://www.ncbi.nlm.nih.gov/pubmed/28871245
Front Microbiol. 2017 Aug 21;8:1584. doi: 10.3389/fmicb.2017.01584. eCollection 2017.
Comparative Genomic Analysis of Neutrophilic Iron(II) Oxidizer Genomes for Candidate Genes in Extracellular Electron Transfer.
He S1,2,3, Barco RA4,5, Emerson D4, Roden EE1,2.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5566968/
- Extracellular electron transfer (EET) is recognized as a key biochemical process in circumneutral pH Fe(II)-oxidizing bacteria (FeOB).
- The “porin-cytochrome c complex” (PCC) gene clusters homologous to MtoAB/PioAB are present in eight FeOB, accounting for 19% of complete and close-to-complete genomes examined, whereas PCC genes homologous to OmbB-OmaB-OmcB in Geobacter sulfurreducens are absent. 
- EET systems have been extensively studied in model Fe(III)-reducing bacteria (FeRB), such as Geobacter sulfurreducens and Shewanella oneidensis, for which electrons from the oxidation of organic substrates are passed to extracellular Fe(III) minerals. 
- Like the Fe(III)-reducing Geobacter and Shewanella spp., most neutrophilic FeOB are Gram-negative bacteria with the inner and outer membranes separated by a periplasmic space, thus EET systems also need to span the periplasm to bridge the outer membrane to the inner membrane. 
- Conductive nanowires
Alternative to the direct contact between extracellular Fe and a dedicated outer membrane Fe(II) oxidase, EET might be achieved through conductive nanowires analogous to those known for Geobacter and Shewanella, that allow for long-range EET over micrometer distances (Reguera et al., 2005; Gorby et al., 2006).

2016-2017
http://www.u-shizuoka-ken.ac.jp/file/88sekikawa.pdf
磁性粒子生産能を有するジオバクター属細菌の コレクションとその応用
主な異化的鉄還元細菌としてGeobacter属細菌(ジオバクター)とShewanella属細菌(シュ ワネラ)が挙げられます。

2014
https://www.jstage.jst.go.jp/article/jsmeja/29/2/29_KJ00009468463/_article/-char/ja/
細胞外電子伝達 : 固体を呼吸基質とする微生物たち
加藤 創一郎
https://www.jstage.jst.go.jp/article/jsmeja/29/2/29_KJ00009468463/_pdf
鉄 還元 能 を持つ Geobacter属や Shewanella

2011
http://www.jseb.jp/jeb/11-0102/11-12-033.pdf
微生物の細胞外電子伝達鉄還元細菌の細胞外電子伝達についての研究は Shewanella oneidensis や Geobacter sulfurreducens を中心に 先駆的な研究がなされてきた。

2009-12
環境バイオテクノロジー学会
https://ci.nii.ac.jp/naid/40018914443
微生物燃料電池での電流生産を可能にするShewanella oneidensisの細胞外電子伝達機構
https://www.jseb.jp/wordpress/wp-content/uploads/09-02-105.pdf
異化的金属還元細菌（dissimilatory
metal-reducing bacteria） と 呼 ば れ， 約 20 年前に Shewanella oneidensis MR-1 株1) と Geobacter metallireducens2) の 2 株が発見された．








----------
## heavy metals

https://ja.wikipedia.org/wiki/重金属
鉄(Fe)、鉛(Pb)、金(Au)、白金(Pt)、銀(Ag)、銅(Cu)、クロム(Cr)、カドミウム(Cd)、水銀(Hg)、亜鉛(Zn)、ヒ素(As)、マンガン(Mn)、コバルト(Co)、ニッケル(Ni)、モリブデン(Mo)、タングステン(W)、錫(Sn)、ビスマス(Bi)、ウラン(U)、プルトニウム(Pu)などが挙げられる。

https://www.ncbi.nlm.nih.gov/pubmed/30242595
Ecotoxicology. 2018 Sep 21. doi: 10.1007/s10646-018-1981-x. [Epub ahead of print]
Analysis of microbial communities in heavy metals-contaminated soils using the metagenomic approach.
Hemmat-Jou MH1, Safari-Sinegani AA2, Mirzaie-Asl A3, Tahmourespour A4.
Geobacter (Proteobacteria)
Pseudomonas (Proteobacteria)

----------
https://ja.wikipedia.org/wiki/第15族元素
周期表において第15族に属する窒素・リン・ヒ素・アンチモン・ビスマス・モスコビウムのこと。

## arsenic
https://ja.wikipedia.org/wiki/ヒ素

https://ja.wikipedia.org/wiki/亜ヒ酸
亜ヒ酸（あヒさん、arsenous acid、arsenious acid）は、化学式がAs(OH)3の無機化合物である。

https://www.frontiersin.org/articles/10.3389/fmicb.2019.01210/full
Frontiers | Arsenite Oxidation by a Newly Isolated Betaproteobacterium Possessing arx Genes and Diversity of the arx Gene Cluster in Bacterial Genomes | Microbiology
- Figure 6. Phylogeny and grouping of ARX based on ArxAB sequences.
- Figure 7. Phylogeny of the organisms possessing the arx genes, based on the concatenated amino acid sequences of 10 ribosomal proteins. 

https://www.pnas.org/content/early/2019/04/25/1818349116
Complete arsenic-based respiratory cycle in the marine microbial communities of pelagic oxygen-deficient zones | PNAS
We identified the presence and expression of genes for both arsenic reduction and oxidation in marine ODZs, suggesting the microbial community in these waters is also cycling arsenic for respiratory gain

https://github.com/haruosuz/mgsa/blob/master/references/README.memo.md#distribution
原核生物におけるヒ素耐性遺伝子の分布
https://www.ncbi.nlm.nih.gov/pubmed/30405552
Front Microbiol. 2018 Oct 23;9:2473. doi: 10.3389/fmicb.2018.02473. eCollection 2018.
Distribution of Arsenic Resistance Genes in Prokaryotes.
Ben Fekih I1, Zhang C1, Li YP1, Zhao Y2, Alwathnani HA3, Saquib Q4, Rensing C1,5, Cervantes C6.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6205960/
- The arsR gene encodes ArsR, a member of the SmtB/ArsR family of metalloregulatory proteins (reviewed in Busenlehner et al., 2003). ArsR is a trans-acting transcriptional repressor protein that binds to the promoter region of ars operons. The interaction of ArsR with arsenite dissociates the repressor protein from the DNA thus allowing transcription of the operon.
- The arsM gene, encoding ArsM, an As(III) S-adenosylmethionine methyl transferase enzyme, was first identified in a megaplasmid from the arsenic-resistant archaea Halobacterium sp. (Figure ​(Figure1).1). This gene was linked to an ars operon, and arsenite resistance was lost when arsM was deleted (Wang et al., 2004). 
- ArsM is regulated by the As(III)-responsive repressor ArsR and is thus thus dependent on the presence of arsenite.

https://www.ncbi.nlm.nih.gov/pubmed/23669886
Nat Rev Microbiol. 2013 Jun;11(6):371-84. doi: 10.1038/nrmicro3028. Epub 2013 May 13.
Antimicrobial activity of metals: mechanisms, molecular targets and applications.
Lemire JA1, Harrison JJ, Turner RJ.

https://www.ncbi.nlm.nih.gov/pubmed/14970228
J Biol Chem. 2004 Apr 30;279(18):18334-41. Epub 2004 Feb 16.
As(III) and Sb(III) uptake by GlpF and efflux by ArsB in Escherichia coli.
Meng YL1, Liu Z, Rosen BP.

亜ヒ酸酸化
https://www.ncbi.nlm.nih.gov/pubmed/29250936
Microbiologyopen. 2018 Jun;7(3):e00567. doi: 10.1002/mbo3.567. Epub 2017 Dec 17.
Expression of the arsenite oxidation regulatory operon in Rhizobium sp. str. NT-26 is under the control of two promoters that respond to different environmental cues.
Corsini PM1, Walker KT1, Santini JM1.

http://www.microbial-ecology.jp/meeting/JSME2016/pdf/P-170.pdf
日本微生物生態学会第 31 回横須賀大会2016 JSME annual meetingP-170土壌環境のヒ素挙動に関与する微生物群集機能の解明○濱村 奈津子1, 片岡 剛文2
亜ヒ酸 (AsIII)はヒ酸(AsV)と比べ土壌中での移動性および生体毒性が高いため、反応速度の高い微 生物代謝によるヒ素の形態変化は、環境挙動に影響し汚染リスクを悪化させる要因と考えられ ている。

https://www.ncbi.nlm.nih.gov/pubmed/26790947
FEMS Microbiol Rev. 2016 Mar;40(2):299-322. doi: 10.1093/femsre/fuv050. Epub 2016 Jan 19.
The microbial genomics of arsenic.
Andres J1, Bertin PN2.
https://academic.oup.com/femsre/article/40/2/299/2570226
Figure 1.
Filled circles, squares and triangles stand for organisms bearing aio, arr and arx genes, respectively. Open squares stand for organisms respiring As(V) in which the arr genes have not yet been clearly identified (see details in the text for P. calidifontis). 

亜ヒ酸・アンチモン酸化細菌
https://www.ncbi.nlm.nih.gov/pubmed/23666539
Microbes Environ. 2013;28(2):257-63. Epub 2013 May 11.
Identification of antimony- and arsenic-oxidizing bacteria associated with antimony mine tailing.
Hamamura N1, Fukushima K, Itai T.

https://www.ncbi.nlm.nih.gov/pubmed/19130261
Biometals. 2009 Feb;22(1):117-30. doi: 10.1007/s10534-008-9195-y. Epub 2009 Jan 7.
Microbial responses to environmental arsenic.
Páez-Espino D1, Tamames J, de Lorenzo V, Cánovas D.

https://www.nies.go.jp/kanko/news/26/26-5/26-5-03.html
微生物を利用して汚染土壌からヒ素を除去する （2007年度 26巻5号）｜国環研ニュース 26巻｜国立環境研究所

### arsM
- https://ja.wikipedia.org/wiki/ヒ素の生化学#ヒ素のメチル化
- https://ja.wikipedia.org/wiki/メチオニン-S-メチルトランスフェラーゼ
- https://ja.wikipedia.org/wiki/メチルトランスフェラーゼ

https://www.ncbi.nlm.nih.gov/pubmed/28266584
Sci Rep. 2017 Mar 7;7:42198. doi: 10.1038/srep42198.
Arsenic Methylation and its Relationship to Abundance and Diversity of arsM Genes in Composting Manure.
Zhai W1, Wong MT2, Luo F2, Hashmi MZ3, Liu X1, Edwards EA2, Tang X1,2, Xu J1.

2016-04-01 – 2019-03-31
https://kaken.nii.ac.jp/ja/grant/KAKENHI-PROJECT-16K00589/
KAKEN — 研究課題をさがす | ヒ素メチル基転移酵素を使ったメチル化反応による無機ヒ素の無毒化に関する研究 (KAKENHI-PROJECT-16K00589)
ヒ素メチル基転移酵素（arsM）

平成26年 6月16日  2011~2013
https://kaken.nii.ac.jp/file/KAKENHI-PROJECT-23380044/23380044seika.pdf
GSRB54 株はヒ素メチルトランスフェラーゼ(arsM)遺伝子を持ち、
The arsenite S-adenosylmethionine methyltransferase (arsM) gene of GSRB54

平成26年度 (第31集)
http://www.naro.affrc.go.jp/archive/niaes/sinfo/result/result31/result31_40.html
ヒ素をメチル化する根圏微生物の単離とその分子機構の解明 －稲に含まれるメチル化ヒ素の謎に迫る－（農業環境技術研究所）
亜ヒ酸のメチル基転移酵素遺伝子（arsM）
arsM 遺伝子の機能は、亜ヒ酸に S-アデノシル－L-メチオニン（SAM）のメチル基を転位する酵素であることが知られているため、

### arsR

https://www.wikigenes.org/e/gene/e/948013.html
arsR  -  arsenical resistance operon...
Synonyms: arsE

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5007761/
The ArsR acts as a transcriptional repressor regulating the expression of itself and downstream genes of the ars operon. 

土木学会第57回年次学術講演会（平成14年9月）
http://library.jsce.or.jp/jsce/open/00035/2002/57-7/57-7-0279.pdf
ArsR タンパク質は、大腸菌などヒ素耐性菌のゲノムにコードさ
れているヒ素耐性遺伝子群で遺伝子発現の調節を行うリプレッサー
のタンパク質である。ヒ素のない環境下で ArsR タンパク質は、ヒ
素耐性遺伝子群のオペレーション(O.P)部位に結合することで遺伝
子群の発現を抑制している。しかし、ヒ素のある環境下で ArsR タ
ンパク質は、ヒ素耐性遺伝子群を発現させるためにヒ素と優先的に
結合し、O.P 部位から離れる働きがある。

----------
## antimony
https://ja.wikipedia.org/wiki/アンチモン

https://www.ncbi.nlm.nih.gov/pubmed/28768219
J Hazard Mater. 2018 Jan 5;341:36-45. doi: 10.1016/j.jhazmat.2017.07.042. Epub 2017 Jul 21.
Removal of antimonate from wastewater by dissimilatory bacterial reduction: Role of the coexisting sulfate.
Zhu Y1, Wu M2, Gao N1, Chu W1, An N1, Wang Q1, Wang S1.
- The priority pollutant antimony (Sb) exists primarily as Sb(V) and Sb(III) in natural waters, and Sb(III) is generally with greater mobility and toxicity than Sb(V).
- , new genera of Pseudomonas and Geobacter emerged with large proportions in both Sb-fed and Sb-sulfate-fed sludge, 

https://www.sciencedirect.com/science/article/pii/S096483051630261X
Volume 115, November 2016, Pages 148-155
Bioremoval of antimony from contaminated waters by a mixed batch culture of sulfate-reducing bacteria - ScienceDirect
The dominant SRB species detected in the enrichment were closely related to Clostridium sp. K10 with 99% sequence similarity.

https://www.ncbi.nlm.nih.gov/pubmed/27342551
Appl Environ Microbiol. 2016 Aug 30;82(18):5482-95. doi: 10.1128/AEM.01375-16. Print 2016 Sep 15.
Microbial Antimony Biogeochemistry: Enzymes, Regulation, and Related Metabolic Pathways.
Li J1, Wang Q1, Oremland RS2, Kulp TR3, Rensing C4, Wang G5.
- In this review, we summarize the recent research on bacterial antimony transformations, especially those regarding antimony uptake, efflux, antimonite oxidation, and antimonate reduction.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5007761/
Antimony (Sb) occurs widely in soil and aquatic systems. It is a group 15 element in the periodic table, positioned directly below arsenic (As). It exists in four oxidation states (+V, +III, 0, and −III), of which pentavalent antimonate [Sb(V)] and trivalent antimonite [Sb(III)] are the prevalent forms in the environment (1).


https://www.ncbi.nlm.nih.gov/pubmed/26519630
Water Res. 2016 Jan 1;88:467-474. doi: 10.1016/j.watres.2015.10.042. Epub 2015 Oct 23.
Autotrophic antimonate bio-reduction using hydrogen as the electron donor.
- a short rod-shaped phylotype of Rhizobium (α-Protobacteria) that may have been active in Sb(V) reduction.

既存のSb還元細菌
- Sinorhizobium sp. JUK-1
- Desulfuribacillus stibiiarsenatis MLFW-2

#### Sinorhizobium sp. JUK-1
https://www.tandfonline.com/doi/abs/10.1080/01490451.2014.901440
01 Apr 2014
Isolation and Characterization of Antimony-Reducing Bacteria from Sediments Collected in the Vicinity of an Antimony Factory
Van Khanh Nguyen & Jong-Un Lee
- An antimonate-respiring bacterium
- Antimonate (pentavalent antimony) was reduced to antimonite (trivalent antimony) 
- a new strain of the Sinorhizobium genus.

#### Desulfuribacillus stibiiarsenatis MLFW-2
https://www.ncbi.nlm.nih.gov/pubmed/27974090
Int J Syst Evol Microbiol. 2017 Apr;67(4):1011-1017. doi: 10.1099/ijsem.0.001732. Epub 2017 May 5.
Desulfuribacillus stibiiarsenatis sp. nov., an obligately anaerobic, dissimilatory antimonate- and arsenate-reducing bacterium isolated from anoxic sediments, and emended description of the genus Desulfuribacillus.
Abin CA1, Hollibaugh JT2.

https://www.ncbi.nlm.nih.gov/pubmed/24319985
Environ Sci Technol. 2014;48(1):681-8. doi: 10.1021/es404098z. Epub 2013 Dec 17.
Dissimilatory antimonate reduction and production of antimony trioxide microcrystals by a novel microorganism.
Abin CA1, Hollibaugh JT.
- a bacterium that is capable of using antimonate [Sb(V)] as a terminal electron acceptor for anaerobic respiration, resulting in the precipitation of antimonite [Sb(III)] as microcrystals of antimony trioxide. The bacterium, designated strain MLFW-2, is a sporulating member of a deeply branching lineage within the order Bacillales (phylum Firmicutes).


https://www.ncbi.nlm.nih.gov/nuccore/MJAT00000000
Desulfuribacillus stibiiarsenatis strain MLFW-2, whole genome shotgun sequencing project

	$grep "MLFW-2" assembly_summary_refseq.txt 
	GCF_001742305.1	PRJNA224116	SAMN05771867	MJAT00000000.1	representative genome	1390249	1390249	Desulfuribacillus stibiiarsenatis	strain=MLFW-2		latest	Contig	Major	Full2016/09/22	ASM174230v1	University of Georgia	GCA_001742305.1	identical	ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/001/742/305/GCF_001742305.1_ASM174230v1		assembly from type material
	
	$grep "MLFW-2" prokaryotes.txt 
	Desulfuribacillus stibiiarsenatis	1390249	PRJNA343013	343013	Terrabacteria group	Firmicutes	3.1197	38.2	-	MJAT01	40	3076	2935	2016/09/22	2017/04/13	Contig	University of Georgia	SAMN05771867	GCA_001742305.1	REFR	ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/001/742/305/GCA_001742305.1_ASM174230v1	-	MLFW-2

#### kaken
2015-04-01 – 2018-03-31
https://kaken.nii.ac.jp/ja/grant/KAKENHI-PROJECT-15K14594/
KAKEN — 研究課題をさがす | 毒性元素の生物利用機構から解き明かす微生物エネルギー代謝の多様性進化 (KAKENHI-PROJECT-15K14594)
濱村 奈津子
ヒ素と同族の毒性元素であるアンチモンに着目し、平成27年度は高濃度のアンチモンで汚染した環境試料からアンチモン代謝微生物群の集積培養を試みた。
これまでに分離が報告されているアンチモン還元菌は2株のみであるが、本研究でアンチモン還元活性を示したコンソーシアからは、ヒ素やその他金属の還元菌と系統的類似度の高い多数のFirmicutes属の16S rRNA遺伝子配列とともに、アンチモン酸化および還元菌として報告のあるα-Proteobacteriaに近縁な配列が検出されたことから、これらの菌がアンチモン還元に関与している可能性が高い。

https://kaken.nii.ac.jp/ja/grant/KAKENHI-PROJECT-15K12230/
KAKEN — 研究課題をさがす | 微生物機能を利用した廃水からのアンチモン除去技術の基礎開発 (KAKENHI-PROJECT-15K12230)

アンチモン還元菌の単離と特徴づけ

----------
## halogen

https://ja.wikipedia.org/wiki/第17族元素
halogen ハロゲン）は周期表において第17族に属する元素の総称。フッ素・塩素・臭素・ヨウ素・アスタチン・テネシンがこれに分類される。

https://www.genome.jp/dbget-bin/www_bget?ko:K01560	
2-haloacid dehalogenase [EC:3.8.1.2]

https://www.ncbi.nlm.nih.gov/pubmed/7944368
Appl Environ Microbiol. 1994 Sep;60(9):3375-80.
Comparative studies of genes encoding thermostable L-2-halo acid dehalogenase from Pseudomonas sp. strain YL, other dehalogenases, and two related hypothetical proteins from Escherichia coli.
Nardi-Dei V1, Kurihara T, Okamura T, Liu JQ, Koshikawa H, Ozaki H, Terashima Y, Esaki N, Soda K.

https://repository.kulib.kyoto-u.ac.jp/dspace/bitstream/2433/182417/1/ynogk01019.pdf
Catalytic Mechanism of L-2-Haloacid Dehalogenase from Pseudomonas sp.YL

https://pdbj.org/mom/220
220: 脱ハロゲン酵素 (Dehalogenases) - 今月の分子 - PDBj

最初の酵素は ハロアルカン脱ハロゲン酵素 (haloalkane Dehalogenases、PDBエントリー 2dhc )で、一方の塩素原子を取り除きヒドロキシ基と置き換える。次に別の2つの酵素がヒドロキシ基を酸に変換する(ここには示していない)。最後に ハロ酸脱ハロゲン酵素 (haloacid Dehalogenases、PDBエントリー 1jud )がもう一つの塩素原子を取り除く。

http://www.bio.nite.go.jp/mifup/wiki/index.php?title=ハロアルカン分解
ハロアルカンデハロゲナーゼ (EC 3.8.1.5) は α/β-hydrolase family に属する加水分解酵素で、アルコールとハロゲン化水素を生じる反応を触媒する。 生じたアルコールは alcohol dehydrogenase、および aldehyde dehydrogenase により haloacetate に変換され、haloacetate dehalogenase (EC 3.8.1.3) により glycolate に、haloalkanoic acid dehalogenase (EC 3.8.1.2) により 2-hydroxyacid に代謝される。

- NRULE_0051 (S)-2-haloacid dehalogenase
- NRULE_0075 Haloacetate dehalogenase

https://www.ncbi.nlm.nih.gov/pubmed/27347470
Springerplus. 2016 May 20;5(1):695. doi: 10.1186/s40064-016-2328-9. eCollection 2016.
l-2-Haloacid dehalogenase (DehL) from Rhizobium sp. RC1.
Adamu A1, Wahab RA2, Huyop F1.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4899344/

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4899344/table/Tab1/?report=objectonly
Table 1
Known haloacid-dehalogenating bacteria and their dehalogenases

P. putida PP3
Pseudomonas putida 109
Pseudomonas sp. 113
Pseudomonas sp. CBS3
P. putida strain AJ1
Pseudomonas fluorescens 1
Pseudomonas acidovoran
Pseudomonas sp. YL
Pseudomonas sp. R1
Pseudomonas sp. B6P
P. putida S3

Pseudomonas putida 109
Pseudomonas sp. 113
Pseudomonas sp. CBS3


https://link.springer.com/chapter/10.1007/978-3-662-06066-7_9
Bacterial Reductive Dehalogenases | SpringerLink
MB Habash 著 - ‎2004 - 
This chapter examines bacterial reductive dehalogenases that catalyze the substitution of a halogen substituent, typically a chlorine, bromine or iodine atom, with a hydrogen atom.

https://www.ncbi.nlm.nih.gov/pubmed/15171686
J Endocrinol. 2004 Jun;181(3):385-92.
Comparative kinetic characterization of rat thyroid iodotyrosine dehalogenase and iodothyronine deiodinase type 1.
Solís-S JC1, Villalobos P, Orozco A, Valverde-R C.
A distinct group of dehalogenases, the so-called iodothyronine deiodinases (IDs), that specifically remove iodine atoms from iodothyronines were subsequently discovered and have been extensively characterized.
The results reported here support the view that the rat thyroid gland has a distinctive set of dehalogenases specialized in iodine metabolism.

https://www.ncbi.nlm.nih.gov/pubmed/15289438
FASEB J. 2004 Oct;18(13):1574-6. Epub 2004 Aug 2.
Iodotyrosine dehalogenase 1 (DEHAL1) is a transmembrane protein involved in the recycling of iodide close to the thyroglobulin iodination site.
In the thyroid, iodotyrosine dehalogenase acts on the mono and diiodotyrosines released during the hydrolysis of thyroglobulin to liberate iodide, which can then reenter the hormone-producing pathways. 

https://www.ncbi.nlm.nih.gov/pubmed/10389261
Lett Appl Microbiol. 1999 Jun;28(6):445-7.
An easy colorimetric assay for screening and qualitative assessment of deiodination and dehalogenation by bacterial cultures.
Kuritz T1.
https://onlinelibrary.wiley.com/doi/full/10.1046/j.1365-2672.1999.00538.x
 using this method for a variety of bacterial cultures, including Pseudomonas and Arthrobacter, with a variety of substrates. 

https://www.ncbi.nlm.nih.gov/pubmed/3624201
J Bacteriol. 1987 Sep;169(9):4049-54.
Purification and properties of haloalkane dehalogenase from Corynebacterium sp. strain m15-3.
Yokota T, Omori T, Kodama T.
https://www.ncbi.nlm.nih.gov/pubmed/7068529
J Bacteriol. 1982 May;150(2):522-7.
Purification and properties of a new enzyme, DL-2-haloacid dehalogenase, from Pseudomonas sp.
Motosugi K, Esaki N, Soda K.

----------
## iodide

https://ja.wikipedia.org/wiki/ヨウ素
iodine
ハロゲン元素の一つ。ヨード（沃度）ともいう。
反応性は塩素、臭素より小さい。水にはあまり溶けないが、ヨウ化カリウム水溶液にはよく溶ける。これは下式のように、ヨウ化物イオンとの反応が起こることによる。

ヨウ化物イオン

https://ja.wikipedia.org/wiki/ヨウ化物
iodide) イオンは、-1の電荷を帯びたヨウ素原子である[1]。
ヨウ素はヨウ化物水溶液によく溶け、茶色の三ヨウ化物イオンを形成する。

https://ja.wikipedia.org/wiki/三ヨウ化物
triiodide）は、主に3つのヨウ素原子による多原子アニオン、三ヨウ化物イオン (I3−) を含む塩を意味する。

ヨウ素酸イオン

https://ja.wikipedia.org/wiki/ヨウ素酸塩
iodate）は、ヨウ素酸イオンを含む塩である[1]。ヨウ素酸イオン (IO3-) は、ヨウ素原子に3個の酸素原子が結合した、三角錐形構造をとる。

https://ja.wikipedia.org/wiki/ヨウ素酸
iodic acid) とは、ヨウ素のオキソ酸の一種で、化学式 HIO3 の化合物。
水溶液中では、解離してヨウ素酸イオンを生じる。

http://www.wmsym.org/archives/2015/papers/15226.pdf
WM2015 Conference, March 15-19, 2015, Phoenix, Arizona, USA.Dissimilatory Iodate Reduction by Hanford Microbial Isolates – 15226Joshua T. Ellis*, Alex Dodwell*, Danielle Saunders*, Brady Lee*, and M. Hope Lee* *Pacific Northwest National Laboratory (USA)

An iodate reducing organism, designated as Agrobacterium tumefaciens strain AD35, was isolated from

### iodate reduction
ヨウ素 還元

https://twitter.com/jcamthrash/status/1107773917534392321
Cameron Thrash on Twitter: "Dalton Hardisty is talking about iodate as a redox tracer in marine systems in today’s @USC_earth seminar."
6:40 PM - 18 Mar 2019
Btw, Shewanella, Pseudomonas, and Desulfovibrio can all reduce iodate.

https://www.ncbi.nlm.nih.gov/pubmed/30446562
Appl Environ Microbiol. 2019 Jan 23;85(3). pii: e02115-18. doi: 10.1128/AEM.02115-18. Print 2019 Feb 1.
Metal Reduction and Protein Secretion Genes Required for Iodate Reduction by Shewanella oneidensis.
Toporek YJ1, Mok JK1, Shin HD1, Lee BD2, Lee MH3, DiChristina TJ4.

28 Mar 2018
https://www.tandfonline.com/doi/abs/10.1080/01490451.2018.1430189?journalCode=ugmb20
Iodate Reduction by Shewanella oneidensis Does Not Involve Nitrate Reductase
Jung Kee Mok, Yael J. Toporek, Hyun-Dong Shin, Brady D. Lee, M. Hope Lee & Thomas J. DiChristina
https://pdfs.semanticscholar.org/213e/63ea4dfb579e2e08f1a3e8d74eedbc0e03e2.pdf

----------

## Bifidobacterium

https://www.ncbi.nlm.nih.gov/pubmed/29222102
Appl Environ Microbiol. 2018 Jan 31;84(4). pii: e02249-17. doi: 10.1128/AEM.02249-17. Print 2018 Feb 15.
Tracking the Taxonomy of the Genus Bifidobacterium Based on a Phylogenomic Approach.
Lugli GA#1, Milani C#1, Duranti S1, Mancabelli L1, Mangifesta M2, Turroni F1,3, Viappiani A2, van Sinderen D4,5, Ventura M6,3.

https://www.ncbi.nlm.nih.gov/pubmed/28855614
Sci Rep. 2017 Aug 30;7(1):9926. doi: 10.1038/s41598-017-10546-0.
Predicting synonymous codon usage and optimizing the heterologous gene for expression in E. coli.
Tian J1, Yan Y1,2, Yue Q1,3, Liu X1, Chu X1, Wu N4, Fan Y1.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5577221/
Figure 1
Clustering results of the codon usage pattern of different species. The row and column represent the codon usage pattern and the different bacterial subspecies.
 57 Bifidobacterium animalis, 58 Bifidobacterium longum, 

https://www.ncbi.nlm.nih.gov/pubmed/25842224
World J Microbiol Biotechnol. 2015 Jun;31(6):959-81. doi: 10.1007/s11274-015-1850-1. Epub 2015 Apr 5.
Comparative investigation of the various determinants that influence the codon and amino acid usage patterns in the genus Bifidobacterium.
Roy A1, Mukhopadhyay S, Sarkar I, Sen A.

----------

## Clostridia
https://ja.wikipedia.org/wiki/クロストリジウム綱
クロストリジウム綱（Clostridia）は、クロストリジウム属（Clostridium）および他の同様の属を含み、高度な多系統群であるフィルミクテス門の綱である。

https://www.ncbi.nlm.nih.gov/pubmed/31076745
Genome Biol Evol. 2019 May 10. pii: evz096. doi: 10.1093/gbe/evz096. [Epub ahead of print]
Revisiting the evolution and taxonomy of Clostridia, a phylogenomic update.
Cruz-Morales P1, Orellana CA1, Moutafis G2, Moonen G2, Rincon G2, Nielsen LK1, Marcellin E1.

## Clostridium
- https://ja.wikipedia.org/wiki/クロストリジウム属

https://twitter.com/NatureRevMicro/status/1034357487355539456
Nature Rev Microbiol on Twitter: "A standardized bacterial taxonomy based on genome phylogeny substantially revises the tree of life @NatureBiotech https://t.co/Hro7pynEPQ… https://t.co/ev7En4LKdL"
4:30 AM - 28 Aug 2018

https://www.nature.com/articles/nbt.4229

This inconsistency is partly attributable to historical phenotype-based classification, as exemplified by the clostridia: microorganisms sharing morphological similarities have been erroneously classified in the genus Clostridium9,10.

The genus Clostridium was the most polyphyletic, representing 121 genera spanning 29 families, and was followed by Bacillus (81 genera across 25 families) and Eubacterium (30 genera across 8 families). 

The genus Clostridium is widely acknowledged to be polyphyletic, and efforts have been made to rectify this problem, including a global attempt to reclassify the genus by using a combination of phylogenetic markers9. The authors of that study have proposed the reclassification of 78 Clostridium species, and nine other species, into six novel genera9,52. Of these, we confirmed that Erysipelatoclostridium (with the exception of Clostridium innocuum str. 2959), Gottschalkia and Tyzzerella (excepting Clostridium nexile CAG:348) represent monophyletic genus-level groups. 

Similarly to its type genus, the order Clostridiales was extensively subdivided (Fig. 5a), largely as a consequence of an anomalous RED for this rank (Fig. 2a).

Figure 5: Comparisons of NCBI and GTDB classifications of genomes designated as Clostridia or Bacteroidetes in the GTDB taxonomy.

(a) Comparison of NCBI (left) and GTDB (right) order-level classifications of the 2,368 bacterial genomes assigned to the class Clostridia in the GTDB taxonomy. Genomes classified in a class other than Clostridia by NCBI are indicated in parentheses. 

9.
Yutin, N. & Galperin, M.Y. A genomic update on clostridial phylogeny: Gram-negative spore formers and other misplaced clostridia. Environ. Microbiol. 15, 2631–2641 (2013).


## Clostridium difficile
- https://ja.wikipedia.org/wiki/クロストリジウム・ディフィシル腸炎
- https://ja.wikipedia.org/wiki/クロストリジウム・ディフィシル
- https://institute.yakult.co.jp/bacteria/4205/
クロストリジウム ディフィシル | 菌の図鑑 | ヤクルト中央研究所

9:03 AM · Aug 7, 2020
https://twitter.com/waki1711/status/1291525456328450048
松永 和紀 on Twitter: "「危ない」という論文情報は拡散しやすいけれど、「そんなことは言えない」論文の情報は、メディアにはほとんど取り上げられなかった……。そんな流れの中で奮闘する企業の話です。新型コロナでも、似たようなことがたびたび起きていますね。https://t.co/UuB6oUQbP3 @WEDGE_Infinityより" / Twitter

2020年8月7日
https://wedge.ismedia.jp/articles/-/20415
ネイチャーが招いた風評。トレハロース問題の顛末は 　WEDGE Infinity(ウェッジ)
松永和紀 （科学ジャーナリスト）


2018/02/13
https://gunosy.com/articles/ao7SQ
「トレハロース問題」の真相、「感染症の原因に」論文は矛盾だらけ ネットに氾濫する不十分情報に注意を

2018年2月13日
https://lab.mykinso.com/news/180213_1/
トレハロースで増えるクロストリジウム・ディフィシル腸炎強毒性株

2018/01/24
https://www.hayashibara.co.jp/data/1416/press_tp/
雑誌Natureに掲載されたトレハロースに関する論文について｜株式会社 林原

2018年1月7日
http://aasj.jp/news/watch/7897
１月７日：食物中のトレハロースがクロストリジウム強毒株進化を促進した（Natureオンライン版掲載論文） | AASJホームページ

https://www.ncbi.nlm.nih.gov/pubmed/29310122
Nature. 2018 Jan 18;553(7688):291-294. doi: 10.1038/nature25178. Epub 2018 Jan 3.
Dietary trehalose enhances virulence of epidemic Clostridium difficile.
Collins J1, Robinson C2, Danhof H1, Knetsch CW3, van Leeuwen HC3, Lawley TD4, Auchtung JM1, Britton RA1.

米国ではC. difficileは、メチシリン耐性黄色ブドウ球菌（MRSA）以上に主要な医療関連感染の原因微生物として認識されるようになっている。近年では、CDIは、欧米において、、リボタイプ078株などに代表されるいわゆる強毒株による重症例や ...
[PDF]ディフィシル菌感染症の基礎と臨床
www.eiken.co.jp/modern_media/backnumber/pdf/MM1010_01.pdf
神谷茂 著 - ‎被引用数: 8 - ‎関連記事
（Clostridium difficile）は一部の健. 常者の腸内

ディフィシル菌（C.difficile）リボタイプ027株の比較ゲノムにより、台湾の患者から分離されたNCKUH-21株に特異的なプロファージを発見
2017-11-29
https://pubmed.ncbi.nlm.nih.gov/29213333/
Case Reports Gut Pathog
. 2017 Nov 29;9:70. doi: 10.1186/s13099-017-0219-4. eCollection 2017.
Comparative genomic analysis of Clostridium difficile ribotype 027 strains including the newly sequenced strain NCKUH-21 isolated from a patient in Taiwan
Haruo Suzuki 1 2, Masaru Tomita 1 2, Pei-Jane Tsai 3, Wen-Chien Ko 4, Yuan-Pin Hung 5 6, I-Hsiu Huang 7, Jenn-Wei Chen 7
https://gutpathogens.biomedcentral.com/articles/10.1186/s13099-017-0219-4
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC5708112/
[Additional file 1. Materials and methods.(43K, docx)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5708112/bin/13099_2017_219_MOESM1_ESM.docx)

https://www.newsweekjapan.jp/stories/world/2018/04/ni150.php
うんちを寄付して報酬が年150万円
健康な人のうんちに含まれる腸内細菌を患者の腸に注入することで腸内環境を整える「便移植療法」
「クロストリジウム・ディフィシル感染症」（CDI）

2017.12.25
https://medley.life/news/5a3c53f7af9647751ea32743/
プロバイオティクスはC. ディフィシル関連下痢症を予防できるか？

2017年12月25日
https://www.newsweekjapan.jp/stories/world/2017/12/post-9194.php
「うんちカプセル」のすご過ぎる効果

2017/2/20
http://yfuruta.sakura.ne.jp/blog/?p=839
糞便移植と糞便バンク

2017/02/03
http://www.carenet.com/news/general/hdn/43324
腸感染症に対する「糞便移植」の効果に疑問

2016年9月29日
http://www.natureasia.com/ja-jp/nature/pr-highlights/11232
微生物学：クロストリジウム・ディフィシル感染症の治療標的と考えられる受容体が新たに判明 | Nature

2016年4月7日 
http://www.natureasia.com/ja-jp/nrdp/primer/74132
クロストリジウム・ディフィシル感染症
Clostridium difficile infection

http://journal.kyorin.co.jp/journal/jscm/detail_j.php?-DB=jscm&-recid=981&-action=browse
糞便中Clostridium difficile毒素産生関連遺伝子検出キットVerigene CDFパネルの臨床的性能評価
臨床微生物：27（3），188─194，2017

http://www.chemotherapy.or.jp/guideline/index.html
特定感染症サーベイランス
「Clostridium difficile 感染症“1日”多施設共同研究」
日本化学療法学会雑誌 Vol. 65, 2017年1号（1月）　p.1～3

2016/12/24
https://www.nikkei.com/article/DGXMZO10934990R21C16A2000000/
腸内細菌は宝の山　薬剤耐性菌に効く新薬も　　：日本経済新聞
薬剤耐性菌、再発性クロストリジウム・ディフィシルの治療で、健康な人から採取した便を生理食塩水に溶解して、抗菌剤とともに患者に投与

平成28年2月8日(月)
https://www.alic.go.jp/content/000122381.pdf
腸内フローラと食品・乳製品
ディフィシル

2015.10.12
https://www.cnn.co.jp/fringe/35071695.html
ふん便が薬に、新治療「ふん便移植」が脚光　９割で症状改善 

http://terra58.hatenablog.com/entry/2015/01/04/134521
ディフィシル菌（Clostridium difficile） - てら５８



2015/3/13
http://medical.nikkeibp.co.jp/leaf/mem/pub/hotnews/nejm/201503/541189.html
C.ディフィシル感染症、米で年間45万人が発症：日経メディカル

2015年1月8日
http://www.natureasia.com/ja-jp/nature/highlights/59738
感染症：	腸内細菌がClostridium difficile耐性をもたらす

2014年11月14日
http://first.lifesciencedb.jp/archives/9479
インターロイキン22は補体系を増強することにより腸管上皮の傷害から侵入する細菌に対し抵抗性をあたえる : ライフサイエンス 新着論文レビュー

http://www.micks.jp/jhi/2014/12/20141206.html
JHIサマリー日本語版: クロストリジウム・ディフィシル（Clostridium difficile）リボタイプ 027 は長期ケア施設から転院した入院患者での保有率が高い

http://www.micks.jp/jhi/2010/08/20100808.html
JHIサマリー日本語版: クロストリジウム・ディフィシル（Clostridium difficile）リボタイプ078による感染：アイルランド共和国における初の症例報告

2013/10/10
http://www.carenet.com/news/journal/carenet/36333
ゲノム解析で判明、C. difficile感染の伝播経路は複数存在／NEJM

2013年4月15日
http://www.city.yokohama.lg.jp/kenko/eiken/idsc/disease/clostridium1.html
横浜市衛生研究所：クロストリジウム-ディフィシル感染症について

2012年12月10日
http://www.natureasia.com/ja-jp/ng/pr-highlights/1996
医療に関連したClostridium difficileの流行 | Nature Genetics

2010年10月
http://www.eiken.co.jp/modern_media/backnumber/pdf/MM1010_01.pdf
ディフィシル菌感染症の基礎と臨床

2006年7月1日
http://www.natureasia.com/ja-jp/ng/38/7/ng1830/
クロストリジウム-ディフィシレ菌のゲノム | Nature Genetics

2005年11月
http://www.bdj.co.jp/safety/articles/ignazzo/1f3pro00000sj5kj.html
I's eye： クロストリジウム・ディフィシルの真実 | 日本BD

----------
## Lactobacillus

### Lactobacillus salivarius
https://www.sciencedirect.com/topics/medicine-and-dentistry/lactobacillus-salivarius

https://ja.wikipedia.org/wiki/乳酸菌LS1
(= Lactobacillus salivarius TI2711)

https://ja.wikipedia.org/wiki/プロバイオティクス
プロバイオティクス（Probiotics）とは、人体に良い影響を与える微生物（善玉菌）、または、それらを含む製品、食品のこと。

宿主適応に焦点を当てたLactobacillus salivarius株の比較ゲノム
https://www.ncbi.nlm.nih.gov/pubmed/28942844
Microbiol Res. 2017 Dec;205:48-58. doi: 10.1016/j.micres.2017.08.008. Epub 2017 Aug 24.
Comparative genomics of Lactobacillus salivarius strains focusing on their host adaptation.
Lee JY1, Han GG1, Kim EB2, Choi YJ3.
https://www.sciencedirect.com/science/article/pii/S0944501317303531

哺乳類の腸内細菌Lactobacillus salivariusの比較ゲノム。レプリコン（染色体とプラスミド）
https://www.ncbi.nlm.nih.gov/pubmed/29026656
Microb Genom. 2017 Jun 13;3(8):e000115. doi: 10.1099/mgen.0.000115. eCollection 2017 Aug.
Phylogenomics and comparative genomics of Lactobacillus salivarius, a mammalian gut commensal.
Harris HMB1, Bourin MJB1, Claesson MJ1, O'Toole PW1.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5610712/
- Lactobacillus salivarius, a species commonly isolated from the gastrointestinal tract of humans and animals, has been described as having potential probiotic properties and results of previous studies have revealed considerable functional diversity existing on both the chromosomes and plasmids. 
- The greatest genomic variation lies on the small plasmids, followed by the repA-type circular megaplasmid, with the chromosome varying least of all. Additionally, the presence of extra linear and circular megaplasmids is noted in several strains, while small plasmids are not always present. 
- Lactobacillus hayakitensis DSM18933T was also included in the study as a related outgroup. 
- Fig. 2.
A phylogenetic tree generated from 938 core genes shows considerable variation in divergence across strains. Branch lengths (continuous black lines) represent evolutionary divergence and strain labels are lined up for ease of comparison (dashed lines). Bootstrap values are included to show robustness of tree topology. The tree is rooted on L. hayakitensis DSM18933T and this branch is artificially reduced to provide a clearer visualisation of the other branch lengths relative to each other. Bar, average number of amino acid substitutions per site.

![](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5610712/bin/mgen-3-115-g002.jpg)

```
$grep "hayakitensis" prokaryotes.txt | cut -f1,16,17
Lactobacillus hayakitensis DSM 18933 = JCM 14209	Contig	The University of Tokyo
Lactobacillus hayakitensis DSM 18933 = JCM 14209	Scaffold	Shanghai Majorbio
```

https://www.ncbi.nlm.nih.gov/pubmed/29942291
Front Microbiol. 2018 Jun 11;9:1244. doi: 10.3389/fmicb.2018.01244. eCollection 2018.
Comparative Genomics of Lactobacillus acidipiscis ACA-DC 1533 Isolated From Traditional Greek Kopanisti Cheese Against Species Within the Lactobacillus salivarius Clade.
Kazou M1, Alexandraki V1, Blom J2, Pot B3, Tsakalidou E1, Papadimitriou K1.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6004923/
- Lactobacillus acidipiscis belongs to the Lactobacillus salivarius clade and it is found in a variety of fermented foods. 
- FIGURE 1. Whole genome phylogenetic tree for representative species of the L. salivarius clade. The tree was calculated based on the core-genome and it is presented as a cladogram. Species were separated in two main groups, namely A and B, as depicted in the figure. Weissella kandleri DSM 20593T and Lactobacillus delbrueckii subsp. bulgaricus ATCC 11842T were used as outgroups.

![](https://www.frontiersin.org/files/Articles/324270/fmicb-09-01244-HTML/image_m/fmicb-09-01244-g001.jpg)

## vaginal microbiota
膣微生物

https://www.ncbi.nlm.nih.gov/pubmed/30291903
Anaerobe. 2018 Oct 3. pii: S1075-9964(18)30173-2. doi: 10.1016/j.anaerobe.2018.10.001. [Epub ahead of print]
Characterization of the vaginal microbiota of Japanese women.
Matsumoto A1, Yamagishi Y2, Miyamoto K3, Oka K4, Takahashi M4, Mikamo H5.

日本の非妊婦、妊婦、性産業従事者(CSWs)の膣微生物群集の構成を比較した。微生物多様性は妊婦よりも非妊婦で有意に高かった。Lactobacillus検出率は、妊婦（50％）や非妊婦（68.2％）よりも性産業従事者(25％)で低かった。

The vaginal microbiota were clustered into three groups. Group 1 was most often dominated by Lactobacillus spp., whereas groups 2 and 3 included not only Lactobacillus spp. but also Bifidobacterium, Atopobium, Prevotella, and Gardnerella spp., in addition to a few other taxa.

The alpha diversity of non-pregnant women was significantly greater than that of pregnant women. The detection rate of live Lactobacillus spp. in CSWs was lower than in pregnant and non-pregnant women (25% vs. 50% and 68.2%, respectively). 


Gardnerella vaginalis：細菌性膣症（bacterial vaginosis: BV）に関与する細菌の一つ
https://www.ncbi.nlm.nih.gov/pubmed/29632552
Evol Appl. 2017 Nov 16;11(3):312-324. doi: 10.1111/eva.12555. eCollection 2018 Mar.
Focusing the diversity of Gardnerella vaginalis through the lens of ecotypes.
Cornejo OE1, Hickey RJ2,3,4, Suzuki H5, Forney LJ2,3.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5881158/
- These data suggest the genomes of Gardnerella have undergone a substantial reduction in size after diverging from their Bifidobacterium relatives. 
- G. vaginalis genomes are substantially smaller and have a significantly lower GC content when compared to Bifidobacterium, a pattern that has previously been observed in many bacterial symbionts (McCutcheon & Moran, 2012). 

Lactobacillus属：膣から分離された種は、他の環境から分離された種に比べ、ゲノムのサイズが小さく、GC含有量も低かった。
https://pubmed.ncbi.nlm.nih.gov/24488312/
Comparative Study J Bacteriol
. 2014 Apr;196(7):1458-70. doi: 10.1128/JB.01439-13. Epub 2014 Jan 31.
Comparative functional genomics of Lactobacillus spp. reveals possible mechanisms for specialization of vaginal lactobacilli to their environment
Helena Mendes-Soares 1, Haruo Suzuki, Roxana J Hickey, Larry J Forney
https://journals.asm.org/doi/10.1128/JB.01439-13
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3993339/
We found that the genomes of the vaginal species were significantly smaller and had significantly lower GC content than those of the nonvaginal species.


Lactobacillus属：体重増加に関連
https://pubmed.ncbi.nlm.nih.gov/24567124/
Nutr Diabetes
. 2014 Feb 24;4(2):e109. doi: 10.1038/nutd.2014.6.
Comparative genomics analysis of Lactobacillus species associated with weight gain or weight protection
F Drissi 1, V Merhej 1, E Angelakis 1, A El Kaoutari 1, F Carrière 2, B Henrissat 3, D Raoult 1
https://www.nature.com/articles/nutd20146
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3940830/



----------
## Mycobacterim

https://ja.wikipedia.org/wiki/抗酸菌

http://jsbac.org/
日本細菌学会｜Japanese Society for Bacteriology
http://jsbac.org/infectious_disease/bsl_level.pdf
```
病原細菌のBSLレベル （アルファベット順）
属名 種形容語 亜種形容語 血清型等 由来等　(旧名等） 細菌学会BSL分類

Helicobacter pylori 旧名＝Campylobacter pylori, Campylobacter pyloridis BSL2 Class 2

Lactobacillus catenaformis BSL1*
Lactobacillus gasseri BSL1*
Lactobacillus rhamnosus BSL1*
Lactococcus garvieae BSL1*

Mycobacterium avium avium BSL2 Class 2
Mycobacterium avium paratuberculosis 旧名＝Mycobacterium paratuberculosis BSL2 Class 2
Mycobacterium avium silvaticum BSL2
```

----------

## Pseudomonas
- https://ja.wikipedia.org/wiki/シュードモナス属
- https://en.wikipedia.org/wiki/Pseudomonas

- review

https://www.ncbi.nlm.nih.gov/pubmed/21361996
FEMS Microbiol Rev. 2011 Jul;35(4):652-80. doi: 10.1111/j.1574-6976.2011.00269.x. Epub 2011 Mar 25.
Pseudomonas genomes: diverse and adaptable.
Silby MW1, Winstanley C, Godfrey SA, Levy SB, Jackson RW.

Members of the genus Pseudomonas inhabit a wide variety of environments, which is reflected in their versatile metabolic capacity and broad potential for adaptation to fluctuating environmental conditions. 

https://onlinelibrary.wiley.com/doi/full/10.1111/j.1574-6976.2011.00269.x
Bioremediation 
Pseudomonas stutzeri– a nitrogen‐fixing nonfluorescent pseudomonad

- 分類
https://www.ncbi.nlm.nih.gov/pubmed/21811795
Microb Ecol. 2012 Feb;63(2):239-48. doi: 10.1007/s00248-011-9914-8. Epub 2011 Aug 3.
Defining the Pseudomonas genus: where do we draw the line with Azotobacter?
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3275731/
 [59]. 
Owing to the low resolution in 16S rRNA sequences for these genomes, they conducted a phylogenetic analysis of 25 protein-coding genes, some of which were housekeeping genes.

Although they are clustered together in the 16S rRNA tree, the low resolution of the 16S rRNA phylogenetic analysis has been noted by others [39, 59]. It does show, however, that they have a common ancestral rrs gene that is closer to each other than to other species. 

----------
### Pseudomonas aeruginosa

https://www.ncbi.nlm.nih.gov/pubmed/28606056
BMC Genomics. 2017 Jun 12;18(1):459. doi: 10.1186/s12864-017-3842-z.
Comparative genome and transcriptome analysis reveals distinctive surface characteristics and unique physiological potentials of Pseudomonas aeruginosa ATCC 27853.
Cao H1, Lai Y2,3, Bougouffa S4, Xu Z2, Yan A5.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5467263/
Prophages

https://www.ncbi.nlm.nih.gov/pubmed/25961859
PLoS One. 2015 May 11;10(5):e0126468. doi: 10.1371/journal.pone.0126468. eCollection 2015.
What It Takes to Be a Pseudomonas aeruginosa? The Core Genome of the Opportunistic Pathogen Updated.

https://www.ncbi.nlm.nih.gov/pubmed/25168460
BMC Genomics. 2014 Aug 29;15:737. doi: 10.1186/1471-2164-15-737.
Characterization of the core and accessory genomes of Pseudomonas aeruginosa using bioinformatic tools Spine and AGEnt.

https://www.ncbi.nlm.nih.gov/pubmed/21119020
Microbiol Mol Biol Rev. 2010 Dec;74(4):621-41. doi: 10.1128/MMBR.00027-10.
The accessory genome of Pseudomonas aeruginosa.
Kung VL1, Ozer EA, Hauser AR.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3008168/
The majority of the P. aeruginosa accessory genome can be grouped into four broad categories: (i) integrative and conjugative elements (ICEs), (ii) replacement islands, (iii) prophages and phage-like elements, and (iv) transposons, insertion sequences (ISs), and integrons.

secretion system

- コドン使用
https://www.ncbi.nlm.nih.gov/pubmed/12036591
Gene. 2002 May 1;289(1-2):131-9.
Synonymous codon usage in Pseudomonas aeruginosa PA01.
Grocock RJ1, Sharp PM.

----------
### Pseudomonas stutzeri
https://microbewiki.kenyon.edu/index.php/Pseudomonas_stutzeri

https://ja.wikipedia.org/wiki/シュードモナス・スタッツェリ
P. stutzeriは、環境中から同種および異種のプラスミドDNAを取り込み、自身のゲノムに統合する自然形質転換[ 英: natural transformation ]をすることができる。この能力は多様な環境に適応することを可能にし、P. stutzeriは世界中の幅広い環境中から見出せる。

http://www.jarmam.gr.jp/situmon3/pseudomonas-stutzeri.html
Pseudomonas stutzeriの病原性と分布

https://www.ncbi.nlm.nih.gov/pubmed/16760312
Microbiol Mol Biol Rev. 2006 Jun;70(2):510-47.
Biology of Pseudomonas stutzeri.
Lalucat J1, Bennasar A, Bosch R, García-Valdés E, Palleroni NJ.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1489536/
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1489536/table/t1/
TABLE 1.
P. stutzeri strains cited in the text, with relevant characteristics, origins, and references

19SMN4	DSM6084	Ref, gv 4	1988	Marine sediment; naphthalene degrader; Barcelona, Spain

JM300	DSM10701	Ref, gv 8	Pre-1980	Soil; California; natural transformation model organism

CMT.9.A	DSM4166		1987	Sorghum nutans rhizosphere; nitrogen fixer; Germany	189

A15	LMG10652	gv 1	1981	Rice paddy; nitrogen fixer; Southeast Asia; formerly Alcaligenes faecalis

JM300	DSM10701	Ref, gv 8	Pre-1980	Soil; California; natural transformation model organism	60

SP1402	DSM6083	Former Ref, gv 6; P. balearica	1988	Wastewater; 2-methylnaphthalene degrader; Mallorca, Spain


- NYC地下鉄
https://www.ncbi.nlm.nih.gov/pubmed/26594662
Cell Syst. 2015 Jul 29;1(1):72-87. Epub 2015 Mar 3.
Geospatial Resolution of Human and Bacterial Diversity with City-Scale Metagenomics.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4651444/
The most prevalent bacterial species on the subway was Pseudomonas stutzeri, with enrichment in lower Manhattan (Figure 1E),

https://github.com/haruosuz/metasub/blob/master/references/README.md#new-york-subway-microbiome

- ファージ
https://www.ncbi.nlm.nih.gov/pubmed/28649138
Nat Rev Microbiol. 2017 Sep;15(9):517-530. doi: 10.1038/nrmicro.2017.61. Epub 2017 Jun 26.
Pseudomonas predators: understanding and exploiting phage-host interactions.
De Smet J1, Hendrix H1, Blasdel BG1, Danis-Wlodarczyk K1, Lavigne R1.
The Pseudomonas genus is one of the most ecologically important groups of bacteria and includes species that are plant commensals (Pseudomonas stutzeri and Pseudomonas fluorescens) and 

- ファージ
https://www.ncbi.nlm.nih.gov/pubmed/26025640
Res Microbiol. 2015 Nov;166(9):668-76. doi: 10.1016/j.resmic.2015.05.001. Epub 2015 May 27.
Identification and genomic analysis of temperate Pseudomonas bacteriophage PstS-1 from the Japan trench at a depth of 7000 m.
Yoshida M1, Yoshida-Takashima Y2, Nunoura T2, Takai K2.
A temperate bacteriophage (termed vB_PstS-1) was identified from the psychrotolerant gammaproteobacterium Pseudomonas stutzeri 1-1-1b, 

- 比較ゲノム
https://www.ncbi.nlm.nih.gov/pubmed/27233093
Environ Microbiol. 2016 Oct;18(10):3535-3549. doi: 10.1111/1462-2920.13390. Epub 2016 Jul 7.
Comparative genomic, proteomic and exoproteomic analyses of three Pseudomonas strains reveals novel insights into the phosphorus scavenging capabilities of soil bacteria.
Lidbury ID1, Murphy AR2, Scanlan DJ2, Bending GD2, Jones AM2, Moore JD3, Goodall A4, Hammond JP4,5, Wellington EM2.
Pseudomonas stutzeri DSM4166
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5082522/
Table 1
Comparative genomic analysis of selected proteins involved in the recycling of P among Pseudomonas isolates.
P. stutzeri A1501	P. stutzeri DSM4166	P. stutzeri DSM10701

- VI型分泌装置
https://www.ncbi.nlm.nih.gov/pubmed/27560648
Genet Mol Biol. 2016 Jul-Sep;39(3):465-73. doi: 10.1590/1678-4685-GMB-2015-0186. Epub 2016 Jul 7.
Comparative genomics of an endophytic Pseudomonas putida isolated from mango orchard.
Asif H1, Studholme DJ2, Khan A1, Aurongzeb M1, Khan IA1, Azim MK1.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5004826/
TypeVI secretion system (T6SS) in Pseudomonas putida TJ151

- 比較ゲノム
https://www.ncbi.nlm.nih.gov/pubmed/26519390
Appl Environ Microbiol. 2015 Oct 30;82(1):375-83. doi: 10.1128/AEM.02612-15. Print 2016 Jan 1.
Diversity of Pseudomonas Genomes, Including Populus-Associated Isolates, as Revealed by Comparative Genome Analysis.
Jun SR1, Wassenaar TM2, Nookaew I3, Hauser L3, Wanchai V3, Land M3, Timm CM3, Lu TY3, Schadt CW3, Doktycz MJ3, Pelletier DA3, Ussery DW4.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4702629/
(iv) The Azotobacter genome is found in the P. stutzeri group. The A. vinelandii DJ genome was included in this analysis because previous findings, based on comparative genomics of Azotobacter and Pseudomonas (25), showed that it was closely related. Its position is indicated by arrow B in Fig. 1 and is consistent with inclusion in the P. stutzeri group. 

- 比較ゲノム
https://www.ncbi.nlm.nih.gov/pubmed/25898829
BMC Genomics. 2015 Apr 18;16:313. doi: 10.1186/s12864-015-1488-2.
Pan-genome dynamics of Pseudomonas gene complements enriched across hexachlorocyclohexane dumpsite.
Sharma A1, Sangwan N2, Negi V3, Kohli P4, Khurana JP5, Rao DL6, Lal R7.
Pseudomonas sp. strain RL
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4405911/
largely divergent P. stutzeri group (Figure 1A). 
However, two sub-groups were identified for P. stutzeri clade, first with P. stutzeri CCUG 29243 and ATCC 14405, and second containing P. stutzeri A1501, T13 and XLDN-R (Figure 1A). 

- 比較ゲノム、可動性遺伝因子
https://www.ncbi.nlm.nih.gov/pubmed/24803113
Environ Microbiol. 2015 Jan;17(1):91-104. doi: 10.1111/1462-2920.12498. Epub 2014 May 28.
Comparative genome analysis of Pseudomonas knackmussii B13, the first bacterium known to degrade chloroaromatic compounds.
The B13 genome contains at least eight genomic islands [prophages and integrative conjugative elements (ICEs)], which were absent in closely related pseudomonads. 
https://onlinelibrary.wiley.com/doi/pdf/10.1111/1462-2920.12498
P. stutzeri DSM 4166

- ファージ
https://www.ncbi.nlm.nih.gov/pubmed/22923799
J Virol. 2012 Sep;86(18):10239. doi: 10.1128/JVI.01605-12.
Genome sequence of the broad-host-range Pseudomonas phage Φ-S1.
Sillankorva S1, Kropinski AM, Azeredo J.
The lytic phage Φ-S1 was isolated from the Greater Vancouver Sewage Processing plant using Pseudomonas fluorescens PW. This virus lysed most of the fluorescent Pseudomonas species tested and one strain of Pseudomonas stutzeri. 

- 比較ゲノム
https://www.ncbi.nlm.nih.gov/pubmed/27922098
Sci Rep. 2016 Dec 6;6:38699. doi: 10.1038/srep38699.
Comparison of 432 Pseudomonas strains through integration of genomic, functional, metabolic and expression data.
Koehorst JJ1, van Dam JC1, van Heck RG1, Saccenti E1, Dos Santos VA1,2, Suarez-Diez M1, Schaap PJ1.
http://www.nature.com/articles/srep38699
- P. aeruginosa and P. stutzeri clades are conserved while P. putida and P. fluorescence clades shows the addition of different species.
- Figure 4: Domain based distance tree of 58 Pseudomonas strains.
- The phylogenetic clusters corresponding to the most abundant species (P. stutzeri, P. mendocina, P. aeruginosa and P. putida) are colour-shadowed.

- 亜ヒ酸酸化細菌 Pseudomonas stutzeri TS44 のドラフトゲノム配列
https://www.ncbi.nlm.nih.gov/pubmed/22843599
J Bacteriol. 2012 Aug;194(16):4473-4. doi: 10.1128/JB.00907-12.
Genome sequence of the moderately halotolerant, arsenite-oxidizing bacterium Pseudomonas stutzeri TS44.
Li X1, Gong J, Hu Y, Cai L, Johnstone L, Grass G, Rensing C, Wang G.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3416245/
The genome contains at least 20 genes encoding functions related to arsenite oxidation and resistance, mainly located on contig 00030 (AJXE01000030; ars operon, arsC1-arsR-arsC2-ACR3-arsH-DSP-GAPDH-MFS, and an adjacent ars-aox operon, arsD-arsA-aioA-aioB) and contig 00031 (AJXE01000031; ars operon, ACR3-arsR-arsH-GAPDH-MFS) (9, 11, 13). Notably, aioAB (encoding arsenite oxidase) was absent in the other five P. stutzeri genomes, suggesting recent acquisition of aioAB by strain TS44 via horizontal gene transfer. However, a two-component system, aioS aioR, involved in regulating the expression of aioAB, was not identified in the genome, indicating a potentially novel regulatory mechanism (10, 14). In addition, numerous genes responsible for resistance to other metals (copper, mercury, chromate, cadmium, and zinc) were also identified.

https://www.sciencedirect.com/science/article/pii/S0964830516306515
August 2017
In silico approach for bioremediation of arsenic by structure prediction and docking studies of arsenite oxidase from Pseudomonas stutzeri TS44

- ヒ素蓄積
https://www.ncbi.nlm.nih.gov/pubmed/19568912
Environ Health Prev Med. 2008 Sep;13(5):257-63. doi: 10.1007/s12199-008-0038-9. Epub 2008 Jul 4.
Arsenic accumulation by Pseudomonas stutzeri and its response to some thiol chelators.
Joshi DN1, Patel JS, Flora SJ, Kalia K.

#### Pseudomonas stutzeri genomes

##### P.stutzeri_group

- Pseudomonas mendocina ymp

https://www.ncbi.nlm.nih.gov/nuccore/CP000680
Pseudomonas mendocina ymp, complete genome

https://www.ncbi.nlm.nih.gov/bioproject/17457
Pseudomonas mendocina ymp (ID 17457) - BioProject - NCBI

https://www.ncbi.nlm.nih.gov/bioproject/?term=Pseudomonas%20mendocina%20ymp
A bacterium which is able to degrade pentachlorophenol

https://genome.jgi.doe.gov/portal/pseme/pseme.home.html
Strain ymp was isolated as part of the Yucca Mountain Project from sediment in a surface holding pond of a drilling operation at the Nevada Test Site, NV (the proposed site of the  high-level nuclear repository).

http://bacmap.wishartlab.com/organisms/495
P.mendocina strain ymp, a strict aerobe, is a pentachlorophenol (PCP)-degrading bacterium isolated from PCP-contaminated soil.
Isolation site	sediment, surface holding pond, Nevada Test Site, NV
Isolation country	USA

- LH-42

https://www.ncbi.nlm.nih.gov/pubmed/28684578
Genome Announc. 2017 Jul 6;5(27). pii: e00589-17. doi: 10.1128/genomeA.00589-17.
Draft Genome Sequence of Pseudomonas stutzeri LH-42, Isolated from Petroleum-Contaminated Soil.
Zhang M1, Ma L1, Yang Y2, Hu T1, Liu X1, Zhang X1, Hua Y1, Gao Y1, Zhu Z1.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5502859/

- KOS6

https://www.ncbi.nlm.nih.gov/pubmed/23405317
Genome Announc. 2013 Jan;1(1). pii: e00072-12. doi: 10.1128/genomeA.00072-12. Epub 2013 Jan 31.
Draft Genome of the Nitrogen-Fixing Bacterium Pseudomonas stutzeri Strain KOS6 Isolated from Industrial Hydrocarbon Sludge.
Grigoryeva TV1, Laikov AV, Naumova RP, Manolov AI, Larin AK, Karpova IY, Semashko TA, Alexeev DG, Kostryukova ES, Müller R, Govorun VM.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3569306/
Strain KOS6 was most closely related to strain CCUG 29243 (blast total score, 4.560e+06, 70% query coverage), 

- Pseudomonas balearica DSM 6083T

https://www.ncbi.nlm.nih.gov/pubmed/27103708
Genome Announc. 2016 Apr 21;4(2). pii: e00217-16. doi: 10.1128/genomeA.00217-16.
Complete Genome Sequence of Pseudomonas balearica DSM 6083T.
Bennasar-Figueras A1, Salvà-Serra F2, Jaén-Luchoro D3, Seguí C3, Aliaga F4, Busquets A3, Gomila M3, Moore ER5, Lalucat J3.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4841123/

##### P.stutzeri_1

- RCH2

https://www.ncbi.nlm.nih.gov/nuccore?term=60029%5BBioProject%5D

https://www.ncbi.nlm.nih.gov/pubmed/28194258
Stand Genomic Sci. 2017 Feb 8;12:23. doi: 10.1186/s40793-017-0233-7. eCollection 2017.
Complete genome sequence of Pseudomonas stutzeri strain RCH2 isolated from a Hexavalent Chromium [Cr(VI)] contaminated site.
Chakraborty R1, Woo H2, Dehal P1, Walker R1, Zemla M1, Auer M1, Goodwin LA3, Kazakov A1, Novichkov P1, Arkin AP1, Hazen TC1,2.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5299692/
and includes 3 circular plasmids of 12,763 bp, 9,865 bp and 2,804 bp for a total genome size of 4,600,489 bp. 
No orthologs for plasmid genes of RCH2 were found in five other P. stutzeri strains. 

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5299692/table/Tab1/?report=objectonly
MIGS-6	Habitat	Cr(VI) contaminated aquifer	IDA

- 19SMN4

https://www.ncbi.nlm.nih.gov/nuccore/CP007509
Pseudomonas stutzeri strain 19SMN4 , complete genomes

https://www.ncbi.nlm.nih.gov/nuccore/CP007510
Pseudomonas stutzeri strain 19SMN4 plasmid pLIB119, complete plasmid

https://www.ncbi.nlm.nih.gov/bioproject/PRJNA242326
Pseudomonas stutzeri strain:19SMN4 (ID 242326) - BioProject - NCBI

https://www.ncbi.nlm.nih.gov/pubmed/8161187
Appl Environ Microbiol. 1994 Mar;60(3):966-72.
Comparative biochemical and genetic analysis of naphthalene degradation among Pseudomonas stutzeri strains.
Rosselló-Mora RA1, Lalucat J, García-Valdés E.

- CCUG 29243

https://www.ncbi.nlm.nih.gov/nuccore/CP003677
Pseudomonas stutzeri CCUG 29243, complete genome

https://www.ncbi.nlm.nih.gov/pubmed/23144395
J Bacteriol. 2012 Dec;194(23):6642-3. doi: 10.1128/JB.01753-12.
Complete genome sequence of the naphthalene-degrading bacterium Pseudomonas stutzeri AN10 (CCUG 29243).
Brunet-Galmés I1, Busquets A, Peña A, Gomila M, Nogales B, García-Valdés E, Lalucat J, Bennasar A, Bosch R.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3497477/
Finally, a putative prophage sequence (42.7 kb in length) was also detected within the 400-kb region (coordinates 1,476,863 to 1,524,049). All these data suggest that this region might be a specific area of the chromosome used to stabilize foreign genetic material. In relation to this, as observed in other P. stutzeri strains (4, 16), the complete repertoire of genes needed for the acquisition of foreign DNA by natural transformation (pilA, pilBCD, pilTU, exbB, and comA) has been detected.

##### P.stutzeri_2

- SLG510A3-8

https://www.ncbi.nlm.nih.gov/nuccore/CP011854
Pseudomonas stutzeri strain SLG510A3-8, complete genome

https://www.ncbi.nlm.nih.gov/pubmed/26144046
J Biotechnol. 2015 Oct 10;211:1-2. doi: 10.1016/j.jbiotec.2015.06.421. Epub 2015 Jul 2.
Complete genome sequence of the petroleum-emulsifying bacterium Pseudomonas stutzeri SLG510A3-8.
Hu B1, Nie Y1, Geng S1, Wu XL2.
https://linkinghub.elsevier.com/retrieve/pii/S0168-1656(15)30055-9

- DSM 4166

https://www.ncbi.nlm.nih.gov/nuccore/CP002622
Pseudomonas stutzeri DSM 4166, complete genome

https://www.ncbi.nlm.nih.gov/pubmed/21515765
J Bacteriol. 2011 Jul;193(13):3422-3. doi: 10.1128/JB.05039-11. Epub 2011 Apr 22.
Complete genome sequence of the nitrogen-fixing and rhizosphere-associated bacterium Pseudomonas stutzeri strain DSM4166.
Yu H1, Yuan M, Lu W, Yang J, Dai S, Li Q, Yang Z, Dong J, Sun L, Deng Z, Zhang W, Chen M, Ping S, Han Y, Zhan Y, Yan Y, Jin Q, Lin M.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3133286/

- CGMCC 1.1803

https://www.ncbi.nlm.nih.gov/nuccore/CP002881
Pseudomonas stutzeri strain CGMCC 1.1803, complete genome

https://www.ncbi.nlm.nih.gov/pubmed/21994926
J Bacteriol. 2011 Nov;193(21):6095. doi: 10.1128/JB.06061-11.
Complete genome sequence of the type strain Pseudomonas stutzeri CGMCC 1.1803.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3194927/
, and no plasmid was detected. 

-  A1501

https://www.ncbi.nlm.nih.gov/nuccore/CP000304
Pseudomonas stutzeri A1501, complete genome

https://www.ncbi.nlm.nih.gov/pubmed/18495935
Proc Natl Acad Sci U S A. 2008 May 27;105(21):7564-9. doi: 10.1073/pnas.0801093105. Epub 2008 May 21.
Nitrogen fixation island and rhizosphere competence traits in the genome of root-associated Pseudomonas stutzeri A1501.
Yan Y1, Yang J, Dou Y, Chen M, Ping S, Peng J, Lu W, Zhang W, Yao Z, Li H, Liu W, He S, Geng L, Zhang X, Yang F, Yu H, Zhan Y, Li D, Lin Z, Wang Y, Elmerich C, Lin M, Jin Q.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2396677/

##### P.stutzeri_3

- 28a24

https://www.ncbi.nlm.nih.gov/nuccore/CP007441
Pseudomonas stutzeri strain 28a24, complete genome

https://www.ncbi.nlm.nih.gov/pubmed/24903873
Genome Announc. 2014 Jun 5;2(3). pii: e00543-14. doi: 10.1128/genomeA.00543-14.
Complete Genome Sequence of the Highly Transformable Pseudomonas stutzeri Strain 28a24.
Smith BA1, Dougherty KM1, Baltrus DA2.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4047452/
USA

- 273

https://www.ncbi.nlm.nih.gov/nuccore/CP015641
Pseudomonas stutzeri strain 273, complete genome

https://www.ncbi.nlm.nih.gov/pubmed/29675016
Front Microbiol. 2018 Apr 5;9:682. doi: 10.3389/fmicb.2018.00682. eCollection 2018.
Genetic and Physiological Adaptations of Marine Bacterium Pseudomonas stutzeri 273 to Mercury Stress.
Zheng R1,2,3, Wu S4, Ma N1,2,3, Sun C1,2.

https://www.ncbi.nlm.nih.gov/pubmed/28698510
Mar Drugs. 2017 Jul 10;15(7). pii: E218. doi: 10.3390/md15070218.
Genome Sequence of Pseudomonas stutzeri 273 and Identification of the Exopolysaccharide EPS273 Biosynthesis Locus.
Wu S1, Zheng R2,3,4, Sha Z5, Sun C6,7.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5532660/
In our previous study, P. stutzeri 273 was isolated from the sediments of the East China Sea and exhibited great potential in combating biofilm-related infection and biofouling in marine aquaculture [13].

https://www.ncbi.nlm.nih.gov/pubmed/26903981
Front Microbiol. 2016 Feb 8;7:102. doi: 10.3389/fmicb.2016.00102. eCollection 2016.
Antibiofilm and Anti-Infection of a Marine Bacterial Exopolysaccharide Against Pseudomonas aeruginosa.
Wu S1, Liu G2, Jin W3, Xiu P2, Sun C3.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4744842/
The marine bacteria strains used in this study were isolated from the sediments of East China Sea


- DSM 10701

https://www.ncbi.nlm.nih.gov/nuccore/CP003725
Pseudomonas stutzeri DSM 10701, complete genome

https://www.ncbi.nlm.nih.gov/pubmed/22965097
J Bacteriol. 2012 Oct;194(19):5477-8. doi: 10.1128/JB.01257-12.
Genome sequence of Pseudomonas stutzeri strain JM300 (DSM 10701), a soil isolate and model organism for natural transformation.
Busquets A1, Peña A, Gomila M, Bosch R, Nogales B, García-Valdés E, Lalucat J, Bennasar A.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3457215/
Genes coding for products necessary for natural transformation ability were found (type IV pili, pilAI, pilC, pilT, and pilU, comA, and exbB). No nitrogen fixation genes (nif) or extrachromosomal elements were found. Predicted phage-related sequences, transposons, and insertion elements were detected.

#### P.stutzeri_frontier

- 
https://www.frontiersin.org/articles/10.3389/fmicb.2018.01458/full
Front. Microbiol., 02 July 2018 | https://doi.org/10.3389/fmicb.2018.01458
Frontiers | Comparative Genomics of Pseudomonas sp. Strain SI-3 Associated With Macroalga Ulva prolifera, the Causative Species for Green Tide in the Yellow Sea | Microbiology

Keywords: Pseudomonas, genome sequence, Comparative genomics, Ulva prolifera, algae-associated bacteria

- 
https://www.frontiersin.org/articles/10.3389/fmicb.2018.00682/full
Front. Microbiol., 05 April 2018 | https://doi.org/10.3389/fmicb.2018.00682
Frontiers | Genetic and Physiological Adaptations of Marine Bacterium Pseudomonas stutzeri 273 to Mercury Stress | Microbiology

Keywords: marine, Pseudomonas stutzeri, mercury, stress, motility, flagella

- 
https://www.frontiersin.org/articles/10.3389/fmicb.2015.00214/full
Front. Microbiol., 18 March 2015 | https://doi.org/10.3389/fmicb.2015.00214
Phylogenomics and systematics in Pseudomonas

Keywords: Pseudomonas, genomics, multilocus sequence analysis, taxonomy, systematics

#### P.stutzeri_denitrification

https://www.ncbi.nlm.nih.gov/pubmed/16483137
Sci China C Life Sci. 2005 Dec;48(6):585-92.
Structural and functional analysis of denitrification genes in Pseudomonas stutzeri A1501.
Yan Y1, Yang J, Chen L, Yang F, Dong J, Xue Y, Xu X, Zhu Y, Yao Z, Lin M, Wang Y, Jin Q.
Four gene clusters associated with denitrification were identified in the genome of A1501 strain, nar, nir, nor and nos, including 40 genes totally, which encode proteins for substance transportation, gene regulation and reductases. The three gene clusters, nir, nor and nos are adjacent on chromosome and are far from nar gene cluster. 

https://www.ncbi.nlm.nih.gov/pubmed/12618453
J Bacteriol. 2003 Mar;185(6):1895-902.
Operon structure and regulation of the nos gene region of Pseudomonas stutzeri, encoding an ABC-Type ATPase for maturation of nitrous oxide reductase.
Honisch U1, Zumft WG.

https://www.ncbi.nlm.nih.gov/pubmed/12576577
Microbiology. 2003 Jan;149(Pt 1):29-36.
Transcriptional regulation of the nos genes for nitrous oxide reductase in Pseudomonas aeruginosa.
Arai H1, Mizutani M, Igarashi Y.
The genes for nitrous oxide (N(2)O) reduction, nosRZDFYL, are clustered on the chromosome of Pseudomonas aeruginosa. 
http://mic.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.25936-0#tab2 
Nitrate (NO−3) is reduced through nitrite (NO−2), nitric oxide (NO) and nitrous oxide (N 2O) to dinitrogen (N 2) by denitrifying bacteria. N 2O reductase (N 2OR) catalyses the last step of the denitrification pathway, the two-electron reduction of N 2O to N 2. N 2OR has been purified and characterized from many bacteria, such as Pseudomonas stutzeri ( Coyle et al., 1985 ), 

The organization and regulation of the N 2OR genes have been investigated intensively in P. stutzeri. In this bacterium, the genes for N 2O reduction, nosRZDFYL, are clustered with the genes for other denitrification enzymes, NO−2 reductase (NIR) and NO reductase (NOR) ( R9 Braun & Zumft, 1992; Jüngst et al., 1991 ). The structural gene for N 2OR is nosZ ( Viebrock & Zumft, 1988 ), the nosDFY gene products are thought to be involved in the processing and insertion of copper into N 2OR ( Zumft et al., 1990 ), the nosL gene product is proposed to be an outer membrane disulfide isomerase ( Dreusch et al., 1996 ) and nosR encodes a putative membrane protein ( Cuypers et al., 1992 ). The nosR gene product is proposed to be an N 2O-sensing regulator that is required for the expression of the N 2OR genes ( Vollack & Zumft, 2001 ). The nos gene cluster of P. stutzeri is transcribed as at least three operons from promoters located upstream of nosR, nosZ and nosD ( Cuypers et al., 1995 ).

Genome sequencing of P. aeruginosa PAO1 has shown that the arrangement of the nosRZDFYL genes (PA3391–PA3396) is identical to that of P. stutzeri ( Stover et al., 2000 ); 

https://www.ncbi.nlm.nih.gov/pubmed/9841665
Microbiol Mol Biol Rev. 1998 Dec;62(4):1046-78.
Molecular genetics of the genus Paracoccus: metabolically versatile bacteria with bioenergetic flexibility.
Baker SC1, Ferguson SJ, Ludwig B, Page MD, Richter OM, van Spanning RJ.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC98939/
Organization of denitrification genes.
In P. denitrificans, nir genes (associated with nitrite reduction) and nor genes (associated with nitric oxide reduction) are clustered on a 17.7-kbp fragment, essentially transcribed divergently from one another (Fig. ​(Fig.4).4).
In Pseudomonas stutzeri, the other organism for which the genetics of denitrification have been extensively studied, the nos genes (nitrous oxide reductase associated) are also found in the same 30-kbp region of the genome as the nir and nor genes (34, 97, 347, 348, 350). The order of the genes within the nir, nor, and nos clusters varies among organisms, but the structural genes (nirS and norCB) are transcribed from the 5′ end of an operon (34, 59, 252). 

FIG. 4
Denitrification gene clusters of Pseudomonas stutzeri (A), Pseudomonas aeruginosa (B), and P. denitrificans Pd1222 (C). Reproduced with permission from reference 251a.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC98939/figure/F4/

#### P.stutzeri_hydrocarbon

http://www.bio.nite.go.jp/mifup/wiki/index.php?title=芳香族炭化水素分解

https://www.ncbi.nlm.nih.gov/pubmed/29217799
Genome Announc. 2017 Dec 7;5(49). pii: e01373-17. doi: 10.1128/genomeA.01373-17.
Draft Genome Sequence of Pseudomonas stutzeri Strain 19, an Isolate Capable of Efficient Degradation of Aromatic Hydrocarbons.
Brown LM1, Gunasekera TS1, Ruiz ON2.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5721144/
P. stutzeri 19 was shown, through gas chromatography-mass spectrometry (GC-MS) analysis, to efficiently metabolize toluene, xylenes, and 1,2,4-trimethyl benzene.

The NCBI PGAP predicted multiple genes involved in hydrocarbon degradation, including catechol 1,2-dioxygenase, 

https://www.sciencedirect.com/science/article/pii/S1878818117300038
Biocatalysis and Agricultural Biotechnology
Volume 10, April 2017, Pages 20-29
Optimization of conditions for polycyclic aromatic hydrocarbons (PAHs) degradation by Pseudomonas stutzeri P2 isolated from Chirimiri coal mines

PCR amplification of dioxygenase genes produced fragments of 350 bp, 867 bp and 350 bp in length for PAH-RHD, catechol 2, 3-dioxygenase and catechol 1, 2-dioxygenase, respectively suggesting the representative of Pseudomonas as the significant PAHs degrader.

https://www.tandfonline.com/doi/abs/10.1080/10406638.2015.1033434
Characterization of Phenol Metabolization by P. stutzeri N2
Published online: 07 Oct 2015

https://www.ncbi.nlm.nih.gov/pubmed/8161187
Appl Environ Microbiol. 1994 Mar;60(3):966-72.
Comparative biochemical and genetic analysis of naphthalene degradation among Pseudomonas stutzeri strains.
Rosselló-Mora RA1, Lalucat J, García-Valdés E.

----------
## Rhodococcus

- https://ja.wikipedia.org/wiki/ロドコッカス属
- https://ja.wikipedia.org/wiki/ロドコッカス・エクイ
- https://en.wikipedia.org/wiki/Rhodococcus
- https://microbewiki.kenyon.edu/index.php/Rhodococcus
Bacteria; Actinobacteria; Actinobacteria (class); Actinobacteridae; Actinomycetales; Corynebacterineae; Nocardiaceae; Rhodococcus

https://twitter.com/search?q=Rhodococcus

https://twitter.com/MattPerkins3/status/950176390749589505
In vitro studies have found Rhodococcus equi to usually be susceptible to erythromycin, rifampin, ciprofloxacin, vancomycin, aminoglycosides, imipenem & meropenem. Usually resistant to most other beta-lactams, variably susceptible to clinda, ceftriaxone, TMP-SMX & tetracycline.

https://www.ncbi.nlm.nih.gov/pubmed/28369330
Genome Biol Evol. 2017 May 1;9(5):1241-1247. 
Comparative Genomics of Rhodococcus equi Virulence Plasmids Indicates Host-Driven Evolution of the vap Pathogenicity Island.
MacArthur I1, Anastasi E1, Alvarez S1, Scortti M1,2, Vázquez-Boland JA1,2,3.

https://www.ncbi.nlm.nih.gov/pubmed/27638249
Genome Biol Evol. 2016 Oct 23;8(10):3140-3148.
Pangenome and Phylogenomic Analysis of the Pathogenic Actinobacterium Rhodococcus equi.
Anastasi E1, MacArthur I1, Scortti M1,2, Alvarez S1, Giguère S3, Vázquez-Boland JA4,2,5.

https://www.ncbi.nlm.nih.gov/pubmed/27924912
Sci Rep. 2016 Dec 7;6:38392. doi: 10.1038/srep38392.
Next-generation systematics: An innovative approach to resolve the structure of complex prokaryotic taxa.
Sangal V1, Goodfellow M2, Jones AL1, Schwalbe EC1, Blom J3, Hoskisson PA4, Sutcliffe IC1.

https://www.ncbi.nlm.nih.gov/pubmed/27074033
PLoS One. 2016 Apr 13;11(4):e0152887. doi: 10.1371/journal.pone.0152887. eCollection 2016.
Plasmid Profiles of Virulent Rhodococcus equi Strains Isolated from Infected Foals in Poland.

https://www.ncbi.nlm.nih.gov/pubmed/25237311
Front Plant Sci. 2014 Aug 19;5:406. doi: 10.3389/fpls.2014.00406. eCollection 2014.
Use of whole genome sequences to develop a molecular phylogenetic framework for Rhodococcus fascians and the Rhodococcus genus.



https://www.ncbi.nlm.nih.gov/pubmed/18179219
J Am Chem Soc. 2008 Jan 30;130(4):1126-7. doi: 10.1021/ja077821p. Epub 2008 Jan 8.
Rhodostreptomycins, antibiotics biosynthesized following horizontal gene transfer from Streptomyces padanus to Rhodococcus fascians.
Kurosawa K1, Ghiviriga I, Sambandan TG, Lessard PA, Barbara JE, Rha C, Sinskey AJ.


	Rhodococcus hoagii 103S	685727	PRJEA41335	41335	Terrabacteria group	Actinobacteria	5.04317	68.8	chromosome:NC_014659.1/FN563149.1	-	1	4649	4540	2010/11/09	2017/05/19	Complete Genome	IREC (International Rhodococcus equi Genome Consortium)	SAMEA2272695	GCA_000196695.1	REFR	ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000/196/695/GCA_000196695.1_ASM19669v1	20941392	103S

https://www.ncbi.nlm.nih.gov/pubmed/20941392
PLoS Genet. 2010 Sep 30;6(9):e1001145. doi: 10.1371/journal.pgen.1001145.
The genome of a pathogenic rhodococcus: cooptive virulence underpinned by key gene acquisitions.
Letek M1, González P, Macarthur I, Rodríguez H, Freeman TC, Valero-Rello A, Blanco M, Buckley T, Cherevach I, Fahey R, Hapeshi A, Holdstock J, Leadon D, Navas J, Ocampo A, Quail MA, Sanders M, Scortti MM, Prescott JF, Fogarty U, Meijer WG, Parkhill J, Bentley SD, Vázquez-Boland JA.

Figure 3. Schematic overview of relevant metabolic and virulence-related features of R. equi 103S. Complete glycolytic, PPP, and TCA cycle pathways, and all components for aerobic respiration, are present. The TCA cycle incorporates the glyoxylate shunt, which diverts two-carbon metabolites for biosynthesis. The methylcitrate pathway enzymes (pprCBD, REQ09040-60) are also present. The lutABC operon may take over the function of the D-lactate dehydrogenase (cytochrome) REQ00650, which is a pseudogene in 103S. REQ15040 (L-lactate 2-monoxygenase) and REQ27530 (pyruvate dehydrogenase [cytochrome]) can directly convert lactate and pyruvate into acetate. 

### Br-6

http://environmental-microbiology.org/2017/pdfs/要旨集/要旨集_口頭.pdf
既知細菌と比 較して臭素酸還元速度が100倍以上速いRhodococcus sp. Br-6株の分離に成功している。Br-6株は、微好気条件下においての み、臭素酸を臭化物に還元する。このとき、2,6-ジクロロインドフェノール(DCIP)と鉄イオンがそれぞれ酸化還元メディエー ターとして機能していることが明らかになっている。

http://environmental-microbiology.org/2017/pdfs/2014%20Program.pdf
O12-08 Rhodococcus sp. Br-6 株による複数の酸化還元メディエーターを介した臭素酸還元メカニズムの解明https://ci.nii.ac.jp/els/contentscinii_20180102151831.pdf?id=ART0010524150

https://ci.nii.ac.jp/naid/110009974433
CiNii Articles -  PG-003 Rhodococcus equi Br-6株による臭素酸還元メカニズムの解明(生理・代謝・増殖,ポスター発表)
日本微生物生態学会講演要旨集 (29), 125, 2013

https://www.ncbi.nlm.nih.gov/pubmed/27612520
Environ Sci Technol. 2016 Oct 4;50(19):10527-10534. Epub 2016 Sep 23.
Bromate Reduction by Rhodococcus sp. Br-6 in the Presence of Multiple Redox Mediators.
Tamai N1, Ishii T1, Sato Y1, Fujiya H1, Muramatsu Y2, Okabe N2, Amachi S1.

- Analysis of an approximately 1480-bp 16S rRNA gene sequence revealed that the strain is most closely related to Rhodococcus equi, with a sequence similarity of more than 99% (Figure S2). http://pubs.acs.org/doi/suppl/10.1021/acs.est.6b02261/suppl_file/es6b02261_si_001.pdf
- Activity Staining of Diaphorase and Bromate Reductase.
- Figure 6. Activity staining of diaphorase, ferric iron reductase, and bromate reductase
- , suggesting that strain Br-6 has multiple isoforms of diaphorase.
-  (Figure 6). These results strongly suggest that diaphorase is involved in bromate reduction by strain Br-6.

### DLD

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5465047/
3 Biotech. 2017 Jun; 7(2): 153.
Recombinant expression, characterization and application of a dihydrolipoamide dehydrogenase with diaphorase activity from Bacillus sphaericus

- Dihydrolipohyl dehydrogenase (DLD; EC: 1.8.1.4), also known as lipoamide dehydrogenase, is a FAD-dependent enzyme that catalyses the reversible oxidation of dihydrolipoamide to lipoamide (Vaubel et al. 2011). 
- Some of DLD enzymes are also termed diaphorase for their NAD(P)H dehydrogenase activities (Madiagan and Mayhew 1993). 
- we report a strain of B. sphaericus which produced a DLD with NADH-dependent diaphorase activity

誤植
Dihydrolipohyl
->
dihydrolipoyl

### Bromate Reduction

http://pubs.acs.org/doi/abs/10.1021/acs.est.7b00915?journalCode=esthag
Accelerated Reduction of Bromate in Frozen Solution - Environmental Science & Technology (ACS Publications)
Environ. Sci. Technol., 2017, 51 (15), pp 8368–8375

http://pubs.acs.org/doi/full/10.1021/acs.estlett.7b00488
Biological Bromate Reduction Driven by Methane in a Membrane Biofilm Reactor - Environmental Science & Technology Letters (ACS Publications)
Environ. Sci. Technol. Lett., 2017, 4 (12), pp 562–566

----------
## Shewanella

- https://en.wikipedia.org/wiki/Shewanella
- https://microbewiki.kenyon.edu/index.php/Shewanella_oneidensis
Bacteria; Proteobacteria; Gammaproteobacteria; Alteromonadales; Shewanellaceae; Shewanella; Shewanella oneidensis MR-1
- https://ja.wikipedia.org/wiki/シェワネラ属
、代謝の結果電子を放出し、電流を発生させることからバイオ燃料電池への利用が研究されている。

https://www.sciencedirect.com/topics/medicine-and-dentistry/shewanella
Shewanella - an overview | ScienceDirect Topics
Shewanella can be recovered from a variety of water sources, natural gas and petroleum reserves, dairy products, meat, and fish.

総説
https://www.ncbi.nlm.nih.gov/pubmed/31922549
FEMS Microbiol Rev. 2020 Jan 10. pii: fuz031. doi: 10.1093/femsre/fuz031. [Epub ahead of print]
The Shewanella genus: ubiquitous organisms sustaining and preserving aquatic ecosystems.
Lemaire ON1, Méjean V1, Iobbi-Nivol C1.

https://www.ncbi.nlm.nih.gov/pubmed/31482142
Commun Biol. 2019 Aug 29;2:323. doi: 10.1038/s42003-019-0567-3. eCollection 2019.
Cold adaptation in the environmental bacterium Shewanella oneidensis is controlled by a J-domain co-chaperone protein network.
Maillot NJ1, Honoré FA1, Byrne D2, Méjean V1, Genest O1.

https://www.ncbi.nlm.nih.gov/pubmed/31555221
Front Microbiol. 2019 Aug 28;10:1861. doi: 10.3389/fmicb.2019.01861. eCollection 2019.
Whole-Genome Sequencing Redefines Shewanella Taxonomy.
Thorell K1, Meier-Kolthoff JP2, Sjöling Å3, Martín-Rodríguez AJ3.

### Shewanella_reference_representative
参照ゲノム・代表ゲノム

```
$grep "Shewanella" prok_reference_genomes.txt | cut -f3
Shewanella oneidensis MR-1

$grep "Shewanella" prok_representative_genomes.txt | cut -f3
Shewanella algae
Shewanella amazonensis SB2B
Shewanella baltica OS678
Shewanella benthica KT99
Shewanella colwelliana ATCC 39565
Shewanella denitrificans OS217
Shewanella frigidimarina NCIMB 400
Shewanella japonica
Shewanella loihica PV-4
Shewanella mangrovi
Shewanella marina JCM 15074
Shewanella pealeana ATCC 700345
Shewanella piezotolerans WP3
Shewanella psychrophila
Shewanella sediminis HAW-EB3
Shewanella violacea DSS12
Shewanella waksmanii ATCC BAA-643
```

https://github.com/haruosuz/microbe/blob/master/references/microbe.2019.md#temperature
最適増殖温度の異なる細菌（好熱菌・常温菌・低温菌）
低温性細菌（Psychrophiles），中温性細菌（Mesophiles）お よび高温性細菌（Thermophiles）
https://github.com/haruosuz/microbe/blob/master/references/microbe.2019.md#psychrophile

#### Shewanella algae
中温性細菌

https://microbewiki.kenyon.edu/index.php/Shewanella_algae
S. algae is found in warm Marine environments throughout the world, and is isolated from seawater with 15-20ppt salinity [6].
Shewanella algae are mesophilic organisms found in aquatic environments. They have the capacity to also live in low temperature waters[5]. They are detected in seawater during the fall months, where temperatures are around 13°C. 

http://journal.kyorin.co.jp/journal/jscm/detail_j.php?-DB=jscm&-recid=980&-action=browse
Shewanella algaeによる敗血症・膿胸症例と分離株の性状について
要旨	　Shewanella属はVibrio科に属する好気性グラム陰性桿菌であり，海水，淡水，石油，ガスなどの自然界に広く分布し，ヒトに対する病原性は低いとされている。

#### Shewanella amazonensis SB2B
中温性細菌

https://en.wikipedia.org/wiki/Shewanella_amazonensis
isolated from shallow water sediments from the Amazon River.

https://microbewiki.kenyon.edu/index.php/Shewanella_amazonensis
- Shewanella amazonensis (strain ATC BAA-1098/SB2B) is a Gram-negative, facultatively anaerobic, motile, polarly flagellated, rod-shaped eubacterium. It was isolated from shelf coastal muds, in intertidal sediments in the Amazon River delta, off the Amapá coast of Brazil. 
- It is able to grow at mesophilic temperatures with optimal growth at 37°C and

https://www.ncbi.nlm.nih.gov/pubmed/9734053
Int J Syst Bacteriol. 1998 Jul;48 Pt 3:965-72.
Shewanella amazonensis sp. nov., a novel metal-reducing facultative anaerobe from Amazonian shelf muds.
Venkateswaran K1, Dollhopf ME, Aller R, Stackebrandt E, Nealson KH.
This organism, isolated from shallow-water marine sediments derived from the Amazon River delta, is 

#### Shewanella japonica
海水とムール貝Protothaca jedoensisから分離された寒天分解細菌

https://www.ncbi.nlm.nih.gov/pubmed/11411670
Int J Syst Evol Microbiol. 2001 May;51(Pt 3):1027-33.
Shewanella japonica sp. nov.
Ivanova EP1, Sawabe T, Gorshkova NM, Svetashev VI, Mikhailov VV, Nicolau DV, Christen R.
Two strains of agar-digesting bacteria, KMM 3299T and KMM 3300, respectively isolated from sea water and the mussel Protothaca jedoensis, have been characterized. 

#### Shewanella loihica PV-4
深海の鉄分が豊富な微生物マットから分離された
耐冷性細菌

https://www.ncbi.nlm.nih.gov/pubmed/16902030
Int J Syst Evol Microbiol. 2006 Aug;56(Pt 8):1911-6.
Shewanella loihica sp. nov., isolated from iron-rich microbial mats in the Pacific Ocean.
Gao H1, Obraztova A, Stewart N, Popa R, Fredrickson JK, Tiedje JM, Nealson KH, Zhou J.
A novel marine bacterial strain, PV-4(T), isolated from a microbial mat located at a hydrothermal vent of Loihi Seamount in the Pacific Ocean, 

https://genome.jgi.doe.gov/portal/she_p/she_p.home.html
Shewanella loihica , formerly Shewanella sp. strain PV-4, was isolated from iron-rich microbial mats at an active, deep sea, hydrothermal Naha Vent (1325 m below sea level), located on the South Rift of Loihi Seamount, Hawaii (http://www.soest.hawaii.edu/GG/HCV/loihivents.html). PV-4 is psychrotolerant, 

https://microbewiki.kenyon.edu/index.php/Shewanella_loihica
The PV-4T marine bacterial strain of Shewanella loihica was isolated at the deep-sea, hydrothermal Naha Vent from iron-rich microbial mats on the South Rift of Loihi Seamount, Hawaii, in the Pacific Ocean. 
Growth on Luria–Bertani agar plates showed S. loihica to be psychrotolerant at temperatures between 0-42 degrees C (Haichun Gao et al.). 


深海堆積物から分離された
#### Shewanella piezotolerans WP3
耐冷性細菌
#### Shewanella psychrophila WP2
好冷性細菌

https://www.ncbi.nlm.nih.gov/pubmed/17220442
Int J Syst Evol Microbiol. 2007 Jan;57(Pt 1):60-5.
Shewanella psychrophila sp. nov. and Shewanella piezotolerans sp. nov., isolated from west Pacific deep-sea sediment.
Xiao X1, Wang P, Zeng X, Bartlett DH, Wang F.
Strain WP2(T) was psychrophilic, growing optimally at about 10-15 degrees C, whereas strain WP3(T) was psychrotolerant, growing optimally at 15-20 degrees C. 


#### Shewanella violacea DSS12

https://www.jstage.jst.go.jp/article/kagakutoseibutsu1962/41/2/41_2_92/_article/-char/ja/
深海由来好圧好冷性微生物の全ゲノム解析の意義
仲宗 根薫, 森 浩禎, 馬場 知哉, 加藤 千明

https://en.wikipedia.org/wiki/Shewanella_violacea
Shewanella violacea DSS12 (S. violacea) is a gram-negative bacterium located in marine sediment in the Ryukyu Trench at a depth of 5,110m.[1]
Shewanella violacea is an obligate psychrophile (cryophile). Its optimum growth temperature is 8ᵒC.

### Shewanella_phylogeny
系統解析

https://www.ncbi.nlm.nih.gov/pubmed/30785388
Int J Syst Evol Microbiol. 2019 May;69(5):1259-1264. doi: 10.1099/ijsem.0.003298. Epub 2019 Feb 20.
Proposal of Parashewanella gen. nov. to accommodate Parashewanella curva sp. nov. and Parashewanella spongiae comb. nov. in the Shewanellaceae.
Xu S1,2,3, Yu K1,2,3, Su H1,2,3, Chen B1,2,3, Huang W1,2,3, Liang J1,2,3, Wang Y1,2,3, Wang G1,2,3.
- Phylogenetic analyses based on the 16S rRNA gene, the gyrB gene and the Up-to-date Bacterial Core Gene set (92 genes) indicated that strain C51T forms a stable cluster with Shewanella spongiae KCTC 22492T, and had 94 % 16S rRNA gene similarity to its closest type strain S. spongiae KCTC 22492T. 
https://ijs.microbiologyresearch.org/content/journal/ijsem/10.1099/ijsem.0.003298#tab2

https://www.ncbi.nlm.nih.gov/pubmed/30028285
Int J Syst Evol Microbiol. 2018 Sep;68(9):2960-2966. doi: 10.1099/ijsem.0.002929. Epub 2018 Jul 20.
Shewanella saliphila sp. nov., Shewanella ulleungensis sp. nov. and Shewanella litoralis sp. nov., isolated from coastal seawater.
Yun BR1,2, Park S3, Kim MK1, Park J1, Kim SB1.
https://www.microbiologyresearch.org/content/journal/ijsem/10.1099/ijsem.0.002929#tab2
Fig. 1.
Neighbour-joining phylogenetic tree based on 16S rRNA gene sequences showing the relationships among strains Shewanella saliphila MMS16-UL250T, Shewanella ulleungensis MMS16-UL253T and Shewanella litoralis MMS16-UL482T with related taxa. 
![](https://www.microbiologyresearch.org/docserver/fulltext/ijsem/68/9/ijsem002929-f1.gif)


https://www.ncbi.nlm.nih.gov/pubmed/27562367
Int J Syst Evol Microbiol. 2016 Dec;66(12):4947-4953. doi: 10.1099/ijsem.0.001450. Epub 2016 Aug 25.
Shewanella inventionis sp. nov., isolated from deep-sea sediment.
Wang MQ1, Sun L2.
On the basis of combined phenotypic, chemotaxonomic and phylogenetic analysis, strain KX27T represents a novel species of the genus Shewanella, for which the name Shewanella inventionis sp. nov. is proposed.
https://ijs.microbiologyresearch.org/content/journal/ijsem/10.1099/ijsem.0.001450#tab2
Taken together, phylogenetic analysis of 16S rRNA and gyrB gene sequences, DNA–DNA hybridization, phenotypic, and chemotaxonomic characteristics suggest that strain KX27 T represents a novel species of the genus Shewanella , for which the name Shewanella inventionis sp. nov. is proposed.
https://ijs.microbiologyresearch.org/content/ijsem/10.1099/ijsem.0.001450.F1
Fig. 1.
Neighbour-joining tree based on 16S rRNA gene sequences, showing the phylogenetic position of strain KX27T within the genus Shewanella . 
Pseudoalteromonas tetraodonis IAM 14160T was used as an outgroup.
https://ijs.microbiologyresearch.org/content/ijsem/10.1099/ijsem.0.001450.F2
Fig. 2.
Neighbour-joining tree based on gyrB gene sequences, showing the phylogenetic position of strain KX27T within the genus Shewanella . 
Pseudoalteromonas haloplanktis ATCC 14394T and Escherichia coli ATCC 25922T were used as the outgroup.

https://www.ncbi.nlm.nih.gov/pubmed/18398463
PLoS One. 2008 Apr 9;3(4):e1937. doi: 10.1371/journal.pone.0001937.
Environmental adaptation: genomic analysis of the piezotolerant and psychrotolerant deep-sea iron reducing bacterium Shewanella piezotolerans WP3.
Wang F1, Wang J, Jian H, Zhang B, Li S, Wang F, Zeng X, Gao L, Bartlett DH, Yu J, Hu S, Xiao X.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2276687/
Figure 2
Phylogenetic tree based on 16S rRNA (A) and whole proteome analysis (B).

https://www.ncbi.nlm.nih.gov/pubmed/15653901
Int J Syst Evol Microbiol. 2005 Jan;55(Pt 1):363-8.
Shewanella decolorationis sp. nov., a dye-decolorizing bacterium isolated from activated sludge of a waste-water treatment plant.
Xu M1, Guo J, Cen Y, Zhong X, Cao W, Sun G.
https://ijs.microbiologyresearch.org/content/ijsem/10.1099/ijs.0.63157-0.F1
Fig. 1.
Phylogenetic trees of the genus Shewanella based on 16S rRNA and gyrB gene sequences.
![](https://www.microbiologyresearch.org/docserver/fulltext/ijsem/55/1/IJE52769-1.gif)

### Shewanella_comparative_genomics
比較ゲノム

https://www.ncbi.nlm.nih.gov/pubmed/30026808
Biotechnol Biofuels. 2018 Jul 17;11:193. doi: 10.1186/s13068-018-1201-1. eCollection 2018.
Pan-genome analyses of 24 Shewanella strains re-emphasize the diversification of their functions yet evolutionary dynamics of metal-reducing pathway.
Zhong C1, Han M1, Yu S1, Yang P1, Li H1, Ning K1.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6048853/
Phylogeny of Shewanella
To gain insights into similarity and distance of the strains, two phylogenetic trees were constructed, one was based on the concatenated alignment of 1788 single-copy core genes (Fig. 1c),
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6048853/figure/Fig1/
c Maximum likelihood phylogenetic tree based on 1788 single-copy gene families using 100 bootstrap replications.

Evolution of the mtr clusters in Shewanella
 (Fig. 5a). The complete mtrBAC–omcA–mtrFED cluster was conserved in 15 out of the 24 strains. In addition, the mtrABC operon was conserved in all the genomes except for S. denitrificans OS217 and S. violacea DSS12. 
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6048853/figure/Fig5/
Global comparison of the mtr–omc clusters in Shewanella and orthologous gene clusters in other strains.

Methods
Gene family
To understand the evolutionary relationship of Shewanella, we conducted systematic comparative genomic studies. Full protein-coding genes of 24 Shewanella strains were used to construct gene families using OrthoMCL (v2) [44] with a BLAST E-value cut-off of 1e−5 and an inflation parameter of 1.5. Here, the clustering results yielded 7830 homologous clusters, 1788 of which were single-copy gene families, and then they were parsed and concatenated.


https://www.ncbi.nlm.nih.gov/pubmed/29504926
Int J Syst Evol Microbiol. 2018 Apr;68(4):1356-1360. doi: 10.1099/ijsem.0.002678. Epub 2018 Mar 5.
Whole-genome sequencing reveals that Shewanella haliotis Kim et al. 2007 can be considered a later heterotypic synonym of Shewanella algae Simidu et al. 1990.
Szeinbaum N1, Kellum CE2, Glass JB2, Janda JM3, DiChristina TJ1.
https://ijs.microbiologyresearch.org/content/journal/ijsem/10.1099/ijsem.0.002678#tab2
https://ijs.microbiologyresearch.org/content/ijsem/10.1099/ijsem.0.002678.F1
Fig. 1.
Molecular phylogenetic analysis of 16S rRNA genes of S. algae and closely related strains.

https://www.ncbi.nlm.nih.gov/pubmed/21569439
BMC Genomics. 2011 May 12;12:237. doi: 10.1186/1471-2164-12-237.
Genome-level homology and phylogeny of Shewanella (Gammaproteobacteria: lteromonadales: Shewanellaceae).
Dikow RB1.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3107185/
- Nineteen taxa from Shewanella (16 species and 3 additional strains of one species) as well as three outgroup species representing the genera Aeromonas (Gammaproteobacteria: Aeromonadales: Aeromonadaceae), Alteromonas (Gammaproteobacteria: Alteromonadales: Alteromonadaceae) and Colwellia (Gammaproteobacteria: Alteromonadales: Colwelliaceae) are included for a total of 22 taxa.
- The reason three outgroup taxa were included was because there had not been a previous study showing that Alteromonas and Colwellia were indeed basal to Shewanella. One could not assume that Shewanella was monophyletic a priori; the seven-gene analysis (Figure ​(Figure3B)3B) finds a non-monophyletic Shewanella. 
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3107185/figure/F2/
Figure 2
Genome tree. 
![](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3107185/bin/1471-2164-12-237-2.jpg)

### 

https://www.ncbi.nlm.nih.gov/pubmed/29994366
IEEE/ACM Trans Comput Biol Bioinform. 2018 Feb 9. doi: 10.1109/TCBB.2018.2804393. [Epub ahead of print]
A Comparative Study of Network Motifs in the Integrated Transcriptional Regulation and Protein Interaction Networks of Shewanella.
Ding D, Sun X.

https://www.ncbi.nlm.nih.gov/pubmed/28382331
mSystems. 2017 Mar 28;2(2). pii: e00165-16. doi: 10.1128/mSystems.00165-16. eCollection 2017 Mar-Apr.
A Genome-Scale Model of Shewanella piezotolerans Simulates Mechanisms of Metabolic Diversity and Energy Conservation.
Dufault-Thompson K1, Jian H2, Cheng R2, Li J2, Wang F2, Zhang Y3.

https://www.ncbi.nlm.nih.gov/pubmed/25233219
PLoS Comput Biol. 2014 Sep 18;10(9):e1003848. doi: 10.1371/journal.pcbi.1003848. eCollection 2014 Sep.
Genome-scale metabolic network validation of Shewanella oneidensis using transposon insertion frequency analysis.
Yang H1, Krumholz EW2, Brutinel ED3, Palani NP2, Sadowsky MJ4, Odlyzko AM5, Gralnick JA3, Libourel IG1.

https://www.ncbi.nlm.nih.gov/pubmed/24621294
BMC Syst Biol. 2014 Mar 12;8:31. doi: 10.1186/1752-0509-8-31.
Comparisons of Shewanella strains based on genome annotations, modeling, and experiments.
Ong WK, Vu TT, Lovendahl KN, Llull JM, Serres MH, Romine MF, Reed JL1.

https://www.ncbi.nlm.nih.gov/pubmed/21642407
Appl Environ Microbiol. 2011 Aug;77(15):5352-60. doi: 10.1128/AEM.00097-11. Epub 2011 Jun 3.
Large-scale comparative phenotypic and genomic analyses reveal ecological preferences of shewanella species and identify metabolic pathways conserved at the genus level.
Rodrigues JL1, Serres MH, Tiedje JM.

https://www.ncbi.nlm.nih.gov/pubmed/21810205
BMC Genomics. 2011 Jun 15;12 Suppl 1:S3. doi: 10.1186/1471-2164-12-S1-S3.
Comparative genomic reconstruction of transcriptional networks controlling central metabolism in the Shewanella genus.
Rodionov DA1, Novichkov PS, Stavrovskaya ED, Rodionova IA, Li X, Kazanov MD, Ravcheev DA, Gerasimova AV, Kazakov AE, Kovaleva GY, Permina EA, Laikova ON, Overbeek R, Romine MF, Fredrickson JK, Arkin AP, Dubchak I, Osterman AL, Gelfand MS.

### 

http://kazumaxneo.hatenablog.com/entry/2018/08/05/203339
32のバクテリアの１万以上の機能未知遺伝子欠損の影響をまとめた Fitness Browser - macでインフォマティクス

登録されているバクテリア:　30種以上のデータが利用可能（Organisms）。
```
Organism	Division	Experiments	# Carbon Sources	# Nitrogen Sources	# Stress Compounds	# Other Conditions	Updated Annotations
Shewanella oneidensis MR-1	Gammaproteobacteria	176	16	17	42	15	40
```

http://logos.ls.toyaku.ac.jp/~bioenergy1/service.html
電流生成の分子メカニズムの解明
我々は、ゲノム解読が完了し、遺伝子組換えが可能なモデル電流生成菌Shewanella　oneidensis MR-1株（シュワネラ、右の写真）を用いて、電極呼吸のメカニズムを解明するための研究を行っています（Newton et al. 2009. Appl. Environ. Microbiol. 75:7674; Kouzuma et al. 2010. Appl. Environ. Microbiol. 76:4151-4157）。 この菌は、細胞外に導電性ナノワイヤー（ナノスケールの電線）を作り、それを使って電極に電子を流すとも考えられています（Gorby et al. 2006. Proc. Nat. Acad. Sci. USA 103:11358）。

### Shewanella_space
宇宙

https://ntrs.nasa.gov/search.jsp?R=20190033397
NASA Technical Reports Server (NTRS) - Results of the Micro-12 Flight Experiment: Effects of Microgravity on Shewanella oneidensis MR-1
Publication Date:	November 20, 2019
https://ntrs.nasa.gov/archive/nasa/casi.ntrs.nasa.gov/20190033397.pdf
```
Top 20 differentially expressed protein – flight vs. ground
Top 20 differentially expressed protein – soluble iron vs. insoluble iron
```
• Conclusions
– Many differentially expressed proteins were identified via proteomic analysis – the biological significance of these is unclear


https://www.nasa.gov/spacebio/microbiology/our-experiments
Our Experiments | NASA
Investigating the Physiology and Fitness of an Exoelectrogenic Microorganism Under Microgravity Conditions (Micro-12)
https://www.nasa.gov/mission_pages/station/research/experiments/explorer/Investigation.html?#id=7470
The three specific aims of this project include:

Aug. 6, 2018
https://www.nasa.gov/mission_pages/station/research/news/shewanella_microgravity
Getting More Out of Microbes: Studying Shewanella in Microgravity | NASA

June 27, 2018
https://www.nasa.gov/feature/ames/could-electricity-producing-bacteria-help-power-future-space-missions
Could Electricity-Producing Bacteria Help Power Future Space Missions? | NASA

June 14, 2018
https://www.nasa.gov/ames/research/space-biosciences/micro-12-spacex-15
Micro-12 (SpaceX-15) | NASA



2018.07.03
https://www.discoverychannel.jp/0000027328/
ISSへの補給宇宙船、様々な最新実験機器を積み飛び立つ – Discovery Channel Japan | ディスカバリーチャンネル

今回運ばれる研究資材の中には「Micro-12」というものがある。これは、「エクソ電子発生微生物」（exoelectrogenic microbe）であるシュワネラ属の「Shewanella oneidensis MR-1」の成長や遺伝子発現に微重力がどのような影響を与えるかを調べるものだ。

このエクソ電子発生微生物というものは、有機物を食べて電気を生み出すことが知られている。そのためこれを微生物燃料電池として、廃棄された有機物質から電気を作ることなどが模索されているが、この生物に微重力が与える影響を調べることで将来の宇宙ミッションでも使えるかどうかを探ることができるのだ。

July 18, 2018
https://www.space.com/41193-electricity-producing-bacteria-international-space-station.html
How 'Electric Microbes' Could Generate Power (And More) for Space Missions (Video)

JUL 01, 2018
https://www.labroots.com/trending/microbiology/11945/-26
Using Bacteria to Help Power Space Missions | Microbiology

#### Shewanella oneidensis MR-1

https://www.ncbi.nlm.nih.gov/nuccore/NC_004347
Shewanella oneidensis MR-1 chromosome, complete genome

https://www.ncbi.nlm.nih.gov/nuccore/NC_004349
Shewanella oneidensis MR-1 plasmid megaplasmid, complete sequence

https://www.ncbi.nlm.nih.gov/pubmed/29422884
Front Microbiol. 2018 Jan 25;8:2568. doi: 10.3389/fmicb.2017.02568. eCollection 2017.
Redox Sensing within the Genus Shewanella.
Harris HW1, Sánchez-Andrea I2, McLean JS3,4, Salas EC5, Tran W1, El-Naggar MY1, Nealson KH1.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5789149/
- The model proposed that the swimming cells constantly sense their surroundings with specialized outer membrane cytochromes capable of extracellular electron transport (EET). 
- We found a strong correlation between the ability of strain MR-1 to accumulate on mineral surface and the presence of key EET genes such as mtrBC/omcA (SO_1778, SO_1776, and SO_1779) and gene coding for methyl-accepting protein (MCPs) with Ca+ channel chemotaxis receptor (Cache) domain (SO_2240).

https://www.ncbi.nlm.nih.gov/pubmed/22125499
PLoS Genet. 2011 Nov;7(11):e1002385. doi: 10.1371/journal.pgen.1002385. Epub 2011 Nov 17.
Evidence-based annotation of gene function in Shewanella oneidensis MR-1 using genome-wide fitness profiling across 121 conditions.
Deutschbauer A1, Price MN, Wetmore KM, Shao W, Baumohl JK, Xu Z, Nguyen M, Tamse R, Davis RW, Arkin AP.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3219624/

https://www.ncbi.nlm.nih.gov/pubmed/12368813
Nat Biotechnol. 2002 Nov;20(11):1118-23. Epub 2002 Oct 7.
Genome sequence of the dissimilatory metal ion-reducing bacterium Shewanella oneidensis.
https://www.nature.com/articles/nbt749
- multicomponent, branched electron transport
- Genome analysis revealed 39 c-type cytochromes, including 32 previously unidentified in S. oneidensis, and 

Aug 1, 2014
https://www.youtube.com/watch?v=cRN0iC9TSyE
B&B: Oxygen Allows Shewanella oneidensis MR-1 to Overcome Mediator Washout - YouTube
Video Highlight from Michaela TerAvest on her recently published B&B paper entitled "Oxygen allows Shewanella oneidensis MR-1 to overcome mediator washout in a continuously fed bioelectrochemical system."

### Shewanella_outer_membrane_cytochromes

https://www.ncbi.nlm.nih.gov/pubmed/27342561
Appl Environ Microbiol. 2016 Aug 15;82(17):5428-43. doi: 10.1128/AEM.01615-16. Print 2016 Sep 1.
Regulation of Gene Expression in Shewanella oneidensis MR-1 during Electron Acceptor Limitation and Bacterial Nanowire Formation.
Barchinger SE1, Pirbadian S2, Sambles C3, Baker CS1, Leung KM2, Burroughs NJ4, El-Naggar MY5, Golbeck JH6.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4988178/
outer membrane containing the cytochromes MtrC and OmcA
MtrA, MtrC, and OmcA, and 
the expression of the mtrA and mtrC homologs mtrF and mtrD
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4988178/table/T2/?report=objectonly
TABLE 2
Selected genes discussed in this work
Cytochromes

https://www.ncbi.nlm.nih.gov/pubmed/25886963/
BMC Microbiol. 2015 Mar 21;15:68. doi: 10.1186/s12866-015-0406-8.
Transcriptional mechanisms for differential expression of outer membrane cytochrome genes omcA and mtrC in Shewanella oneidensis MR-1.
Kasai T1, Kouzuma A2, Nojiri H3, Watanabe K4.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4417206/
the Mtr respiratory pathway, which consists of the outer membrane cytochromes OmcA and MtrC and associated proteins MtrA and MtrB. These proteins are encoded in the mtr gene cluster (omcA-mtrCAB) in the MR-1 chromosome.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4417206/figure/Fig1/
Figure 1
The organization and transcriptional units of the mtr genes. (A) Schematic illustration of the organization of the mtr genes. Solid arrows indicate the location and direction of the transcriptional promoters upstream of omcA (PomcA) and mtrC (PmtrC).

![](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4417206/bin/12866_2015_406_Fig1_HTML.jpg)

https://ja.wikipedia.org/wiki/遺伝子発現#ラクトースオペロン
真正細菌及び古細菌では、機能の関連した遺伝子が染色体上で隣接して存在し遺伝子クラスター（gene cluster）を形成している。この遺伝子クラスターのうち、単一のプロモーターで転写される単位をオペロン（operon）という。

----------
## Sinorhizobium
Sinorhizobium (renamed Ensifer)

- https://ja.wikipedia.org/wiki/シノリゾビウム属
- https://en.wikipedia.org/wiki/Sinorhizobium
- https://microbewiki.kenyon.edu/index.php/Sinorhizobium_meliloti
Bacteria; Proteobacteria; Alphaproteobacteria; Rhizobiales; Rhizobiazeae; Sinorhizobium (2).

https://twitter.com/search?q=Sinorhizobium

https://www.uniprot.org/taxonomy/382
Rhizobium meliloti (Ensifer meliloti) (Sinorhizobium meliloti)

https://en.wikipedia.org/wiki/Sinorhizobium_meliloti
Synonyms
Rhizobium meliloti Dangeard 1926
Ensifer meliloti (Dangeard 1926) Young 2003

https://www.ncbi.nlm.nih.gov/genome/browse/#!/prokaryotes/Sinorhizobium%20meliloti
Sinorhizobium melilotiには、Repliconが複数ある株（例えば、chromosomeが3つある株、plasmidが5つある株）がいます。

http://www.arsenic-sci-soc.jp/ArsenicLetterNo24rev%20HP.pdf
第24回ヒ素シンポジウム
「亜ヒ酸酸化細菌Sinorhizobium sp. KGO-5株のゲノム配列解析」
 配列決定の結果、KGO-5株のゲノム配列は1本の染色体と3本のプラスミドで構成されてお
り、総塩基数は6,564,102 bp、GC含有量は62.3%、CDS（タンパク質のコーディング領域）は
6,220個であることが予測されました。KGO-5株と近縁細菌31株の全てに保存されたオーソ
ログ遺伝子（コア遺伝子）に基づく最尤系統樹推定により、KGO-5株は Sinorhizobium
meliloti に最も近縁な系統群（姉妹群）であることが示唆されました。また重金属耐性遺伝
子データベースに対してBLASTプログラムを用いた配列類似性検索を行ったところ、プラス
ミド様配列上にヒ素耐性遺伝子のクラスターが検出されました。これらのヒ素耐性遺伝子群
はオペロン構造を形成して、ヒ素酸化システムとして機能していると考えられました。以上の
結果を踏まえ、KGO-5株に至る系統でヒ素関連遺伝子が獲得され、Sinorhizobium 属

### Sinorhizobium phylogeny
系統解析

https://www.ncbi.nlm.nih.gov/pubmed/28283520
Appl Environ Microbiol. 2017 May 1;83(10). pii: e03446-16. doi: 10.1128/AEM.03446-16. Print 2017 May 15.
Biogeography of a Novel Ensifer meliloti Clade Associated with the Australian Legume Trigonella suavissima.
Eardly B1, Elia P2, Brockwell J3, Golemboski D4, van Berkum P2.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5411501/
In our comparative studies, we included representatives of three other Ensifer species (E. medicae, E. arboris, and E. numidicus) that were previously shown to be closely related to E. meliloti (11,–13).

FIG 1
Chromosomal maximum likelihood tree based on concatenated nucleotide sequences of 10 loci in 84 strains.

https://www.ncbi.nlm.nih.gov/pubmed/26380038
Stand Genomic Sci. 2015 Jul 2;10:34. doi: 10.1186/s40793-015-0005-1. eCollection 2015.
High-quality permanent draft genome sequence of Ensifer meliloti strain 4H41, an effective salt- and drought-tolerant microsymbiont of Phaseolus vulgaris.
Mhamdi R1, Ardley J2, Tian R2, Seshadri R3, Reddy TB3, Pati A3, Woyke T3, Markowitz V4, Ivanova N3, Kyrpides N5, Reeve W2.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4571125/
In this context, Ensifer meliloti (formerly ‘Sinorhizobium meliloti ’) strain 4H41 was isolated from root nodules of common bean grown in sandy, slightly alkaline soil from the oasis of Rjim-Maatoug in South Tunisia [9].

https://www.ncbi.nlm.nih.gov/pubmed/24976888
Stand Genomic Sci. 2013 Dec 15;9(2):315-24. doi: 10.4056/sigs.4608286. eCollection 2013 Dec 20.
Genome sequence of Ensifer meliloti strain WSM1022; a highly effective microsymbiont of the model legume Medicago truncatula A17.
Terpolilli J1, Hill Y1, Tian R1, Howieson J1, Bräu L2, Goodwin L3, Han J4, Liolios K4, Huntemann M4, Pati A5, Woyke T4, Mavromatis K5, Markowitz V5, Ivanova N3, Kyrpides N3, Reeve W1.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4062635/
Two bacterial species, Ensifer meliloti and E. medicae are known to nodulate and fix N2 with Medicago spp. [3], 

https://www.ncbi.nlm.nih.gov/pubmed/22876202
PLoS Genet. 2012;8(8):e1002868. doi: 10.1371/journal.pgen.1002868. Epub 2012 Aug 2.
Population genomics of the facultatively mutualistic bacteria Sinorhizobium meliloti and S. medicae.
Epstein B1, Branca A, Mudge J, Bharti AK, Briskine R, Farmer AD, Sugawara M, Young ND, Sadowsky MJ, Tiffin P.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3410850/
Figure 1
Neighbor-joining trees showing relationships among 32 S. meliloti (blue squares) and 12 S. medicae (red circles).

https://www.ncbi.nlm.nih.gov/pubmed/19656922
Int J Syst Evol Microbiol. 2010 Mar;60(Pt 3):664-74. doi: 10.1099/ijs.0.012088-0. Epub 2009 Aug 5.
Multilocus sequence analysis of root nodule isolates from Lotus arabicus (Senegal), Lotus creticus, Argyrolobium uniflorum and Medicago sativa (Tunisia) and description of Ensifer numidicus sp. nov. and Ensifer garamanticus sp. nov.
Merabet C1, Martens M, Mahdhi M, Zakhia F, Sy A, Le Roux C, Domergue O, Coopman R, Bekki A, Mars M, Willems A, de Lajudie P.
http://ijs.microbiologyresearch.org/content/journal/ijsem/10.1099/ijs.0.012088-0#tab2

https://www.ncbi.nlm.nih.gov/pubmed/17329774
Int J Syst Evol Microbiol. 2007 Mar;57(Pt 3):489-503.
Multilocus sequence analysis of Ensifer and related taxa.
Martens M1, Delaere M, Coopman R, De Vos P, Gillis M, Willems A.
http://ijs.microbiologyresearch.org/content/journal/ijsem/10.1099/ijs.0.64344-0#tab2
In this study we used Sinorhizobium (now renamed Ensifer, see below) as a model system to assess the use of MLSA for taxonomic purposes. This genus contains eleven species, most of which have been described in polyphasic studies: Sinorhizobium fredii, Sinorhizobium xinjiangense ( Chen et al., 1988 ), Sinorhizobium meliloti, Sinorhizobium saheli, Sinorhizobium terangae ( de Lajudie et al., 1994 ), Sinorhizobium medicae ( Rome et al., 1996 ), Sinorhizobium arboris, Sinorhizobium kostiense ( Nick et al., 1999 ), Sinorhizobium kummerowiae ( Wei et al., 2002 ), ‘ Sinorhizobium morelense’ ( Wang et al., 2002 ) and Sinorhizobium americanum ( Toledo et al., 2003 ). From 16S rRNA gene comparisons, it is apparent that the monotypic genus Ensifer is highly related to Sinorhizobium ( Balkwill, 2005 ). This organism comprises predatory soil bacteria that were originally described in a separate genus as Ensifer adhaerens, based mainly on morphological and phenotypic characteristics ( Casida, 1982 ). Phylogenetic analyses of 16S rRNA and recA genes ( Balkwill, 2005 ; Willems et al., 2003 ) indicate that Ensifer forms a well-supported cluster together with Sinorhizobium. 
- Fig. 2.
Phylogenetic trees based on six genes calculated using the ML method. 
- Fig. 3.
Phylogenetic trees based on concatenated sequences of dnaK, glnA and thrC, calculated using NJ, MP and ML.
- Fig. 4.
Phylogenetic trees based on concatenated sequences of dnaK, glnA and thrC, gltA, recA and 16S rRNA genes calculated using NJ, MP and ML.

https://www.ncbi.nlm.nih.gov/pubmed/19120461
FEMS Microbiol Ecol. 2009 Jan;67(1):103-17. doi: 10.1111/j.1574-6941.2008.00590.x.
Rhizobia with different symbiotic efficiencies nodulate Acaciella angustissima in Mexico, including Sinorhizobium chiapanecum sp. nov. which has common symbiotic genes with Sinorhizobium mexicanum.
Rincón-Rosales R1, Lloret L, Ponce E, Martínez-Romero E.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2784085/

- ウイルス、ファージ
https://www.ncbi.nlm.nih.gov/pubmed/28842338
J Struct Biol. 2017 Aug 24. pii: S1047-8477(17)30136-3. doi: 10.1016/j.jsb.2017.08.005. [Epub ahead of print]
Structure, proteome and genome of Sinorhizobium meliloti phage ΦM5: A virus with LUZ24-like morphology and a highly mosaic genome.
Johnson MC1, Sena-Velez M2, Washburn BK3, Platt GN2, Lu S2, Brewer TE2, Lynn JS2, Stroupe ME4, Jones KM5.

- KEYWORDS: antimicrobial peptides, host-microbe interactions, symbiosis
https://www.ncbi.nlm.nih.gov/pubmed/28765224
MBio. 2017 Aug 1;8(4). pii: e01060-17. doi: 10.1128/mBio.01060-17.
Genome-Wide Sensitivity Analysis of the Microsymbiont Sinorhizobium meliloti to Symbiotically Important, Defensin-Like Host Peptides.
Arnold MFF1, Shabab M1, Penterman J1, Boehme KL2, Griffitts JS2, Walker GC3.

- プラスミド、亜ヒ酸酸化
https://www.ncbi.nlm.nih.gov/pubmed/27797963
FEMS Microbiol Ecol. 2017 Jan 1;93(1). doi: 10.1093/femsec/fiw215.
Molecular characterization of the pSinB plasmid of the arsenite oxidizing, metallotolerant Sinorhizobium sp. M14 - insight into the heavy metal resistome of sinorhizobial extrachromosomal replicons.
Romaniuk K1, Dziewit L1, Decewicz P1, Mielnicki S2, Radlinska M3, Drewniak L2.

- 亜ヒ酸酸化細菌 Sinorhizobium sp. Strain A2
https://www.ncbi.nlm.nih.gov/pubmed/26477790
Microbes Environ. 2015;30(4):330-4. doi: 10.1264/jsme2.ME15096. Epub 2015 Oct 17.
Cellular Response of Sinorhizobium sp. Strain A2 during Arsenite Oxidation.
Fukushima K1, Huang H, Hamamura N.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4676556/
The gene encoding TPT in Sinorhizobium spp. (e.g. S. meliloti and S. medicae) is located within the ars operon adjacent to arsC, which encodes arsenate reductase (Fig. 3). 

- プラスミド
https://www.ncbi.nlm.nih.gov/pubmed/26104371
Microbiol Spectr. 2014 Oct;2(5). doi: 10.1128/microbiolspec.PLAS-0005-2013.
The Plasmid Mobilome of the Model Plant-Symbiont Sinorhizobium meliloti: Coming up with New Questions and Answers.
Lagares A, Sanjuán J, Pistorio M.

While plasmids carrying symbiotic functions (pSyms) are known to have high structural stability (approaching that of chromosomes), the remaining plasmid mobilome (referred to as the non-pSym, functionally cryptic, or accessory compartment) has been shown to possess remarkable diversity and to be highly active in conjugation.

- arsenite oxidase (Aio)
https://www.ncbi.nlm.nih.gov/pubmed/22982475
Biochim Biophys Acta. 2013 Feb;1827(2):176-88. doi: 10.1016/j.bbabio.2012.08.007. Epub 2012 Sep 7.
Arsenics as bioenergetic substrates.
van Lis R1, Nitschke W, Duval S, Schoepp-Cothenet B.
Fig. 2. Organization of observed prokaryotic aio gene clusters. Organizations a–g are from demonstrated AsIII-oxidizing Bacteria. a: Sinorhizobium sp. str. M14, 

- 比較ゲノム
https://www.ncbi.nlm.nih.gov/pubmed/28386109
Sci Rep. 2017 Apr 6;7(1):675. doi: 10.1038/s41598-017-00730-7.
The underlying process of early ecological and genetic differentiation in a facultative mutualistic Sinorhizobium meliloti population.
Toro N1, Villadas PJ2, Molina-Sánchez MD2, Navarro-Gómez P3, Vinardell JM3, Cuesta-Berrio L4, Rodríguez-Carvajal MA4.

- 比較ゲノム
https://www.ncbi.nlm.nih.gov/pubmed/28138345
J Genomics. 2017 Jan 18;5:12-15. doi: 10.7150/jgen.17863. eCollection 2017.
Genomic characterization of eight Ensifer strains isolated from pristine caves and a whole genome phylogeny of Ensifer (Sinorhizobium).
Kumar HK1, Gan HM1, Tan MH1, Eng WW1, Barton HA2, Hudson AO3, Savka MA3.

- 比較ゲノム
https://www.ncbi.nlm.nih.gov/pubmed/23425606
Genome Biol. 2013 Feb 20;14(2):R17. doi: 10.1186/gb-2013-14-2-r17.
Comparative genomics of the core and accessory genomes of 48 Sinorhizobium strains comprising five genospecies.
Sugawara M, Epstein B, Badgley BD, Unno T, Xu L, Reese J, Gyaneshwar P, Denny R, Mudge J, Bharti AK, Farmer AD, May GD, Woodward JE, Médigue C, Vallenet D, Lajus A, Rouy Z, Martinez-Vaz B, Tiffin P, Young ND, Sadowsky MJ.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4053727/
Figure 1
Neighbor-joining tree based on concatenated sequences for 645 protein coding genes.

the outgroup (Rhizobium leguminosarum bv. trifolii WSM1325).
The tree was rooted by treating the R. leguminosarum strain as an outgroup

- プラスミド
Nucleic Acids Res. 2008 https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2602791/ "For example, Sinorhizobium meliloti megaplasmid pSymB (1 683 333 bp) is not self-transmissible, and has not been successfully cured from S. meliloti (51). It has been suggested that pSymB has acquired genes essential to the host's viability and that its genomic signature became similar to that of the main chromosome due to long-term residence in that host (20)."

https://www.ncbi.nlm.nih.gov/pubmed/16199569
J Bacteriol. 2005 Oct;187(20):6991-7.
Novel pathway for arsenic detoxification in the legume symbiont Sinorhizobium meliloti.
Yang HC1, Cheng J, Finan TM, Rosen BP, Bhattacharjee H.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1251620/
The two biologically relevant oxidation states of inorganic arsenic are arsenite [As(III)] and arsenate [As(V)], the former being more toxic than the later. 

- コドン使用
https://www.ncbi.nlm.nih.gov/pubmed/14597394
Gene. 2003 Nov 27;320:109-16.
The strength of translational selection for codon usage varies in the three replicons of Sinorhizobium meliloti.
Peixoto L1, Zavala A, Romero H, Musto H.

Sinorhizobium melilotiの染色体とプラスミドpSymBのコドン使用は、翻訳過程を効率化する自然選択（translational selection）の寄与が大きいのに対して、（根粒形成と窒素固定に関与する）プラスミドpSymAのコドン使用は、変異バイアス（mutational bias）の寄与が大きいと考えられる。

- 二連続塩基組成
https://www.ncbi.nlm.nih.gov/pubmed/12444420
Funct Integr Genomics. 2002 Nov;2(6):274-81. Epub 2002 Aug 1.
Dinucleotide compositional analysis of Sinorhizobium meliloti using the genome signature: distinguishing chromosomes and plasmids.
Wong K1, Finan TM, Golding GB.

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2602791/
プラスミドpSymBの二連続塩基組成（genomic signature）は、染色体のそれと類似しているので、宿主に長期間滞在したと考えられる。


考察

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4053727/figure/F1/
この研究では、Rhizobium leguminosarum を外群(outgroup)として用いて系統樹に根をつけている。
この645遺伝子配列の結合データ（concatenated sequences）を用いた系統樹では、
先ず、S. terangaeと他のグループが分岐し、
次に、S. frediiと他のグループが分岐し、
さらに、S. saheliと他のグループが分岐し、
最後に、Sinorhizobium medicaeクレードとSinorhizobium melilotiクレードとが分岐している。

一方、 https://www.ncbi.nlm.nih.gov/pubmed/17329774 で複数の系統樹（Fig. 2-4. Phylogenetic trees）の樹形を確認したところ、Sinorhizobium/Ensifer属の中でE. frediiとE. saheliの何れがより祖先的な生物から分岐して誕生したのか（外群になるのか）判断できない。例えば、
Fig. 2. "thrC ML"
Fig. 4. "6 genes MP"
と他の系統樹で、E. fredii、E. saheli、3株（E. meliloti, E. medicae, E. arboris）の系統関係が一致しない。

例えば、どの内群のグループよりも遠縁である E. adhaerens を外群として選択する。

----------
### KGO-5

http://www.arsenic-sci-soc.jp/18th%20symposium.pdf
第 18 回ヒ素シンポジウム 講演要旨集開催日: 2012年11月24日(土)・25日(日)国内土壌より分離された亜ヒ酸酸化細菌 KGO-5 株 ◯董 旦、天知誠吾(千葉大院・園芸・応用生命化学)土壌中のヒ素は主に無機態ヒ素である亜ヒ酸[As(III)]とヒ酸[As(V)]として存在する。
得られた菌株をKGO-5株とし、その亜ヒ酸酸化能を液体培地で確認した。
亜ヒ酸を唯一のエネルギー源として含む集積培養系より、KGO-5株を分離した。KGO-5株はグラム陰性の桿菌で、16S rRNA 遺伝子に基づく系統解析より、根粒菌 の1種であるSinorhizobium(シノリゾビウム)に近縁(98%~99%相同)であるこ とがわかった(図1)。また、SDB1株(Lugtu et al., 2009)やSinorhizobium sp. DAO10 株(Rhine et al., 2006)など既知の亜ヒ酸酸化細菌とも近縁であった。
https://www.uniprot.org/uniprot/A3EWK7
aroA - Arsenite oxidase - Sinorhizobium sp. DAO10 - aroA gene & protein
https://www.ncbi.nlm.nih.gov/protein/116090303
arsenite oxidase, partial [Sinorhizobium sp. DAO10]
https://www.ncbi.nlm.nih.gov/nuccore/?term=Sinorhizobium%20sp.%20DAO10
Sinorhizobium sp. DAO10 arsenite oxidase (aroA) gene, partial cds
Sinorhizobium sp. DAO10 16S ribosomal RNA gene, partial sequence


https://core.ac.uk/download/pdf/96948362.pdf
博士論文土壌環境における亜ヒ酸の挙動に及ぼす細菌群集の影響
2-6-1 供試菌株供試菌株として Alphaproteobacteria 綱に属する亜ヒ酸酸化細菌 Shinorhizobium sp. KGO-5 (Dong et al., 2014)を用いた。KGO-5 株は本研究室にお いて高度ヒ素汚染土壌から単離された通性独立栄養性の亜ヒ酸酸化細菌である。2-6-2 KGO-5 株の培養及び菌体懸濁液の調製



----------
## Tristan

https://pubmed.ncbi.nlm.nih.gov/19304960/
Genome Res
. 2009 Jul;19(7):1224-32. doi: 10.1101/gr.089250.108. Epub 2009 Mar 20.
Pervasive, genome-wide positive selection leading to functional divergence in the bacterial genus Campylobacter
Tristan Lefébure 1, Michael J Stanhope
https://genome.cshlp.org/content/19/7/1224.long
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC2704436/
To eliminate, or at least reduce, the influence of recombination in the positive selection scan, the alignments were tested for intragenic recombination using GARD (Pond et al. 2006). When a recombination breakpoint was found to be significant, the alignment was broken into two or more gene fragments, resulting in a total of 856 alignments of complete genes or gene fragments (192 of the original alignments were judged recombinant). 



----------
## Streptococcus

https://www.clemson.edu/science/departments/biosci/directory/profiles/vpricha

http://stanhope.vet.cornell.edu/publications.html

https://pubmed.ncbi.nlm.nih.gov/24625962/
Genome Biol Evol
. 2014 Apr;6(4):741-53. doi: 10.1093/gbe/evu048.
Phylogenomics and the dynamic genome evolution of the genus Streptococcus
Vincent P Richards 1, Sara R Palmer, Paulina D Pavinski Bitar, Xiang Qin, George M Weinstock, Sarah K Highlander, Christopher D Town, Robert A Burne, Michael J Stanhope
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4007547/

https://pubmed.ncbi.nlm.nih.gov/22666370/
PLoS One
. 2012;7(5):e37607. doi: 10.1371/journal.pone.0037607. Epub 2012 May 30.
Gene repertoire evolution of Streptococcus pyogenes inferred from phylogenomic analysis with Streptococcus canis and Streptococcus dysgalactiae
Tristan Lefébure 1, Vince P Richards, Ping Lang, Paulina Pavinski-Bitar, Michael J Stanhope
https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0037607
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC3364286/


https://pubmed.ncbi.nlm.nih.gov/22155358/
Infect Genet Evol
. 2012 Mar;12(2):274-7. doi: 10.1016/j.meegid.2011.11.004. Epub 2011 Dec 3.
Functional bias of positively selected genes in Streptococcus genomes
Haruo Suzuki 1, Michael J Stanhope
https://www.sciencedirect.com/science/article/abs/pii/S1567134811003893?via%3Dihub
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3294173/



2011
https://pubmed.ncbi.nlm.nih.gov/21282711/
Comparative Study Genome Biol Evol
. 2011;3:168-85. doi: 10.1093/gbe/evr006. Epub 2011 Jan 31.
Comparative genomic analysis of the Streptococcus dysgalactiae species group: gene content, molecular adaptation, and promoter evolution
Haruo Suzuki 1, Tristan Lefébure, Melissa Jane Hubisz, Paulina Pavinski Bitar, Ping Lang, Adam Siepel, Michael J Stanhope
https://academic.oup.com/gbe/article/doi/10.1093/gbe/evr006/576019
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3056289/


----------
## Alphaproteobacteria
https://ja.wikipedia.org/wiki/アルファプロテオバクテリア綱
Pelagibacter属、Pelagibacter ubiqueは他の属と遺伝子の数やGC含量が大きく異なり、系統図上の位置について議論がある 。Pelagibacterales属、Rickettsiales属およびHolosporales属のゲノムは特にATリッチである。
リボソームRNAのGC含量は伝統的な系統学的特徴の標識であり、ゲノムのGC含量への反映が小さいためである。例えば、Holosporales属の種のゲノムのGC含量はPelagibacterales属とRickettsiales属のそれと同様に小さいが、リボソームRNAのGC含量は2属と比べて非常に高い。したがって、Holosporales属は後者の2属よりもゲノム高GC含量の種とより関連が深い。

https://www.ncbi.nlm.nih.gov/pubmed/30789345
Elife. 2019 Feb 21;8. pii: e42535. doi: 10.7554/eLife.42535. [Epub ahead of print]
An updated phylogeny of the Alphaproteobacteria reveals that the parasitic Rickettsiales and Holosporales have independent origins.
Muñoz-Gómez SA1, Hess S1, Burger G2, Lang BF2, Susko E3, Slamovits CH1, Roger AJ1.
https://elifesciences.org/articles/42535
55 The great diversity of the Alphaproteobacteria itself presents a challenge to deciphering
56 the deepest divergences within the group. Such diversity encompasses a broad
57 spectrum of genome (nucleotide) and proteome (amino acid) compositions (e.g., the
58 A+T%-rich Pelagibacterales versus the G+C%-rich Acetobacteraceae) and molecular
59 evolutionary rates (e.g., the fast-evolving Pelagibacteriales, Rickettsiales or
60 Holosporales versus many slow-evolving species in the Rhodospirillales) (Ettema and
61 Andersson 2009). This diversity may lead to pervasive artefacts when inferring the
62 phylogeny of the Alphaproteobacteria, e.g., long-branch attraction (LBA) between the
63 Rickettsiales and Pelagibacterales, especially when including mitochondria (

長枝誘引（LBA: Long-Branch Attraction）

https://ja.wikipedia.org/wiki/最大節約法
より根本的に重大な問題として、形質の数が少なく、比較を行う生物種によって進化速度が大きく異なるような場合には誤りをもたらすことがある（Felsenstein、1978年）。これはロングブランチアトラクション (Long branch attraction) と呼ばれ、次のような場合に起きる。

----------
## mitochondria

https://www.ncbi.nlm.nih.gov/pubmed/25609566
Sci Rep. 2015 Jan 22;5:7949. doi: 10.1038/srep07949.
An integrated phylogenomic approach toward pinpointing the origin of mitochondria.
Wang Z1, Wu M1.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4302308/
3) Extreme sequence composition bias. Mitochondria and the obligate intracellular Alphaproteobacteria are in general extremely AT rich in their genome sequences. It is well established that sequence composition bias could adversely affect the phylogenetic reconstruction and lead to statistically robust but misleading conclusions9,10,11.

However, this hypothesis has been convincingly refuted by more recent studies demonstrating that this sister-clade relationship is a tree reconstruction artifact resulted from sequence composition bias21,23,24.

https://www.ncbi.nlm.nih.gov/pubmed/22291975
PLoS One. 2012;7(1):e30520. doi: 10.1371/journal.pone.0030520. Epub 2012 Jan 23.
The SAR11 group of alpha-proteobacteria is not related to the origin of mitochondria.
Rodríguez-Ezpeleta N1, Embley TM.
https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0030520
Figure 1. Nucleotide and amino acid composition of the dataset used in this study. A

https://www.ncbi.nlm.nih.gov/pubmed/21957463
PLoS One. 2011;6(9):e24857. doi: 10.1371/journal.pone.0024857. Epub 2011 Sep 21.
Phylogenomic analysis of Odyssella thessalonicensis fortifies the common origin of Rickettsiales, Pelagibacter ubique and Reclimonas americana mitochondrion.
Georgiades K1, Madoui MA, Le P, Robert C, Raoult D.

----------
## Rickettsiales
- https://ja.wikipedia.org/wiki/リケッチア目
  - リケッチア科 Rickettsiaceae
  - アナプラズマ科 Anaplasmataceae
  - ホロスポラ科 Holosporaceae

リケッチア目には、主として海洋から見出されるSAR11という難培養系統を含んでいる。ここには自由生活性で浮遊性の種が多数存在しており、例えば世界中の海洋に遍在する細菌（暫定的にCandidatus Pelagibacter ubiqueと命名された）が所属している[4]。この系統はリケッチア目の中でも最も祖先的な位置から派生しており、ミトコンドリアの起源は自由生活性のSAR11とその他の細胞内寄生性のリケッチアとの間に由来すると考えられる[5][6]。一方、ミトコンドリア膜間腔に共生するミディクロリアもリケッチア目に含まれている。

https://ja.wikipedia.org/wiki/ミディクロリア
(Candidatus Midichloria mitochondrii)はミトコンドリアに細胞内共生するリケッチアであり、マダニの卵巣に見出される。この名は映画スター・ウォーズ・シリーズに登場しフォースに寄与する共生微生物ミディ＝クロリアンに因んでおり、監督ジョージ・ルーカスの公認を得て命名されている[1]。

http://www.nikkei-science.com/?p=46394
ミトコンドリアの起源に新説〜日経サイエンス2015年4月号より
リケッチアが祖先である可能性が浮上

https://www.ncbi.nlm.nih.gov/pubmed/25609566
Sci Rep. 2015 Jan 22;5:7949. doi: 10.1038/srep07949.
An integrated phylogenomic approach toward pinpointing the origin of mitochondria.
Wang Z1, Wu M1.

https://www.ncbi.nlm.nih.gov/pubmed/25333787
PLoS One. 2014 Oct 15;9(10):e110685. doi: 10.1371/journal.pone.0110685. eCollection 2014.
Phylogenomic reconstruction indicates mitochondrial ancestor was an energy parasite.
Wang Z1, Wu M1.

https://www.ncbi.nlm.nih.gov/pubmed/16703114
PLoS Genet. 2006 May;2(5):e76. Epub 2006 May 12.
Genome sequence of Rickettsia bellii illuminates the role of amoebae in gene exchanges between intracellular pathogens.
Ogata H1, La Scola B, Audic S, Renesto P, Blanc G, Robert C, Fournier PE, Claverie JM, Raoult D.

----------
## Caedibacter

- https://ja.wikipedia.org/wiki/ケディバクター属
ケディバクター属（Caedibacter）[1] はリケッチア目で科の所属が明らかでない属で、グラム陰性非運動性偏性細胞内共生桿菌ないし球菌。

https://www.ncbi.nlm.nih.gov/pubmed/29348333
Genome Announc. 2018 Jan 18;6(3). pii: e01418-17. doi: 10.1128/genomeA.01418-17.
Draft Genome Sequence and Annotation of the Obligate Bacterial Endosymbiont Caedibacter taeniospiralis, Causative Agent of the Killer Phenotype in Paramecium tetraurelia.
Zaburannyi N1,2, Grosser K3, Gasparoni G4, Müller R1,2, Schrallhammer M5, Simon M6.

https://www.ncbi.nlm.nih.gov/pubmed/26543129
Genome Announc. 2015 Nov 5;3(6). pii: e01310-15. doi: 10.1128/genomeA.01310-15.
Draft Genome Sequence of Caedibacter varicaedens, a Kappa Killer Endosymbiont Bacterium of the Ciliate Paramecium biaurelia.
Suzuki H1, Dapper AL2, Jackson CE3, Lee H4, Pejaver V4, Doak TG2, Lynch M2, Preer JR Jr2.

https://www.ncbi.nlm.nih.gov/pubmed/25609566
Sci Rep. 2015 Jan 22;5:7949. doi: 10.1038/srep07949.
An integrated phylogenomic approach toward pinpointing the origin of mitochondria.
Wang Z1, Wu M1.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4302308/
Candidatus Caedibacter acanthamoebae

----------
## Holospora
- https://en.wikipedia.org/wiki/Holosporaceae
- https://ja.wikipedia.org/wiki/ホロスポラ属


https://www.ncbi.nlm.nih.gov/pubmed/30627761
Microb Ecol. 2019 May;77(4):1092-1106. doi: 10.1007/s00248-018-1311-0. Epub 2019 Jan 9.
"Candidatus Hafkinia simulans" gen. nov., sp. nov., a Novel Holospora-Like Bacterium from the Macronucleus of the Rare Brackish Water Ciliate Frontonia salmastra (Oligohymenophorea, Ciliophora): Multidisciplinary Characterization of the New Endosymbiont and Its Host.
Fokin SI1,2,3, Serra V4, Ferrantini F5, Modeo L5, Petroni G5.

https://www.ncbi.nlm.nih.gov/pubmed/30124811
FEMS Microbiol Ecol. 2018 Nov 1;94(11). doi: 10.1093/femsec/fiy162.
Identification of novel Legionella genes required for endosymbiosis in Paramecium based on comparative genome analysis with Holospora spp.
Watanabe K1,2, Suzuki H3, Nishida T1, Mishima M2, Tachibana M4, Fujishima M5,6, Shimizu T1,2, Watarai M1,2.

https://www.ncbi.nlm.nih.gov/pubmed/29713316
Front Microbiol. 2018 Apr 16;9:738. doi: 10.3389/fmicb.2018.00738. eCollection 2018.
Comparative Genomic Analysis of Holospora spp., Intranuclear Symbionts of Paramecia.
Garushyants SK1,2, Beliavskaia AY3,4, Malko DB5, Logacheva MD1,6, Rautian MS4, Gelfand MS1,2,5,7.
the newly sequenced genome of Holospora curviuscula, a macronuclear symbiont of Paramecium bursaria, 



https://www.ncbi.nlm.nih.gov/pubmed/30208886
Respir Res. 2018 Sep 12;19(1):174. doi: 10.1186/s12931-018-0878-9.
Smoking is associated with quantifiable differences in the human lung DNA virome and metabolome.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6136173/
Some rare viral populations were unique to smoker or nonsmoker total viral communities (Additional file 5: Figure S4). For example, Actinomyces, Capnocytophaga, Haemophilus and Rhodoferax phages were found only in smokers, and Enhydrobacter, Enterobacter, Holospora, Morganella, and Spiroplasma phages were found only in nonsmokers. Eukaryotic DNA viruses were only found in the lungs of smokers (Additional file 5: Figure S4).


https://www.msz.co.jp/book/detail/08767.html
これからの微生物学
マイクロバイオータからCRISPRへ
p.109:
「ホロスポラ属菌」
「また、節足動物、海洋無脊椎動物、あるいは哺乳類にも見られる。」

https://www.ncbi.nlm.nih.gov/pubmed/25680230
Trends Cell Biol. 2015 Jun;25(6):339-46. doi: 10.1016/j.tcb.2015.01.002. Epub 2015 Feb 10.
Intranuclear bacteria: inside the cellular control center of eukaryotes.
Schulz F1, Horn M2.
https://www.sciencedirect.com/science/article/pii/S0962892415000136?via%3Dihub
- Phylogenetically diverse intranuclear bacteria have been discovered in various protist, arthropod, marine invertebrate, and mammalian hosts. 
- Intranuclear bacteria in protozoa
The majority of known intranuclear bacteria are associated with protists (Figure 1). One of the most thoroughly studied and best-understood intranuclear bacteria are Holospora spp., which infect paramecia and are affiliated with rickettsiae (Alphaproteobacteria). 
- Nevertheless, an important first step towards deciphering the molecular basis of the intranuclear life style was the recent sequencing of the genomes of three Holospora spp. [78].

https://www.ncbi.nlm.nih.gov/pubmed/25115770
FEMS Microbiol Lett. 2014 Oct;359(1):16-8.
Draft genome sequences of three Holospora species (Holospora obtusa, Holospora undulata, and Holospora elegans), endonuclear symbiotic bacteria of the ciliate Paramecium caudatum.
Dohra H, Tanaka K, Suzuki T, Fujishima M, Suzuki H.

http://bio.w3.kanazawa-u.ac.jp/bio/seminar_20170929.pdf
タイトル：ゾウリムシを用いた細胞内共生成立のしくみの研究
講演者：藤島政博 山口大学 大学院創成科学研究科
日時：平成２９年９月２９日（金） 13時30分～14時30分

プロテアバクテリアαサブグループのホロスポラ(Holospora)属細菌は、ゾウ
リムシ属を宿主とする核内共生細菌で、真核細胞と原核細胞の細胞内共生（一
次共生）が成立する仕組みの解明に適した材料です。この細菌は、宿主の食胞
を経由して大核か小核の特定核の核膜を識別して核内に侵入し、核内で増殖し、
宿主の特定遺伝子の発現を変化させて温度耐性や塩濃度耐性に宿主を形質転換
させます。これによって、宿主は生息可能範囲を広げています。ホロスポラは
縮小したゲノムサイズのために（1.4 Mbp 程度）、宿主外では増殖できません。
しかし、宿主には細胞分裂回数に依存した寿命があるため、ホロスポラは宿主
のタンパク質合成活性の低下を引き金にして宿主外に脱出し、若い宿主に感染
する機会を得ています。講演の前半ではホロスポラの生存戦略と宿主に及ぼす
影響を説明します。

http://www.lc.shizuoka.ac.jp/pdf/baio30doura.pdf
道羅 英夫
グリーン科学技術研究所 准教授 
ゾウリムシ共生系における分子基盤の解明及び
新規共生系の構築とその利用 
図. ホロスポラ・オブツサの感染過程

----------
## Legionella

https://www.ncbi.nlm.nih.gov/pubmed/22047552
BMC Genomics. 2011 Nov 2;12:542. doi: 10.1186/1471-2164-12-542.
Insight into cross-talk between intra-amoebal pathogens.
Gimenez G1, Bertelli C, Moliner C, Robert C, Raoult D, Fournier PE, Greub G.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3220658/
- The phylogenetic analysis identified 9 cases of potential gene transfer between P. acanthamoebae or L. drancourtii and Rickettsia, Orientia, Wolbachia and Ehrlichia (Additional file 12, Figure S4). 
- In our study, several phylogenies indicated possible HGTs between amoeba-associated bacteria (Legionella and Parachlamydia) and arthropod-associated bacteria, such as Rickettsia, Orientia, Wolbachia, and Ehrlichia. 
- Figure 3
The ABC transporter operon. 
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3220658/figure/F3/
All of the proteins encoded for in the operon exhibit a similar phylogenetic tree, which shows a clustering of P. acanthamoebae and L. drancourtii, thereby supporting the hypothesis of a single transfer event for all four genes.

----------
## Wolbachia

https://ja.wikipedia.org/wiki/ボルバキア

https://kotobank.jp/word/ボルバキア-669591
昆虫類、クモ・ダニ類、フィラリア線虫などの細胞内に寄生する共生細菌。宿主の卵細胞の細胞質を通じて母性遺伝する。
[補説]ボルバキアは宿主の生殖を操作することが知られている。感染したメスが単為生殖を行う、感染したオスと感染していないメスが交配すると受精卵が発育しない、感染したオスの個体がメス化する、感染したオスだけが死亡する、などの生殖異常を引き起こして感染したメスを増加させる。

- https://sites.google.com/site/kageyama000jp/
陰山大輔
- https://twitter.com/Symbionticism
Seth Bordenstein
- https://twitter.com/chicaScientific
Irene Newton
- https://twitter.com/WolbachiaProj
WolbachiaProject

### Wolbachia HGT

https://www.maruzen-publishing.co.jp/item/?book_no=303272
突然変異主導進化論
https://www.maruzen-publishing.co.jp/fixed/files/pdf/303272/index_pdf_303272.pdf
遺伝子水平伝播 219
ボルバキア属の細菌から昆虫への遺伝子の移行
https://www.ncbi.nlm.nih.gov/pubmed/17761848
Science. 2007 Sep 21;317(5845):1753-6. Epub 2007 Aug 30.
Widespread lateral gene transfer from intracellular bacteria to multicellular eukaryotes.
Dunning Hotopp JC1, 

http://www.kashiwashobo.co.jp/book/b286870.html
世界は細菌にあふれ、人は細菌によって生かされる
p. 291 注13
遺伝子水平伝播
循環論法
https://www.ncbi.nlm.nih.gov/pubmed/21334091
Trends Genet. 2011 Apr;27(4):157-63. doi: 10.1016/j.tig.2011.01.005. Epub 2011 Feb 18.
Horizontal gene transfer between bacteria and animals.
Dunning Hotopp JC1.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3068243/
- Box 4Problems identifying bacteria-to-animal HGT in existing genome data
A circular argument ensues where eukaryotic genome sequencing projects remove all bacterial sequences because animals do not have HGT from bacteria, and examination of those same genomes for HGT reinforces the notion that HGT from bacteria to animals does not occur. Resources like the Sequence Read Archive and Trace Archive offer opportunities for assembly-independent analyses, but only when the raw experimental data is deposited before “contaminant” removal.
- Hopefully, the promises of third generation single molecule sequencing will yield a more diverse sampling of vertebrate genomes, facilitate the differentiation of contaminants from HGT, and improve our understanding of the extent of HGT events in a wide array of diverse organisms.


https://en.wikipedia.org/wiki/Wolbachia#Horizontal_gene_transfer
Horizontal gene transfer
Wolbachia species also harbor a bacteriophage called bacteriophage WO or phage WO.[43] Comparative sequence analyses of bacteriophage WO offer some of the most compelling examples of large-scale horizontal gene transfer between Wolbachia coinfections in the same host.[44] It is the first bacteriophage implicated in frequent lateral transfer between the genomes of bacterial endosymbionts. Gene transfer by bacteriophages could drive significant evolutionary change in the genomes of intracellular bacteria that were previously considered highly stable or prone to loss of genes over time.[44]

https://en.wikipedia.org/wiki/WO_virus

ファージ遺伝子wmkは、オス殺しの候補
https://twitter.com/Symbionticism/status/1171814144858841089
Seth Bordenstein on Twitter: "Just out yesterday. Phage WO gene wmk recapitulates aspects of Wolbachia male killing and is a hop, skip, &amp; jump from the cif genes. @JIPerlmutter @srbordenstein @lepage_d @JMetcalfUM @RobUnckless et al. cover it here https://t.co/Z07lcdHSXs @VI4Research @EvolutionVU @VuBiome"
12:53 AM · Sep 12, 2019
https://www.ncbi.nlm.nih.gov/pubmed/31504075
PLoS Pathog. 2019 Sep 10;15(9):e1007936. doi: 10.1371/journal.ppat.1007936. eCollection 2019 Sep.
The phage gene wmk is a candidate for male killing by a bacterial endosymbiont.
Perlmutter JI1,2, Bordenstein SR1,2, Unckless RL3, LePage DP1, Metcalf JA1,4, Hill T3, Martinez J5, Jiggins FM5, Bordenstein SR1,2,6,7.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6736233/
, we identify a candidate gene in the eukaryotic association module of Wolbachia prophage WO, termed WO-mediated killing (wmk), 

https://www.ncbi.nlm.nih.gov/pubmed/29686091
Proc Natl Acad Sci U S A. 2018 May 8;115(19):4987-4991. doi: 10.1073/pnas.1800650115. Epub 2018 Apr 23.
One prophage WO gene rescues cytoplasmic incompatibility in Drosophila melanogaster.
Shropshire JD1, On J2, Layton EM2, Zhou H2, Bordenstein SR1,3,4,5.

https://www.ncbi.nlm.nih.gov/pubmed/27965627
Front Microbiol. 2016 Nov 29;7:1867. eCollection 2016.
Bacteriophage WO Can Mediate Horizontal Gene Transfer in Endosymbiotic Wolbachia Genomes.
Wang GH1, Sun BF2, Xiong TL1, Wang YK3, Murfin KE4, Xiao JH5, Huang DW6.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5126046/
- , we utilized bioinformatic, phylogenetic, and molecular analyses based on all sequenced Wolbachia and phage WO genomes. Our results show that the phage WO can mediate HGT between Wolbachia strains, regardless of whether the transferred genes originate from Wolbachia or other unrelated bacteria.
- Figure 6
A conserved bacterial region associated with phage WO, Wolbachia strains, and Rickettsia plasmid. 

https://www.ncbi.nlm.nih.gov/pubmed/27727237
Nat Commun. 2016 Oct 11;7:13155. doi: 10.1038/ncomms13155.
Eukaryotic association module in phage WO genomes from Wolbachia.
Bordenstein SR1, Bordenstein SR1,2.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5062602/
- However, the presence of bacteriophages in obligate intracellular bacteria of eukaryotes may promote DNA transfers between eukaryotes and bacteriophages.
- To the best of our knowledge, these eukaryotic-like domains have never before been reported in packaged bacteriophages and their phylogeny, distribution and sequence diversity imply lateral transfers between bacteriophage/prophage and animal genomes.

https://www.ncbi.nlm.nih.gov/pubmed/19154594
BMC Genomics. 2009 Jan 20;10:33. doi: 10.1186/1471-2164-10-33.
Horizontal gene transfer between Wolbachia and the mosquito Aedes aegypti.
Klasson L1, Kambris Z, Cook PE, Walker T, Sinkins SP.


https://www.ncbi.nlm.nih.gov/pubmed/30727958
BMC Genomics. 2019 Feb 6;20(1):106. doi: 10.1186/s12864-019-5492-9.
Genome expansion of an obligate parthenogenesis-associated Wolbachia poses an exception to the symbiont reduction model.
Kampfraath AA1, Klasson L2, Anvar SY3,4, Vossen RHAM4, Roelofs D5, Kraaijeveld K5, Ellers J5.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6364476/
Within wFol, we found five phage WO regions representing 25.4% of the complete genome, few pseudogenized genes, and an expansion of DNA-repair genes in comparison to other Wolbachia. These signs of genome conservation were mirrored in the wFol host, the springtail F. candida, which also had an expanded DNA-repair gene family and many horizontally transferred genes. 

https://twitter.com/Daphnia_t_ponyo/status/1075403162817687552
ミクロ・コスモス on Twitter: "【共生細菌が産生するオス殺しの毒素】 ボルバキアなど一部の節足動物の共生細菌は感染宿主がオスだった場合"オス殺し"を行うことが知られていますがその分子基盤は半世紀もの間なぞでした しかし共生細菌がコードするオス殺しの遺伝子がついに発見されたようです https://t.co/Igom7oMyyI"
9:51 AM - 19 Dec 2018

https://www.ncbi.nlm.nih.gov/pubmed/30202647
PeerJ. 2018 Sep 3;6:e5486. doi: 10.7717/peerj.5486. eCollection 2018.
A bioinformatics approach to identifying Wolbachia infections in arthropods.
Pascar J1,2, Chandler CH1.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6126470/
- Here, we look for evidence of Wolbachia infection across a wide array of arthropod species using a bioinformatic approach to detect the Wolbachia genes ftsZ, wsp, and the groE operon in next-generation sequencing samples available through the NCBI Sequence Read Archive. 
- Meta-analysis indicates that the infection distribution in the total number of species may be closer to 66% (Hilgenboecker et al., 2008).
- A metagenomics-based approach can also be useful for characterizing microbiomes, including looking for Wolbachia and other symbionts (e.g. Dittmer & Bouchon, 2018), 


02 May 2018
https://www.nature.com/articles/s41586-018-0086-2
Male-killing toxin in a bacterial symbiont of Drosophila | Nature

https://www.ncbi.nlm.nih.gov/pubmed/29214346
Mol Genet Genomics. 2018 Apr;293(2):525-540. doi: 10.1007/s00438-017-1402-5. Epub 2017 Dec 6.
Comparative genomics reveals the presence of putative toxin-antitoxin system in Wolbachia genomes.
Singhal K1, Mohanty S2.

動画
Nov 29, 2017
https://www.youtube.com/watch?v=S2tND3OmwzY
Mosquitoes Might SAVE Lives, Thanks To Bacteria | I Contain Multitudes - YouTube
http://www.kashiwashobo.co.jp/book/b286870.html
世界は細菌にあふれ、人は細菌によって生かされる

2017年3月9日
https://www.natureasia.com/ja-jp/nature/highlights/83856
微生物学： ボルバキア菌による昆虫の生存操作

ボルバキア由来のプロファージWOがコードする2つの遺伝子が、細胞質不和合を再現する

https://www.ncbi.nlm.nih.gov/pubmed/28005061
Nat Microbiol. 2016 Dec 22;2:16241. doi: 10.1038/nmicrobiol.2016.241.
Comparative genomics provides a timeframe for Wolbachia evolution and exposes a recent biotin synthesis operon transfer.
Gerth M1, Bleidorn C2,3.

2016年10月12日 
https://www.natureasia.com/ja-jp/ncomms/pr-highlights/11040
【進化】細菌に感染するウイルスがクロゴケグモと同じDNAを持っていた
ファージWO（ボルバキア菌に感染するウイルス）のゲノム配列を解読し、そのゲノムの一部を構成する遺伝子が真核生物の遺伝子に類似していることを発見した。
https://www.nature.com/articles/ncomms13155
Eukaryotic association module in phage WO genomes from Wolbachia

https://www.ncbi.nlm.nih.gov/pubmed/27194801
G3 (Bethesda). 2016 Jul 7;6(7):2113-23. doi: 10.1534/g3.116.028449.
Comparative Genomics of a Parthenogenesis-Inducing Wolbachia Symbiont.
Lindsey AR1, Werren JH2, Richards S3, Stouthamer R4.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4938664/
parthenogenesis-inducing (PI) Wolbachia 
PI-Wolbachia 
Figure 1
Phylogenetic relationships of Wolbachia. (A) Phylogeny inferred with RAxML from a nucleotide supermatrix of the five Wolbachia MLST (multi locus sequence typing) genes using 1000 bootstrap replicates. Supergroups are shown in colored boxes, and labeled in the top right corner of each box. Symbols next to taxa denote Wolbachia host and phenotypes. Colors at nodes indicate bootstrap values. Anaplasma marginale str. Florida “Ama” is the outgroup. 

https://www.ncbi.nlm.nih.gov/pubmed/27189996
Genome Biol Evol. 2016 Jun 3;8(5):1526-42. doi: 10.1093/gbe/evw096.
Comparative Genomics of Two Closely Related Wolbachia with Different Reproductive Effects on Hosts.
Newton IL1, Clark ME2, Kent BN3, Bordenstein SR4, Qu J5, Richards S5, Kelkar YD6, Werren JH2.

2015年9月
http://www.nikkei-science.com/201509_046.html
デング熱ストッパー | 日経サイエンス　
自然界によく見られる細菌の助けを借りて，蚊によるデング熱の媒介を阻止する試みが始まっている。

https://www.a.u-tokyo.ac.jp/topics/2015/20150715-1.html
共生細菌ヴォルバキアの「オス殺し」は宿主昆虫のオス化因子の発現を抑制することで達成される
勝間准教授のグループは、2014年にカイコにおいて雄化と遺伝子量補償を担う分子としてMasc遺伝子を同定しています（http://www.a.u-tokyo.ac.jp/topics/2014/20140515-1.html）。RNA干渉法によってカイコのオス胚子でMasc mRNA量を低下させると、性分化のマーカーであるdoublesex (dsx)のスプライシングがメス型になり、遺伝子量補償が破綻することで胚致死してしまいます。このカイコにおける実験ではメスは胚致死しないことから、人為的に「オス殺し」現象を再現できたと言えます。そこで、ヴォルバキアによる「オス殺し」のターゲットがMascまたはその下流の因子であると仮定し、本研究を開始しました。

2015/06/25
https://www.kagakudojin.co.jp/book/b200756.html
消えるオス （DOJIN選書： 66）
昆虫の性をあやつる微生物の戦略
著者	陰山 大輔 著
https://bookmeter.com/books/9768053
タスマニアデビルの伝染性腫瘍は人間の介入でなんとかならないものかと思う。

https://www.ted.com/talks/elizabeth_murchison/transcript?language=ja
エリザベス・マーチソン：感染性のガンと闘う | TED Talk

2014 陰山大輔
https://www.jstage.jst.go.jp/article/konchubiotec/83/3/83_3_243/_pdf
昆虫の生殖を操作する細胞内共生細菌 Wolbachia の機能と特徴

http://www.aist.go.jp/aist_j/press_release/pr2014/pr20140701/pr20140701.html
産総研：トコジラミの生存を支える共生細菌ボルバキアのビタミンB7合成能力

https://www.ncbi.nlm.nih.gov/pubmed/23593012
PLoS Genet. 2013 Apr;9(4):e1003381. doi: 10.1371/journal.pgen.1003381. Epub 2013 Apr 4.
Comparative genomics of Wolbachia and the bacterial species concept.
Ellegaard KM1, Klasson L, Näslund K, Bourtzis K, Andersson SG.

http://www.nikkei-science.com/page/magazine/1006/201006_062.html
日経サイエンス　 2010年6月号
対談
細菌によって変わる虫たち
深津武馬（産業技術総合研究所） 茂木健一郎（ソニーコンピュータサイエンス研究所）

2005年9月1日
http://www.natureasia.com/ja-jp/reviews/highlight/10933
不和合性の解析：ボルバキア(Wolbachia)を対象にする

2002年7月
http://www.nikkei-science.com/page/sci_book/bessatu/51221.html
細菌が操る性転換 | 日経サイエンス　
昆虫などに広く寄生するボルバキア属の細菌は，寄生した相手を雄から雌に性転換させたり雄になる卵を殺したりする。また寄生相手の交配を妨害することさえある。


----------
## people

----------

### Ogura

https://github.com/haruosuz/mgsa/blob/master/references/mgsa.tools.md#escherichia-coli

https://pubmed.ncbi.nlm.nih.gov/19815525/
Comparative Study Proc Natl Acad Sci U S A
. 2009 Oct 20;106(42):17939-44. doi: 10.1073/pnas.0903585106. Epub 2009 Oct 6.
Comparative genomics reveal the mechanism of the parallel evolution of O157 and non-O157 enterohemorrhagic Escherichia coli
Yoshitoshi Ogura 1, Tadasuke Ooka, Atsushi Iguchi, Hidehiro Toh, Md Asadulghani, Kenshiro Oshima, Toshio Kodama, Hiroyuki Abe, Keisuke Nakayama, Ken Kurokawa, Toru Tobe, Masahira Hattori, Tetsuya Hayashi
https://pmc.ncbi.nlm.nih.gov/articles/PMC2764950/
Fig. 3.
Genome-wide phylogenetic analysis and whole gene repertoire comparison of the EHECs and other fully sequenced E. coli/Shigella strains. (A) The neighbor-joining tree constructed by using the concatenated nucleotide sequences of 345 orthologous CDS groups from the 25 sequenced strains. 
(B) The hierarchical clustering tree that was constructed based on a gene repertoire comparison of the 25 strains. The entire gene repertoire of the 25 strains is represented by 12,940 CDS groups that were defined based on the results of an all-to-all BLASTP analysis of CDSs from the 25 strains.


----------
### Gilmore
Michael S. Gilmore

https://pubmed.ncbi.nlm.nih.gov/22617140/
Comparative Study mBio
. 2012 May 22;3(3):e00112-12. doi: 10.1128/mBio.00112-12. Print 2012.
Comparative genomics of vancomycin-resistant Staphylococcus aureus strains and their positions within the clade most commonly associated with Methicillin-resistant S. aureus hospital-acquired infection in the United States
Veronica N Kos 1, Christopher A Desjardins, Allison Griggs, Gustavo Cerqueira, Andries Van Tonder, Matthew T G Holden, Paul Godfrey, Kelli L Palmer, Kip Bodi, Emmanuel F Mongodin, Jennifer Wortman, Michael Feldgarden, Trevor Lawley, Steven R Gill, Brian J Haas, Bruce Birren, Michael S Gilmore
https://journals.asm.org/doi/10.1128/mBio.00112-12
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3372964/
Diversity in plasmid content and Tn1546 location.
FIG 3
Plasmid sequences in VRSA.

https://pubmed.ncbi.nlm.nih.gov/22354958/
Comparative Study mBio
. 2012 Mar 1;3(1):e00318-11. doi: 10.1128/mBio.00318-11. Print 2012.
Comparative genomics of enterococci: variation in Enterococcus faecalis, clade structure in E. faecium, and defining characteristics of E. gallinarum and E. casseliflavus
Kelli L Palmer 1, Paul Godfrey, Allison Griggs, Veronica N Kos, Jeremy Zucker, Christopher Desjardins, Gustavo Cerqueira, Dirk Gevers, Suzanne Walker, Jennifer Wortman, Michael Feldgarden, Brian Haas, Bruce Birren, Michael S Gilmore
https://journals.asm.org/doi/10.1128/mBio.00318-11
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3374389/


----------
### Amachi Seigo

天知 誠吾

Genome sequencing of bacteria contributing to the geochemical cycling of arsenic, bromine, and iodine.
https://trace.ddbj.nig.ac.jp/DRASearch/study?acc=DRP004782
https://trace.ddbj.nig.ac.jp/BPSearch/bioproject?acc=PRJDB5044
https://ddbj.nig.ac.jp/BPSearch/bioproject?acc=PRJDB5044
https://www.ncbi.nlm.nih.gov/bioproject/PRJDB5044

### Shewanella sp. M-Br

日本微生物生態学会 第32回大会
http://meeting-jsme2018.com/program_pdf/Program_Poster%20Session%201.pdf
2018 JSME annual meeting & 10th ASME
＊ P1-043
Bromate reduction by Shewanella sp.M-Br in the presence of chelated iron
鉄を介して臭素酸を還元する？～Shewanella の新たな可能性を探る～
〇Hiroko Fujiya、Seigo Amachi
Horticul, Univ. of the Chiba

### Roseovarius sp. A-2
https://www.ncbi.nlm.nih.gov/assembly/GCA_002072155.1
Genome coverage: 248.96x
https://www.ncbi.nlm.nih.gov/nuccore/BDIY00000000
WGS         BDIY01000001-BDIY01000126
https://www.ncbi.nlm.nih.gov/Traces/wgs/BDIY01

https://www.ncbi.nlm.nih.gov/pubmed/28529651
J Genomics. 2017 Apr 6;5:51-53. doi: 10.7150/jgen.19846. eCollection 2017.
Draft Genome Sequence of Roseovarius sp. A-2, an Iodide-Oxidizing Bacterium Isolated from Natural Gas Brine Water, Chiba, Japan.
Yuliana T, Nakajima N, Yamamura S, Tomita M, Suzuki H, Amachi S.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5436463/
The genome contained four multicopper oxidase genes. Among these, one was closely related to ioxA, which was previously found to be involved in iodide oxidation in Iodidimonas sp. Q-1 4. 

### 
https://www.ncbi.nlm.nih.gov/pubmed/27738047
Genome Announc. 2016 Oct 13;4(5). pii: e01155-16. doi: 10.1128/genomeA.01155-16.
Draft Genome Sequence of Arenibacter sp. Strain C-21, an Iodine-Accumulating Bacterium Isolated from Surface Marine Sediment.
Ito K, Nakajima N, Yamamura S, Tomita M, Suzuki H, Amachi S.

http://www.ncbi.nlm.nih.gov/pubmed/25977440
Genome Announc. 2015 May 14;3(3). pii: e00472-15. doi: 10.1128/genomeA.00472-15.
Draft Genome Sequence of Anaeromyxobacter sp. Strain PSR-1, an Arsenate-Respiring Bacterium Isolated from Arsenic-Contaminated Soil.
Tonomura M, Ehara A, Suzuki H, Amachi S.

http://www.ncbi.nlm.nih.gov/pubmed/25635012
Genome Announc. 2015 Jan 29;3(1). pii: e01478-14. doi: 10.1128/genomeA.01478-14.
Draft Genome Sequence of Geobacter sp. Strain OR-1, an Arsenate-Respiring Bacterium Isolated from Japanese Paddy Soil.
Ehara A, Suzuki H, Amachi S.


https://www.ncbi.nlm.nih.gov/pubmed/27612520
Environ Sci Technol. 2016 Oct 4;50(19):10527-10534. Epub 2016 Sep 23.
Bromate Reduction by Rhodococcus sp. Br-6 in the Presence of Multiple Redox Mediators.
Tamai N1, Ishii T1, Sato Y1, Fujiya H1, Muramatsu Y2, Okabe N2, Amachi S1.

https://www.ncbi.nlm.nih.gov/pubmed/26041311
Biosci Biotechnol Biochem. 2015;79(11):1898-905. doi: 10.1080/09168451.2015.1052767. Epub 2015 Jun 4.
A putative multicopper oxidase, IoxA, is involved in iodide oxidation by Roseovarius sp. strain A-2.
Shiroyama K1, Kawasaki Y1, Unno Y2, Amachi S1.

https://www.ncbi.nlm.nih.gov/pubmed/25051896
Biosci Biotechnol Biochem. 2014;78(11):1963-70. doi: 10.1080/09168451.2014.940276. Epub 2014 Jul 23.
Arsenite oxidation by a facultative chemolithoautotrophic Sinorhizobium sp. KGO-5 isolated from arsenic-contaminated soil.
Dong D1, Ohtsuka T, Dong DT, Amachi S.


http://www.ncbi.nlm.nih.gov/pubmed/24994802
Genome Announc. 2014 Jul 3;2(4). pii: e00659-14. doi: 10.1128/genomeA.00659-14.
Draft genome sequence of strain Q-1, an iodide-oxidizing alphaproteobacterium isolated from natural gas brine water.
Ehara A, Suzuki H, Kanesaki Y, Yoshikawa H, Amachi S.

https://www.ncbi.nlm.nih.gov/pubmed/22447601
Appl Environ Microbiol. 2012 Jun;78(11):3941-9. doi: 10.1128/AEM.00084-12. Epub 2012 Mar 23.
Iodide oxidation by a novel multicopper oxidase from the alphaproteobacterium strain Q-1.
Suzuki M1, Eda Y, Ohsawa S, Kanesaki Y, Yoshikawa H, Tanaka K, Muramatsu Y, Yoshikawa J, Sato I, Fujii T, Amachi S.


https://kaken.nii.ac.jp/en/report/KAKENHI-PROJECT-23580103/235801032012hokoku/
KAKEN — Research Projects | 2012 Fiscal Year Research-status Report (KAKENHI-PROJECT-23580103)
２．	ヒ酸を吸着させたフェリハイドライトからのヒ素の溶出試験：フェリハイドライトにヒ酸を吸着させたものを調製した。これを唯一の電子授与体として含む培地に、前年度分離に成功したヒ酸還元細菌Geobacter sp. OR-1株やAnaeromyxobacter sp. PSR-1株などを接種した。OR-1株とPSR-1株では培養に伴い2価鉄と亜ヒ酸が溶出した。これに対し、ヒ酸還元能を持たない鉄還元細菌Geobacter metallireducens GS-15株やShewanella oneidensis MR-1株では2価鉄の溶出はあったものの、亜ヒ酸の溶出は認められなかった。以上のことから、鉄鉱物に吸着したヒ素の溶出には鉄還元能とヒ酸還元能の両方を持つことが重要と考えられた。


Mini Reviews
https://www.ncbi.nlm.nih.gov/pubmed/21558718
Microbes Environ. 2008;23(4):269-76.
Microbial contribution to global iodine cycling: volatilization, accumulation, reduction, oxidation, and sorption of iodine.
Amachi S1.

https://www.ncbi.nlm.nih.gov/pubmed/17933915
Appl Environ Microbiol. 2007 Dec;73(23):7536-41. Epub 2007 Oct 12.
Hydrogen peroxide-dependent uptake of iodine by marine Flavobacteriaceae bacterium strain C-21.
Amachi S1, Kimura K, Muramatsu Y, Shinoyama H, Fujii T.

https://www.ncbi.nlm.nih.gov/pubmed/17644635
Appl Environ Microbiol. 2007 Sep;73(18):5725-30. Epub 2007 Jul 20.
Dissimilatory iodate reduction by marine Pseudomonas sp. strain SCT.
Amachi S1, Kawaguchi N, Muramatsu Y, Tsuchiya S, Watanabe Y, Shinoyama H, Fujii T.

2016/06/03
http://ipst.adm.ehime-u.ac.jp/pages/?p=5106
新規マルチ銅オキシダーゼIOXを活用した酵素殺菌剤の殺菌メカニズムの解明
https://www.amano-enzyme.co.jp/corporate/foundation/pdf/17/pg05.pdf
Q-1 株

2014年5月26日
https://kaken.nii.ac.jp/file/KAKENHI-PROJECT-23580103/23580103seika.pdf
ヒ酸還元能を持つジオバクター属細菌による水田土壌からのヒ素溶出
OR-1株のゲノムにはヒ素代謝遺伝子群を コードする領域(Arsenic island)が独立して2つ存在し、異化的ヒ酸還元反応を触媒すると考えられるarrAB遺伝子 も見出された。

2012.12.13
http://tenbou.nies.go.jp/news/jnews/detail.php?i=9992
千葉大学、微生物酵素による放射性ヨウ素の土壌中での安定化プロセスを解明

http://www.jsmrs.jp/journal/No32_2/No32_2_115.pdf
Microb. Resour. Syst., 2016
総説
ヨウ素酸化細菌 ─その分離から応用まで─

https://www.jstage.jst.go.jp/article/kagakutoseibutsu/51/5/51_286/_article/-char/ja/
化学と生物 Vol. 51, No. 5, 2013
微生物による無機ヨウ素化合物の酸化還元反応
https://www.jstage.jst.go.jp/article/kagakutoseibutsu/51/5/51_286/_pdf
図5■ Pseudomonas sp. SCT株のヨウ素酸呼吸時に働く推定呼吸鎖コンポーネント

https://ci.nii.ac.jp/naid/110009767287/
地球化学 Chikyukagaku 47(4), 209-219, 2013
ヨウ素の地球化学と微生物 : ヨウ素の揮発,濃縮,酸化,還元,吸着,脱ハロゲン化反応を触媒するバクテリア
http://www.geochem.jp/journal_j/contents/pdf/47-4-209.pdf
- これらの結果から,海水中のヨウ素酸は海洋生物 の有する硝酸還元酵素によってヨウ化物イオンに還元 されるという考え方が広く受け入れられるようになっ た。しかしながら,
- 筆者らは,硝酸還元酵素ではなく,ヨウ素酸に特異 的な還元酵素を有するバクテリアも存在するのではな いかと考え,海洋環境からの分離を試みた。スクリー ニングの結果,相模湾の堆積物から Pseudomonas sp. SCT 株を分離した(Amachi et al., 2007a)。

天知誠吾 著 - ‎2010
https://www.jstage.jst.go.jp/article/geochemproc/57/0/57_0_61/_pdf
ヨウ素の水圏循環に与える微生物の影響
SCT 株

----------

## excerpt
抜粋［引用］

https://haruosuz.hatenadiary.org/entry/20090605
論文 - Haruo Suzuki / Bioinformatics
【結果】S. simiaeのドラフトゲノム配列をRoche/454で決定し、ブドウ球菌属の他種を含む近縁細菌のゲノム配列を比較した。全ゲノム情報に基づく系統解析により、S. simiaeがS. aureusの姉妹群であること、最初にStaphylococcus pseudintermediusとその他の集団が分岐し、次にStaphylococcus carnosusとその他の集団が分岐したこと、ブドウ球菌属のヒトへの適応はS. carnosusと分岐した後に起きたことが示唆された。

2012-01-24
https://pubmed.ncbi.nlm.nih.gov/22272658/
BMC Genomics
. 2012 Jan 24;13:38. doi: 10.1186/1471-2164-13-38.
Comparative genomic analysis of the genus Staphylococcus including Staphylococcus aureus and its newly described sister species Staphylococcus simiae
Haruo Suzuki 1, Tristan Lefébure, Paulina Pavinski Bitar, Michael J Stanhope
https://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-13-38
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3317825/

Results and discussion
Genomic features

Table 1
Genomic features of Macrococcus caseolyticus and 28 Staphylococcus strains.

https://www.ncbi.nlm.nih.gov/pubmed/30910818
G3 (Bethesda). 2019 May 7;9(5):1321-1329. doi: 10.1534/g3.118.200978.
Genomic Analysis of Pseudomonas sp. Strain SCT, an Iodate-Reducing Bacterium Isolated from Marine Sediment, Reveals a Possible Use for Bioremediation.
Harada M, Ito K, Nakajima N, Yamamura S, Tomita M, Suzuki H, Amachi S.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6505155/
Table 1
Genomic features of Pseudomonas strains analyzed
- GENOME FEATURES
- Bacteria, such as Pseudomonas putida KT2440 and P. stutzeri TS44, possess genes whose products allow resistance to and metabolism of arsenic compounds (Andres and Bertin 2016). We report that the SCT genome contains genes putatively involved in such processes; these include several genes encoding ArsR family transcriptional regulators; spxA (locus_tag: PSCT_01329) and arsC (locus_tag: PSCT_01343 and PSCT_04450); genes encoding “arsenate reductase [EC:1.20.4.1]” (KEGG: K00537 and K03741), arsA (locus_tag: PSCT_02478), which encodes “arsenite/tail-anchored protein-transporting ATPase [EC:3.6.3.16 3.6.3.-]” (KEGG: K01551); and ubiG (locus_tag: PSCT_04351), which encodes “arsenite methyltransferase [EC:2.1.1.137]” (KEGG: K07755). There is a cluster of six genes azr-ywlE-aseR-arsC1-czcO-CDS (contig accession number: BDJA01000003; locus_tag range: PSCT_01732 to PSCT_01737); of these, azr encodes “arsenical resistance protein ArsH” (KEGG: K11811), ywlE encodes “arsenate reductase [EC:1.20.4.1]” (KEGG: K03741), aseR and arsC1 encode “ArsR family transcriptional regulator, arsenate/arsenite/antimonite-responsive transcriptional repressor” (KEGG: K03892), czcO encodes “putative oxidoreductase CzcO”, and the CDS encodes “arsenite transporter” (KEGG: K03325). These results suggest that strain SCT can potentially resist and metabolize arsenic compounds.
- DATA AVAILABILITY
Supplemental material available at Figshare: https://doi.org/10.25387/g3.7829321.

Data for Pseudomonas sp. strain SCT genes. The columns are as follows: contig accession number, locus_tag, length in amino acids (Laa), G+C content at the third codon position (GC3), binary number (1 or 0) indicating whether the gene is phage-like or not (Prophage), binary number (1 or 0) indicating whether the gene is strain-specific or not (Strain Specific), functional annotations by Prokka (gene and product names), annotations from the UniRef90 database, and annotations from the KEGG database (KEGG_Entry, KEGG_Definition, and KEGG_Pathway).

https://www.ncbi.nlm.nih.gov/pubmed/30510597
J Genomics. 2018 Nov 15;6:122-126. doi: 10.7150/jgen.27741. eCollection 2018.
Genomic Analysis of Rhodococcus sp. Br-6, a Bromate Reducing Bacterium Isolated From Soil in Chiba, Japan.
Ito K, Harada M, Nakajima N, Yamamura S, Tomita M, Suzuki H, Amachi S.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6275400/
We searched for genes involved in various biological functions of strain Br-6. The Br-6 genome contained at least 13 genes coding for c-type cytochromes and at least 19 genes putatively involved in arsenic resistance or arsenic metabolism: several genes encoding HTH-type, ArsR family transcriptional regulators, arsB encoding the membrane arsenic pump protein (locus_tag: Br6_00718), aioB encoding the arsenite oxidase subunit AioB precursor (locus_tag: Br6_00766), arsA encoding the arsenic pump-driving ATPase (locus_tag: Br6_00780 and Br6_04451), arsC encoding the arsenate reductase (locus_tag: Br6_01479), and a cluster of three genes (locus_tag: Br6_04407, Br6_04408, and Br6_04409; UniProt: E4WCD3, E4WCD4, and E4WCD5) which appear to form the arsRBC operon 18. The draft genome contains a nitrate reductase gene cluster (locus_tag: Br6_04375, Br6_04376, Br6_04377, and Br6_04378) and denitrification regulatory proteins (locus_tag: Br6_04870 and Br6_04921), but does not contain nitrogenase genes. As expected by the fact that Rhodococcus is non-motile, the Br-6 genome did not contain any genes involved in motility.

http://www.ncbi.nlm.nih.gov/pubmed/25977440
Genome Announc. 2015 May 14;3(3). pii: e00472-15. doi: 10.1128/genomeA.00472-15.
Draft Genome Sequence of Anaeromyxobacter sp. Strain PSR-1, an Arsenate-Respiring Bacterium Isolated from Arsenic-Contaminated Soil.
Tonomura M, Ehara A, Suzuki H, Amachi S.
Here, we report a draft genome sequence of Anaeromyxobacter sp. strain PSR-1, an arsenate-respiring bacterium isolated from arsenic-contaminated soil. It contained three distinct arsenic resistance gene clusters (ars operons), while no respiratory arsenate reductase gene (arr) was identified.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4432346/
- The genome contains three distinct arsenic resistance gene clusters (ars operons), each of which includes a detoxifying arsenate reductase gene (arsC). One consists of genes for a regulatory protein (arsR), an arsenite efflux pump (arsB), a pump-driving ATPase (arsA), and a chaperone of the pump (arsD) (arsD-arsA-uspA-arsR-arsB-arsC-arsD-ORFs-arsA). The second one consists of similar genes as well as an arsenite methyltransferase gene (arsM) (arsD-arsC-uspA-arsB-arsR-arsM). Respiratory arsenate reductase genes (arrAB) were not identified, despite a unique characteristic of this strain to respire arsenate. The draft genome contains at least 64 genes coding for c-type cytochromes, among which 54 are multiheme cytochromes. 
- A nearly complete set of genes for the reductive acetyl-CoA pathway was found, except for the formate-tetrahydrofolate ligase gene, but there are no other autotrophic CO2 fixation pathway genes.

http://www.ncbi.nlm.nih.gov/pubmed/25635012
Genome Announc. 2015 Jan 29;3(1). pii: e01478-14. doi: 10.1128/genomeA.01478-14.
Draft Genome Sequence of Geobacter sp. Strain OR-1, an Arsenate-Respiring Bacterium Isolated from Japanese Paddy Soil.
Ehara A, Suzuki H, Amachi S.
Here, we report a draft genome sequence of Geobacter sp. strain OR-1, an arsenate-respiring bacterium isolated from Japanese paddy soil. It contained two distinct arsenic islands, one including genes for a respiratory arsenate reductase (Arr) as well as for arsenic resistance (arsD-arsA-acr3-arsR-arrA-arrB) and the second containing only genes for arsenic resistance.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4319506/
- The genome contains two distinct arsenic islands. One consists of genes for a respiratory arsenate reductase (arrAB), and they are also flanked with genes for arsenic resistance including arsR for a regulatory protein, acr3 for an arsenite efflux pump, arsA for a pump-driving ATPase, and arsD for a chaperone of the pump. Another island consists mainly of genes for arsenic resistance including that for a detoxifying arsenate reductase ArsC (arsR-arsC-acr3-uspA-uspA-ORF-arsA-arsD-ORFs-arsD). The draft genome contains at least 83 genes coding for c-type cytochromes, among which 73 are multiheme cytochromes. 
- A nearly complete set of genes for a reductive acetyl-CoA pathway, except for the formate-tetrahydrofolate ligase gene, is found, but there are no other autotrophic CO2 fixation pathway genes such as ribulose-1,5-bisphosphate carboxylase/oxygenase and ATP-citrate lyase.

http://www.ncbi.nlm.nih.gov/pubmed/24994802
Genome Announc. 2014 Jul 3;2(4). pii: e00659-14. doi: 10.1128/genomeA.00659-14.
Draft genome sequence of strain Q-1, an iodide-oxidizing alphaproteobacterium isolated from natural gas brine water.
Ehara A, Suzuki H, Kanesaki Y, Yoshikawa H, Amachi S.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4082002/
- There are no autotrophic CO2 fixation pathway genes such as those for ribulose-1,5-bisphosphate carboxylase/oxygenase and components of the reductive tricarboxylic acid (TCA) cycle in the draft genome sequence. A key gene encoding phosphofructokinase for glycolysis is absent, but a complete set of genes for the TCA cycle, pentose phosphate pathway, and Entner-Doudoroff pathway are predicted. 

----------








----------


